<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Final//EN">
<HTML>

<HEAD>
<TITLE>Phylogeny Programs (continued)</TITLE>
</HEAD>

<BODY BGCOLOR="#CCFFFF">

<A HREF="software.html">To go to top of the Software pages</A>
<P>
<A HREF="software.etc1.html#bottom">To previous part of Software pages</A>
<P>

<HR>
<A NAME="REDCON"><P>
Mark Wilkinson,</A>
of the Department of Zoology, The Natural History Museum, London, U.K.
(<CODE>marw<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>nhm.ac.uk</CODE>) has
produced <STRONG>REDCON</STRONG>, version 3.0, a program to implement his
method of
reduced consensus trees.  These find a tree with possibly fewer species that
satisfies a strict or a majority rule consensus criterion.  REDCON reads
trees in <A HREF="software.pars.html#PAUP">PAUP*</A> format.  It is a DOS
executable, and is available at
<A
HREF="http://www.nhm.ac.uk/research-curation/research/projects/software/mwphylogeny.html">his
software Web site</A>
at
<CODE>http://www.nhm.ac.uk/research-curation/research/projects/software/mwphylogeny.html</CODE>.
<P>

<HR>
<A NAME="TAXEQ3"><P>
Mark Wilkinson,</A>
of the Department of Zoology, The Natural History Museum, London, U.K.
(<CODE>marw<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>nhm.ac.uk</CODE>) has
produced <STRONG>TAXEQ3</STRONG>, a program to carry out Safe Taxonomic
Reduction, which means dropping some species to get a set whose phylogenetic
relationships are less ambiguous.  The method is described in a paper:
Wilkinson, M. 1995.  Coping with abundant missing entries in phylogenetic
inference using parsimony.  <EM>Systematic Biology</EM> <B>44:</B> 501-514.
TAXEQ3 is distributed
as a DOS executable with documentation and sample data set from
<A
HREF="http://www.nhm.ac.uk/research-curation/research/projects/software/mwphylogeny.html">his
software Web site</A>
at
<CODE>http://www.nhm.ac.uk/research-curation/research/projects/software/mwphylogeny.html</CODE>.
<P>

<HR>
<A NAME="RadCon"><P>
<IMG SRC="icons/radcon.gif" ALT="[RadCon icon]">
Joseph Thorley</A> of Poisson Consulting, Nelson, B.C., Canada
(<TT>info<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>poissonconsulting.com</TT>)
and Rod Page of the University of Glasgow have
written <STRONG>RadCon</STRONG>, version 1.1.6, a program to compute consensus
trees, supertrees, measures of the shape of trees, and to rearrange trees.
It can compute strict, semi-strict, Adams, and
majority-rule consensus trees, Reduced consensus trees, MRP supertrees,
Cladistic Information Content, Leaf Stability, and Double Decay
Analysis.  It can also measure the shape and
resolution of trees, prune and regraft leaves, add outgroups, and reroot
trees.  It is described in a paper:
Thorley, J. L. and R. D. M. Page. 2000. RadCon: Phylogenetic comparison and
consensus. <EM>Bioinformatics</EM> <B>16:</B> 486-487.
RadCon is a Mac OS executable for Mac OS 7.5 or later.  RadCon is available at
<a href="http://www.poissonconsulting.ca/index.php/software/">its web site</a>
at
<code> http://www.poissonconsulting.ca/index.php/software/</code>
and its manual can also be downloaded or viewed from there.
<P>

<HR>
<A NAME="RobinsonFoulds"><P>
Vladimir Makarenkov</A>
(<CODE>makarenv<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>magellan.umontreal.ca</CODE>), then
of the D&eacute;partement de Sciences Biologiques of the
Universit&eacute; de Montr&eacute;al, Qu&eacute;bec and now
of the Departement d'Informatique of the
Universit&eacute; du Qu&eacute;bec &agrave; Montr&eacute;al and
the D&eacute;partement de Sciences Biologiques of the
Universit&eacute; de Montr&eacute;al, Qu&eacute;bec has written
a program to compute the
<strong>Robinson and Foulds topological distance</strong>.  This is
a distance measure between trees, counting the numbers of branches on the
two trees that have no counterpart on the other tree.  The algorithm used
is described in a paper:
Makarenkov, V., and Leclerc, B. 2000. An optimal way to compare additive trees
using circular orders. <em>Journal of Computational Biology</em> <b>7:</b>
731-744.  The program is available as C source code and as DOS executable
and a Mac OS executable from
<a href="http://www.bio.umontreal.ca/Casgrain/en/labo/robinson_foulds.html">its
web site</a>
at
<code>http://www.bio.umontreal.ca/Casgrain/en/labo/robinson_foulds.html</code>.
<P>

<A NAME="TopD/fMts">
<HR>
<P>
<IMG SRC="icons/topd.gif" ALT="[TopD/fMts icon]">&nbsp;&nbsp;Pere Puigb&ograve;
Avalos</A>
of the Evolutionary Genomics Group, Biochemistry and Biotechnology Department
of  Rovira i Virgili University, Tarragona, Spain
<tt>(ppuigbo<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;urv.cat)</tt>
has produced <strong>TopD/fMts</strong>
(TOPological Distance / From Multiple To Single),
version 3.3d, program to calculate distances between trees.  TopD calculates
distances between trees by the following methods: split distance. nodal
distance, disagreement, taxa in common, and quartets distance. fMtS is used
with it to subsample gene families to get trees with one copy per species.
TopD/fMts can also list the taxa that are responsible for the disgreement
between two trees.  It is described in the paper:
Puigb&ograve; P., S. Garcia-Vallv&eacute; and J. O. McInerney. 2007. TOPD/FMTS:
a new software to compare phylogenetic trees. <em>Bioinformatics</em>,
published online on April 25, 2007.
It is available as Perl script.  It can be downloaded from
<a href="http://genomes.urv.es/topd/">its web site</a>
at <code>http://genomes.urv.es/topd/</code>
<P>

<HR>
<A NAME="Clann"><P>
Chris Creevey</A>, then of the Bioinformatics
and Pharmacogenomics Laboratory at National University of Ireland Maynooth
(and now of Teagasc (The Irish Agriculture and Food Development Authority) in
Carlow, Ireland,
has written <strong>Clann</strong> (the Irish word for "family") version
3.0.2, a program to computer supertrees.  It implements most of the major
supertree methods, including matrix representation by parsimony (MRP),
methods involving distance matrices, quartets methods, and splits
methods, and includes a bootstrap method that samples from among the
input trees.  Clann is described in a paper: Creevey, C. J. and J. O.
McInerney.
2005.  Clann: investigating phylogenetic information through supertree
analyses. <em>Bioinformatics </em> <b>21:</b> 390-392.  It is available
as Windows, Mac OS X, and Linux executables from <a
href="http://bioinf.may.ie/software/clann/">its web site</a> at
<code>http://bioinf.may.ie/software/clann/</code>.
<P>

<HR>
<A NAME="SuperTree"><P><IMG SRC="icons/SuperTree.gif" ALT="[SuperTree icon]">
Nicolas Salamin</A> at the Department of Ecology and
Evolution of the University of Lausanne, Switzerland has written
<strong>SuperTree</strong>, version 0.85b, (not, of course to be confused with
<a HREF="software.etc2.html#Supertree">Supertree</a>) a program to take a set
of input trees and build a matrix
representation of them that can be used to compute a supertree using the
MRP (matrix representation by parsimony) method.  The program does not do
the parsimony reconstruction from this matrix itself.  The options available
are described in a paper: Salamin, N., T. R. Hodkinson, V. Savolainen.
2002. Building supertrees: An empirical assessment using the grass family
(Poaceae).  <em>Systematic Biology</em> <b>51:</b> 136 - 150.
SuperTree is a Java runtime executable which will work on
Linux, Windows, and Mac OS X systems, as long as they have the Sun Java
Runtime JRT system version 1.3 or greater.  The Java executables are
distributed in Windows and in Linux/Mac versions.  The Java source code can be
obtained from Salamin, if desired, by contacting him by email.
The program and its distribution are briefly described at
<a href="http://www2.unil.ch/phylo/bioinformatics/supertree.html">its site</a>
at
<code>http://www2.unil.ch/phylo/bioinformatics/supertree.html</code>&nbsp;
The program can be obtained from Salamin by emailing him
(<tt>wwwphylo<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;unil.ch</tt>).
<P>

<HR>
<A NAME="Supertree_scripts"><P> Olaf Bininda-Emonds at
the Institut f&uuml;r Biologie und Umweltwissenschaften of the
Universit&auml;t Oldenburg, Germany
(<tt>olaf.bininda<!deathtospam>&nbsp; (at)
<!deathtospam>&nbsp;uni-oldenburg.de</tt>) has produced a set of Perl scripts
which, taken together,
are a <strong>Supertree package</strong> which can make and examine supertrees
using the program <a href="software.pars.html#PAUP">PAUP*</a>.  Although they
are each downloadable
separately, I will consider them here to be a single entity.  They include,
in rough order of the steps carried out:
<UL>
<li><b>synonoTree.pl</b>, which standardizes the taxon labels in a set of
source trees according to a reference taxonomy.
<li> <b>treePruner.pl</b>, which takes a set of NEXUS-formatted trees and
prunes them to the set of tips that they all have in common.
<li><b>SuperMRP.pl</b>, which converts a NEXUS-formatted treefile into a NEXUS-
formatted MRP (Matrix Representation by Parsimony) data set ready for
analysis by <a href="software.pars.html#PAUP">PAUP*</a>.
<li> <b>QualiTree.pl</b>, which calculates support for clades in a supertree
relative to the clades in the source trees that made it up.
<li> <b>ReverseMRP.pl</b> which does the reverse, making the NEXUS MRP data set
into a treefile of trees.
<li> <b>BootStrip.pl</b> determines the bootstrap frequencies for a given
phylogenetic tree from results of a bootstrap analysis.
<li> <b>chronoGrapher.pl</b> makes branch lengths for a tree out of divergence
times.
<li> <b>partitionMetric.pl</b> calculates the number of unique clades /
partitions for pairs of trees pruned to their common taxon set.
<li> <b>relDate.pl</b> gets relative branch length formulas according to a
local molecular clock. 
boundaries of the source trees are specified (as nexus-format CHARSETs).
</UL>
Additional Perl scripts are also present to help with labeling the
resultant supertrees and interconverting tree formats.
<P>
They are described in the papers:
<ul>
<li>Bininda-Emonds, O. R. P. 2003. Novel versus unsupported clades:
assessing the qualitative support for clades in MRP supertrees.
<em>Systematic Biology</em> <b>52:</b> 839-848.
<li>Bininda-Emonds, O.R.P., K.E. Jones, S.A. Price, M. Cardillo, R.
Grenyer, and A. Purvis. 2004. Garbage in, garbage out: data issues in
supertree construction. Pp. 267-280 in <em>Phylogenetic Supertrees:
Combining Information to Reveal the Tree of Life</em>, ed. O. R. P.
Bininda-Emonds. Computational Biology, volume 4. Kluwer Academic
Publishers, Dordrecht, the Netherlands.
<li>Bininda-Emonds, O.R.P., R.M.D. Beck, and A. Purvis. 2005. Getting to
the roots of matrix representation. <em>Systematic Biology</em> <b>54:</b>
668-672.
</ul>
As they are Perl scripts, they can be run on any system on which Perl has been
made available.  They can be downloaded (individually) from
<a
href="http://www.molekularesystematik.uni-oldenburg.de/33997.html">Bininda-Emonds's
software web page</a>
at <code>http://www.molekularesystematik.uni-oldenburg.de/33997.html</code>
<P>

<A NAME="Supertree">
<HR>
<P>
<IMG SRC="icons/supertree.gif" ALT="[Supertree icon]">&nbsp;&nbsp;Rod Page</A>
of the Environmental and Evolutionary Biology
of the Institute of Biomedical and Life Sciences, University of Glasgow, U.K.
<tt>(r.page<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;bio.gla.ac.uk)</tt>
has written <strong>Supertree</strong> (not, of course to be confused with
<a HREF="software.etc2.html#SuperTree">SuperTree</a>) version 0.3, for
constructing supertrees.  Supertree is an experimental command line program for
constructing supertrees. It implements Semple and Steel's algorithm
MinCutSupertree (original and a modified version due to Page), as well as MRP
coding. It can also compute cluster graphs.
It is described in the paper:
Page, R. D. M. 2002.  Modified MinCut supertrees. pp. 537-551 in <em>Algorithms
in Bioinformatics: Proceedings of the Second International Workshop, WABI 2002,
Rome, Italy, September 17-21, 2002.</em> Springer-Verlag, Heidelberg.
It is available as C source code, Mac OS X PowerMac executables and Mac OS 9
executables.  To compile the source code, one needs a copy of the GTL
Graph Template Library, which must be installed before compiling.
Supertree can be downloaded from
<a href="http://darwin.zoology.gla.ac.uk/%7Erpage/supertree/">its web site</a>
 at  <code>http://darwin.zoology.gla.ac.uk/%7Erpage/supertree/</code>
<P>

<A NAME="HeuristicMRF2">
<HR>
<P>
Duhong Chen,</A>
then of the Computational Biology Laboratory of the Department of Computer
Science
of  Iowa State University
<tt>(duhong<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;iastate.edu)</tt>
has written <strong>HeuristicMRF2</strong> (Matrix Representation with
Flipping version 2),
a program to construct supertrees by the MRF method.
Matrix representation with flipping (MRF) starts with a Matrix Representation
by Parsimony matrix.  The binary MRP matrix from a rooted input tree may be
transformed into a subset of the columns of a rooted supertree by changing some
of the 0's to 1's and 1's to 0's.  Each change in the character state is a
flip, and the minimum number of flips needed to transform the input tree into a
supertree is the flip distance. The MRF heuristic seeks the rooted supertree(s)
that minimizes the total flip distance from all input trees.  It searches for
rooted trees, not unrooted trees.  It is described in the paper:
Chen, D., L. Diao, O. Eulenstein, D. Fern&aacute;ndez-Baca, and M. J.
Sanderson.  2003.  Flipping: A supertree construction method.  Pp. 135-160 in
<em>Bioconsensus.</em>, ed. M. F. Janowitz, F.-J. Lapointe, F. R. McMorris,
B. Mirkin, and F. S. Roberts. DIMACS series in discrete mathematics and
theoretical computer science, Volume 61. American Mathematical Society,
Providence, Rhode Island.
It is available as C++ source code.  It can be downloaded from
<a href="http://genome.cs.iastate.edu/CBL/download/">its web site</a>
at <code>http://genome.cs.iastate.edu/CBL/download/</code>
<P>

<A NAME="Quartet_Suite">
<HR>
<P>
Raul Piaggio</A>
of the Computational Biology Laboratory, Department of Computer Science
at  Iowa State University, Ames, Iowa
<tt>(rpiaggio<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;iastate.edu)</tt>
has written <strong>Quartet Suite</strong>
version 1.0, a set of programs for computing supertrees and also distances 
between trees from quartets.  Quartet Suite is a set of four programs that 
take input trees, break them down into the set of quartets implied by each of 
them, and construct a supertree based on these quartets.  They can also 
compute a distance between trees from the sets of quartets they imply.
The methods are described in the paper:
Willson, S.J. 1999. Building phylogenetic trees from quartets by using local
inconsistency measures. <em>Molecular Biology and Evolution</em> <b>16:</b>
685-693.
It is available as C++ source code, Windows executables and Powermac Mac OS X
executables.  It can be downloaded from
<a href="http://genome.cs.iastate.edu/CBL/download/">its web site</a>
at <code>http://genome.cs.iastate.edu/CBL/download/</code>
<P>

<A NAME="Rainbow">
<HR>
<P>
Duhong Chen, Oliver Eulenstein, and David Fern&aacute;ndez-Baca</A>
of the Computational Biology Laboratory of the Department of Computer Science
at  Iowa State University, Ames, Iowa
<tt>(duhong<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;iastate.edu)</tt>
have released <strong>Rainbow</strong>
version beta 1.2, a toolbox for phylogenetic supertree construction and 
analysis.  Rainbow provides a user-friendly environment in which scientists 
can utilize tools for building and analyzing supertrees. Rainbow provides a 
graphic user interface (GUI) to construct supertrees using several different 
methods.  Currently these include matrix representation with flipping (MRF) 
matrix representation with parsimony (MRP), and the modified Mincut algorithm 
(MMC). Rainbow also provides tools to analyze the quality of the inferred 
supertrees.  The methods are described in the paper:
Chen, D., L. Diao, O. Eulenstein, D. Fern&aacute;ndez-Baca, and M. J.
Sanderson.  2003.  Flipping: A supertree construction method.  Pages 135-160 in
<em>Bioconsensus</em>, ed. M. F. Janowitz, F.-J. Lapointe, F. R. McMorris, B.
Mirkin, and F. S. Roberts. Volume 61 in DIMACS series in discrete mathematics
and theoretical computer science, American Mathematical Society, Providence,
Rhode Island.
It is available as C++ source code, Windows executables, Linux executables
and Powermac Mac OS X executables.  It can be downloaded from
<a href="http://genome.cs.iastate.edu/CBL/download/">its web site</a>
at <code>http://genome.cs.iastate.edu/CBL/download/</code>
<P>

<A NAME="PhySIC">
<HR>
<P>
Vincent Ranwez, Vincent Berry, Alexis Criscuolo, Pierre-Henri Fabre,
Sylvain Guillemot, Celine Scornavacca and Emmanuel Douzery</A>
of the Institut des Sciences de L'evolution and the LIRMM
at the Universit&eacute; Montpellier II, France
<tt>(vberry<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;lirmm.fr)</tt>
have produced <strong>PhySIC_IST</strong>
(Phylogenetic Supertrees with Induction and non-Contradiction),
a supertree program that uses an algorithm they developed.  It carries out
construction of supertrees using the non-contradiction property (PC) and the
induction property (PI). The former requires that the supertree does not
contain relationships that contradict one or a combination of the source
topolo- gies, while the latter requires that all topological information
contained in the supertree is present in a source tree or collectively induced
by several source trees.  The program also can collapse branches in the input
trees that have bootstrap values that are smaller than a threshold set by the
user.
It is described in the papers:
<UL>
<LI> Ranwez, V., V. Berry, A. Criscuolo, P.H. Fabre, S. Guillemot, C.
Scornavacca
and E.J.P. Douzery. 2007. PhySIC: a veto supertree method with desirable
properties. <em>Systematic Biology</em> <b>56 (5):</b> 798-817.
<LI> Scornavacca, C., V. Berry, V. Lefort, E, J, P. Douzery and V. Ranwez.
2008.  PhySIC_IST: cleaning source trees to infer more informative supertrees.
<I>BMC Bioinformatics</I> <b>9:</b> 413
</UL>
It is available as Linux executables and Intel Mac OS X executables.  It can be
downloaded from
<a href="http://www.atgc-montpellier.fr/physic_ist/binaries.php">its web
site</a>
at <code>http://www.atgc-montpellier.fr/physic_ist/binaries.php</code>&nbsp;
It is available as a web server <a
href="http://www.atgc-montpellier.fr/physic_ist/">here</a>.
An earlier version PhySIC, is downloadable <a
href="http://www.atgc-montpellier.fr/physic/binaries.php">here</a> at
<code>http://www.atgc-montpellier.fr/physic/binaries.php</code>
and it is also available as a web server <a href="software.serv.html#ATGC
servers">here</a>.
<P>

<A NAME="PhyloSort">
<HR>
<P>
Ahmed Moustafa</A>, then
of the Computational Genetics program
at the University of Iowa, Iowa City, Iowa
<tt>(ahmed<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;users.sourceforge.net)</tt>
and currently in Bioinformatics and Genomics at the
Department of Biology and Biotechnology Graduate Program of the School of
Science and
Engineering, American University in Cairo, Egypt
(<tt>ahmed<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;pobox.com</tt>)
has written <strong>PhyloSort</strong>
version 1.3, a Java tool for sorting phylogenies searching for user-specified
subtrees that a particular monophyletic group.  PhyloSort is for
<ul>
<li>Searching for monophyletic relationship among groups of taxa.
<li>Filtering by bootstrap support values associated with the monophyletic
clades.
<li>Filtering by tree complexity (number of taxa in a tree).
<li>Filtering by family complexity (number of genes per taxon in a tree).
<li>Clustering trees (genes) into tree clusters (gene families).
<li>Using a reference tree to taxonomically group (tree-format).
</ul>
It is available as Java source code and Java executables. It is described in
the paper:  Moustafa, A. and D. Bhattacharya. 2008. PhyloSort: A
user-friendly phylogenetic sorting tool and its application to estimating the
cyanobacterial contribution to the nuclear genome of Chlamydomonas. <i>BMC
Evolutionary Biology</i> <b>8:</b> 6.
It can be downloaded from
<a href="http://phylosort.sourceforge.net/">its web site</a>
at <code>http://phylosort.sourceforge.net/</code>
<P>

<A NAME="CONSERVE"><P>
<HR>
<P>
<IMG SRC="icons/conserve.gif" ALT="[CONSERVE icon here]">
The late (and lamented) Ross Crozier</A>
of the School of Tropical Biology, James Cook University, Townsville, Australia
and Paul-Michael Agapow, then of the
Department of Biology of Imperial College at Silwood Park, U.K. and more
recently of the Health Protection Agency, U.K.
(<tt>agapow<!deathtospam>&nbsp;
<!deathtospam>(at)&nbsp;<!deathtospam>agapow.net</tt>)
have written <STRONG>CONSERVE</STRONG>, currently at version IV, 1.4b Beta,
a program to use phylogenetic information to calculate biodiversity and test 
the feasability of conservation schemes.  It measures the distinctiveness of
species using genetic distances and also to test whether particular assemblages
of populations preserve statistically significantly more biodiversity
than other assemblages.  Biodiversity is determined using GD (probability
of more than one allele) or PD (length of evolutionary history) methods,
from data in the form of unrooted trees produced in standard treefile format.
It is available as executables for Mac OS X, for Windows, and for Linux
from <A HREF="http://www.agapow.net/software/conserve">Agapow's web site</A> at
<CODE>http://www.agapow.net/software/conserve</CODE>.   
An earlier 680x0 Mac OS version (version III 3.2.2) is available
<a href="http://www.bio.ic.ac.uk/evolve/software/conserve/index.html">here</a>
at <code>http://www.bio.ic.ac.uk/evolve/software/conserve/index.html</code>.
(<a href="http://www.agapow.net/science/conservation/ross-crozier-1943-2009">Here</a>,
<a href="http://gap.entclub.org/taxonomists/Crozier/index.html">here</a>, and
<a href="http://myrmecos.wordpress.com/2009/11/12/ross-crozier/">here</a> are
some of the many tributes to the late Ross Crozier).
<P>

<HR>
<A NAME="TreeDis"><P>
Georg Weiller</A>, of the Genomic Interactions Group at the Research
School of Biological Sciences of the Australian National University, Canberra
(<TT>georg.weiller<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>rsbs.anu.edu.au</TT>) has released
<STRONG>TreeDis</STRONG>
version 2.0.  TreeDis finds the patristic distances (total length of branches
between all pairs of taxa in a phylogeny.  It takes as input the tree file in
Newick standard form or in the format for NJTREE.
It is distributed as a DOS executable (a C++ source code version can also
be obtained from Weiller).  It is available from
<A
HREF="http://www.rsbs.anu.edu.au/Products&Services/BiotechnologyTransferUnit/tredis.php">its
web site</A>
at
<TT>http://www.rsbs.anu.edu.au/Products&Services/BiotechnologyTransferUnit/tredis.asp</TT>.
<P>

<A NAME="apTreeshape">
<HR>
<P>
Nicolas Bortolussi, Eric Durand, Michael Blum, and Olivier Fran&ccedil;ois</A>,
then of the Institut d'Informatique et Math&eacute;matiques Appliqu&eacute;es
in Grenoble, France (Durand is now the the Department of Integrative Biology,
University of California, Berkeley and his email address is
<tt>eric.durand<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;berkeley.edu</tt>)
have written <strong>apTreeshape</strong>
(<em>a</em>nalyses of <em>p</em>hylogenetic <em>Treeshape</em>),
version 1.4-3, an R package for simulation and analysis of phylogenetic tree 
topologies using statistical indices .  apTreeshape computes a variety of 
statistics and tests on tree shape.  It is a companion library of the
<a href="#APE">APE</A>
package. It provides additional functions for reading, plotting, manipulating 
phylogenetic trees. It also offers convenient web-access to public databases, 
and enables testing null models of macroevolution using corrected test 
statistics. Trees of class "phylo" (from the APE package) can be converted
easily.
It is available as an R package, Windows executables and Powermac Mac OS X
executables.  It can be downloaded from
<a href="http://cran.r-project.org/web/packages/apTreeshape/index.html">its web
site</a>
at  <code>http://cran.r-project.org/web/packages/apTreeshape/index.html</code>
<P>

<A NAME="TreeStat">
<HR>
<P>
<IMG SRC="icons/treestat.gif" ALT="[TreeStat icon]">&nbsp;&nbsp;Andrew Rambaut 
of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
and Alexei Drummond of
the University of Auckland, New Zealand
have written <strong>TreeStat</strong>
(Tree Statistics),
version 1.2, a phylogeny package with parsimony, distance and likelihood
methods.  TreeStat is an application that can process a set of trees in a
PHYLIP or NEXUS format tree file and calculate a number of summary statistics
for each. These are saved in a tab-delimited file for analysis in Tracer or
statistics packages. A range of summary statistics are included:
<UL>
<LI> Tree-Balance Statistics. Colless's, B1, N_bar, cherry count.
<LI> Tree shape.Tree height, node heights, treeness, Pybus' gamma.
<LI> Population genetic. Fu and Li's D, external/internal ratio.
<LI> Other. Tree length, root-to-tip lengths.
</UL>
It is available as Java executables, Windows executables and Mac OS X universal
executables.  It can be downloaded from
<a href="http://tree.bio.ed.ac.uk/software/treestat/">its web site</a>
at <code>http://tree.bio.ed.ac.uk/software/treestat/</code>
<P>

<A NAME="PhyRe">
<HR>
<P>
<img src="icons/phyre.jpg">&nbsp;&nbsp; Federico Plazzi, Ronald Robert Ferrucci, and Marco Passamonti</A>
of the Department of Experimental Evolutionary Biology
of the University of Bologna, Italy
<tt>(federico.plazzi<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;unibo.it)</tt>
have released <strong>PhyRe</strong>
(PHYlogenetic REpresentativeness),
version 1.0, a package which aims to estimate taxonomic coverage of 
an ingroup.  PhyRe implements a method to evaluate and assess taxon sampling 
in a phylogenetic study by means of Average Taxonomic Distinctness and related 
indices and algorithms. It was conceived as an a priori checkpoint to be 
performed before starting an evolutionary study, whether based on morphology 
or molecules. The method uses a tree of
the group which is being investigated; PhyRe can also test the stability 
of results upon taxonomical revisions.
It is described in the paper:
Plazzi, F., R. R. Ferrucci and M. Passamonti. 2010. Phylogenetic Representativeness: A new method
for evaluating taxon sampling in evolutionary studies. <i>BMC
Bioinformatics</i> <b>11:</b> 209.
It is available as Python script and Windows executables.  It can be downloaded from
<a href="http://www.mozoolab.net/index.php/software-download.html">its web site</a>
at <code>http://www.mozoolab.net/index.php/software-download.html</code>
<P>

<HR>
<A NAME="CAIC"><P>
<IMG SRC="icons/caic.gif" ALT="[CAIC icon here]">
Andrew Purvis
of the Department of Biology, Imperial College, Silwood Park, U.K.
(<CODE>a.purvis<!deathtospam>&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>imperial.ac.uk</CODE>)
and Andrew Rambaut</A>
of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
have written <STRONG>CAIC</STRONG> (Comparative Analysis of
Independent Contrasts), versions 2.6.9 and 3-Alpha.  It carries out the
contrasts method but with some modifications by others to
cope with lack of resolution of the phylogeny.
It is described in the paper: Purvis, A. and A. Rambaut. 1995. Comparative
analysis by independent contrasts (CAIC): an
Apple Macintosh application for analysing comparative data.  <EM>Computer
Applications in the Biosciences (CABIOS)</EM> <B>11:</B> 247-251.
It will run on Macintosh Mac OS, and is available free from
<A HREF="http://www.bio.ic.ac.uk/evolve/software/caic/index.html">CAIC's Web
page</A>
<CODE>http://www.bio.ic.ac.uk/evolve/software/caic/index.html</CODE>.
It is also available as an "alpha" release of an R package for CAIC that 
implements its methods is available at
<a href="http://r-forge.r-project.org/projects/caic/">its page at R-Forge</a>
at <code>http://r-forge.r-project.org/projects/caic/</code>
<P>

<HR>
<A NAME="COMPARE"><P>
Em&iacute;lia Martins</A> (<CODE>emartins<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>indiana.edu</CODE>), of the
Department of Biology of the University of Indiana, Bloomington, Indiana,
has released
version 4.6b of <STRONG>COMPARE</STRONG>, a package of programs for comparative
methods analysis.  COMPARE includes various programs for conducting
statistical analyses of comparative data in a phylogenetic
context. At the moment, it includes programs to compute
independent contrasts, do spatial autocorrelation analyses, sum of
squares parsimony, generate
random data, trees and/or branch lengths, and various other
things.
COMPARE is written in Java and is available both as standalone Java
(including source code)
and as a Java application that is provided by their site to your
browser.  It requires a Java runtime
environment.  COMPARE is available from
<A HREF="http://compare.bio.indiana.edu/">its web site</A> at
<CODE>http://compare.bio.indiana.edu/</CODE>.
<P>

<HR>
<A NAME="PDAP"><P>
Theodore Garland, Jr.</A>, of the Department of Biology of the University
of California, Riverside (<CODE>tgarland<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ucr.edu</CODE>)
and his colleagues (Jason A. Jones, Allan W. Dickermann, Peter E.
Midford, and Ramon Diaz-Uriarte)
have developed <STRONG>PDAP</STRONG> version 6.0,
Phenotypic Diversity Analysis Programs, a series of DOS programs to
perform various comparative analyses.
At present, the following phylogenetically based statistical methods are
included:  independent contrasts, squared-change parsimony reconstructions of
ancestral states and estimation of evolutionary correlations, and phylogenetic
analysis of covariance via computer-simulated (Monte Carlo) null distributions.
PDTREE can also read, write, and edit trees. 
PDAP distribution is described in
<A HREF="http://www.biology.ucr.edu/people/faculty/Garland/PDAP.html">a web
page</A>
at <CODE>http://www.biology.ucr.edu/people/faculty/Garland/PDAP.html</CODE>.
The original published description of PDAP is the paper:
Garland, T., Jr., A. W. Dickerman, C. M. Janis, and J. A. Jones. 1993. 
Phylogenetic analysis of covariance by computer simulation. <EM>Systematic
Biology</EM> 42:265-292. 
The methods used are described in a number of recent papers by these
authors, including:
<UL>
<LI>Purvis, A., and T. Garland, Jr.
1993. Polytomies in comparative analyses of continuous characters.
<EM>Systematic Biology</EM> <B>42:</B> 569-575.
<LI>Diaz-Uriarte, R., and T. Garland, Jr. 1996. Testing
hypotheses of correlated evolution using phylogenetically independent
contrasts: sensitivity to deviations from Brownian motion.
<EM>Systematic Biology</EM> <B>45:</B> 27-47.
<LI>Garland, T., Jr, P. E. Midford, and A. R. Ives. 1999. An introduction to
phylogenetically based statistical methods, with a new method for confidence
intervals on ancestral states.  <EM>American Zoologist</EM> <B>39:</B>
374-388. 
<LI>Garland, T., Jr., and A. R. Ives. 2000. Using the past to predict the
present: 
Confidence intervals for regression equations in phylogenetic comparative
methods. <EM>American Naturalist</EM> 155:346-364. 
</UL>
PDAP is distributed by email of a self-extracting
executable file, obtainable for free (contact Garland by e-mail at the above
address).  Alternatively, a DOS disk can be mailed.
<P>

<A NAME="IDC">
<HR>
<P>
Liam Revell</A>, then of the Department of Organismic and Evolutionary Biology
of Harvard University (currently at the National Evolutionary Synthesis
Center, Duke University, Durham, North Carolina, email address
<tt>(lrevell<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;nescent.org)</tt>
has released <strong>IDC</strong> (IndepenDent Contrasts program),
version 1, a program for the calculation of
phylogenetically indepedent contrasts.  This program calculates
contrasts for multiple trees, multiple datasets, or both.
It also returns a VCV matrix and the correlation matrix of the
independent contrasts.  It is available as C source code and Windows
executables.  It can be downloaded from
<a href="http://anolis.oeb.harvard.edu/~liam/programs/">its web site</a>
at <code>http://anolis.oeb.harvard.edu/~liam/programs/</code>
<P>

<HR>
<A NAME="PI"><P>
<IMG SRC="icons/pi.gif" ALT="[PI icon]">
Ehab Abouheif</A> (<tt>ehab.abouheif<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>staff.mcgill.ca</tt>) of
the Department of Biology, McGill University, Montr&eacute;al, Qu&eacute;bec
has written (together with J. Reeve)
<strong>Phylogenetic Independence</strong> version 2.0.  It carries out
Abouheif's Test For Serial Independence (TFSI) on continuously valued
characters and his Runs Test on discretely valued characters.  These are
described in his paper: Abouheif, E. 1999. A method to test the assumption of
phylogenetic independence in comparative data.  <em>Evolutionary Ecology
Research</em> <b>1:</b> 895-909.  The methods employed are validate in the
paper: Pavoine, S., S. Ollierb, D. Pontiera, and D. Chessela.  2008.
Testing for phylogenetic signal in phenotypic traits: New matrices of
phylogenetic proximities. <i>Theoretical Population Biology</i> <b>73:</b>
79-91.  The program is available as Windows
and Linux executables at <A
HREF="http://biology.mcgill.ca/faculty/abouheif/">its web site</A>
at <CODE>http://biology.mcgill.ca/faculty/abouheif/</CODE>.
<P>

<A NAME="BayesTraits">
<HR>
<P>
Mark Pagel and Andrew Meade</A> of the School of Biological Sciences,
of the University of Reading, Reading, U.K.
<tt>(m.pagel<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;reading.ac.uk)</tt>
have released <strong>BayesTraits</strong>, version 1.0, a Bayesian package of 
programs to analyze state evolution of discrete and continuous traits.  It is
performs analyses of trait evolution among groups of species for which a
phylogeny or sample of phylogenies is available. It incoporates their earlier
and separate programs Multistate, Discrete and Continuous. BayesTraits can be
applied to the analysis of traits that adopt a finite number of discrete
states, or to the analysis of continuously varying traits. Hypotheses can be
tested about models of evolution, about ancestral states and about
correlations among pairs of traits.  Parts of the package are described in
these papers:
<UL>
<LI> Pagel, M., A. Meade and D. Barker. 2004.
Bayesian estimation of ancestral character states on phylogenies.
<em>Systematic Biology</em> <b>53:</b> 673-684. 
<LI> Pagel, M., and A.Meade. 2006. Bayesian analysis of correlated evolution of
discrete characters by reversible-jump Markov chain Monte Carlo. <em>American
Naturalist</em> <b>167:</b> 808-825.
<LI> Barker, D. and Pagel, M. 2005. Predicting functional gene links using
phylogenetic-statistical analysis of whole genomes. <em>PLoS Computational
Biology</em> <b>1:</b> 24-31.
</UL>
BayesTraits is available as Windows executables, Linux executables,
Powermac Mac OS X executables, and Intel Mac OS X executables.  It can be
downloaded from
<a href="http://www.evolution.rdg.ac.uk/BayesTraits.html">its web site</a>
at <code>http://www.evolution.rdg.ac.uk/BayesTraits.html</code>
<P>

<A NAME="bms_runner">
<HR>
<P>
Daniel Barker and Mark Pagel, respectively</A>
of the School of Biology of the University of St. Andrews and the School of
Biological Sciences
of the University of Reading, U.K.
<tt>(db60<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;st-andrews.ac.uk)</tt>
have written <strong>bms_runner</strong>
version 1.3, a Perl script to use
<a href="software.etc2.html#BayesTraits">BayesTraits</a> to detect correlations 
in gain and loss of genes.  It uses the separately distributed program 
BayesTraits to predict functional links among genes, on the basis of 
correlated gain and loss of genes from species' genomes.  bms_runner helps 
configure BayesTraits on the basis of user-supplied data, and launches 
BayesTraits repeatedly to seek correlated gain and loss within pairs of genes.
The methods are described in the paper:
Barker, D., Meade, A. and Pagel, M. 2007. Constrained models of evolution lead
to improved prediction of functional linkage from correlated gain and loss of
genes. <em>Bioinformatics</em> <b> 23:</b> 14-20. 
It is available as Perl script.
Use of bms_runner requires that BayesTraits also be installed.
It can be downloaded from
<a href="http://eggg.st-andrews.ac.uk/bms_runner">its web site</a>
at <code>http://eggg.st-andrews.ac.uk/bms_runner</code>
<P>

<A NAME="OUCH">
<HR>
<P>
Aaron King and Marguerite Butler</A>
of the Department of Ecology and Evolutionary Biology
of the University of Michigan, Ann Arbor, Michigan and the Department of
Zoology of the Hawaii at Manoa 
(<tt>aaron.king<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;umich.edu</tt> and
<tt>mbutler<!deathstospam>&nbsp;(at)<!deathtospam>&nbsp;hawaii.edu</tt>)
have written <strong>OUCH</strong>
(Ornstein-Uhlenbeck models models for phylogenetic Comparative Hypotheses),
version 2.7-1, an R package using the Ornstein-Uhlenbeck model for comparative
methods tests.
The package fits different versions of Ornstein-Uhlenbeck models to comparative
data, as described by Thomas Hansen.  It is described in the paper:
Butler, M. A. and A. A. King, 2004. Phylogenetic comparative analysis: a
modeling approach for adaptive evolution. <em>American Naturalist</em>
<b>164:</b> 683-695.
It is available as an R package, and requires the R statistical computing
enironment to be available.  It can be downloaded from
<a href="http://tsuga.biology.lsa.umich.edu/king/ouch/">its web site</a>
 at  <code>http://tsuga.biology.lsa.umich.edu/king/ouch/</code>
<! ?? web site not accessible 11/4/10>
and is also available at
<a href="http://cran.at.r-project.org/web/packages/ouch/">its web site at the
CRAN-R archive site for R packages</a>
at <code>http://cran.at.r-project.org/web/packages/ouch/</code>
<P>

<A NAME="CoMET">
<HR>
<P>
Chunghau Lee and Todd Oakley</A>
of the Ecology, Evolution and Marine Biology Department
of the University of California at Santa Barbara, Santa Barbara, California
<tt>(chunghaulee<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;gmail.com</tt>) and
(<tt>oakley<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;lifesci.ucsb.edu)</tt>
have released <strong>CoMET</strong>
(COntinuous-character Model Evaluation and Testing),
version of 2007-09-22, a Mesquite module for computing likelihoods for a given
tree with Brownian motion models.  COMet is a module for the Mesquite Project.
Give it a tree topology as a starting point and some phenotypic data, and CoMET
tells you the likelihoods of the data evolving through nine different
evolutionary models, including both gradualist and punctuational models. 
It is described in the paper:
Lee, C., S. Blay, A. &Oslash;. Mooers, A. Singh, and T. H.  Oakley. 2006.
CoMET: A Mesquite package for comparing models of continuous character
evolution on phylogenies. <em>Evolutionary Bioinformatics Online</em> <b>2:</b>
193-196.
CoMET requires the <a href="software.pars.html#mesquite">Mesquite</a> Java
framework for phylogenies.
It is available as Java source code and Java executables.  It can be downloaded
from
<a href="http://www.lifesci.ucsb.edu/eemb/labs/oakley/software/comet.htm">its
web site</a>
 at
<code>http://www.lifesci.ucsb.edu/eemb/labs/oakley/software/comet.htm</code>
<P>

<A NAME="Brownie">
<HR>
<P>
Brian O'Meara</A>
of the Center for Population Biology
of the University of University of California, Davis.
<tt>(bcomeara<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ucdavis.edu)</tt>
has released <strong>Brownie</strong>
version 2.1.2, a program for analyzing rates of continuous character evolution.
Brownie looks for substantial rate differences in different parts of a tree
using likelihood ratio tests and Akaike Information Criterion (AIC) statistics.
Brownie 2.1.2 can read Nexus tree and data files and can perform analyses
across
a set of weighted input trees (such as trees weighted by posterior
probabilities or bootstrap values) in order to deal with tree uncertainty.
Brownie deals with subtrees (using the censored test of O'Meara et al) and asks
whether the subtrees differ in rates of evolution.
An earlier version, Brownie 1.0, is written as a series of MATLAB routines and
can be used on systems that have MATLAB installed.
It is described in the paper:
O'Meara, B.C., C. An&eacute;, M.J. Sanderson, and P.C. Wainwright. 2006.
Testing for different rates of continuous trait evolution using likelihood.
<em>Evolution</em> <b>60(5):</b> 922-933.
It is available as C++ source code, Windows executables and Mac OS X universal
executables.  It can be downloaded from
<a href="http://www.brianomeara.info/brownie/">its web site</a>
 at  <code>http://www.brianomeara.info/brownie/</code>
<P>

<HR>
<A NAME="APE"><P>
Emmanuel Paradis</A> (<tt>Emmanuel.Paradis<! death to spam>&nbsp;(at)<! death
to
spam>&nbsp;ird.fr</tt>) of the Institut de Recherche pour le
D&eacute;veloppement, Paris,
France,
has written <strong>APE</strong>, version 2.6-1, (Analysis of Phylogenetics
and Evolution) a package in the R statistical
and graphical language which carries out a variety of phylogeny analyses,
including computation of distances from sequences and gene frequencies,
comparative methods, analyses of diversification, computation of minimum
spanning trees, and estimation of rates of evolution and smoothing of rates
in neighboring lineages.  Some code was contributed by a number of other
people, including Korbinian Strimmer, Julien Claude, Gangolf Jobb, Rainer
Opgen-Rhein, Julien Dutheil, Yvonnick Noel, and Ben Bolker.  APE is described
in
a paper:  Paradis, E., J. Claude and K. Strimmer. 2004. APE: analyses of
phylogenetics and evolution in R language. <em>Bioinformatics</em> <b>20:</b>
289-290.
APE can be downloaded
from <A HREF="http://cran.r-project.org/web/packages/ape/index.html">
its page</A> at the CRAN-R archive site of R programs at
<code>http://cran.r-project.org/web/packages/ape/index.html</code>; it is
described and some additional documentation links given at
<A HREF="http://ape.mpl.ird.fr/">its web site</A>
at <CODE>http://ape.mpl.ird.fr/</CODE>
<P>

<A NAME="Phybase">
<HR>
<P>
Liang Liu and Lili Yu</A>
of the Department of Agriculture and Natural Resources
of the Delaware State University
<tt>(lliu<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;desu.edu)</tt>
has released <strong>Phybase</strong>
(An R package for species tree analysis),
version 1.1, a phylogeny package for manipulating, estimating, and summarizing 
phylogenetic trees, especially species trees.  This package provides functions 
to read, write, manipulate, estimate, and summarize phylogenetic trees 
including species trees which contains not only the topology and branch 
lengths but also population sizes. The input/output functions can read tree 
files in which trees are presented in parenthetic format. The trees are read 
in as a string and then transformed to a matrix which describes the 
relationship of nodes and branch lengths. The nodes matrix provides an easy 
access for developers to further manipulate the tree, while the tree string 
provides interface with other phylogenetic R packages such as "ape". The 
input/output functions can also be used to change the format of tree files 
between NEXUS and PHYLIP. Some basic functions have already been established 
in the package for manipulating trees such as deleting and swapping nodes, 
rooting and unrooting trees, changing the root of the tree. The package also 
includes functions such as "consensus", "coaltime, "popsize", "treedist" for 
summarizing phylogenetic trees, calculating the coalescence time, population 
size, and tree distance. 
It is described in the paper:
Liu, L., and L. Yu. 2010. Phybase: an R package for species tree analysis.
<i>Bioinformatics</i> <b>26:</b> 962-963.
It is available as an R package.  Instructions for downloading it are
available at
<a href="http://cars.desu.edu/faculty/lliu/research/phybase.html">its web
site</a>
at <code>http://cars.desu.edu/faculty/lliu/research/phybase.html</code>
<! SITE SEEMS NOT TO BE THERE 11/6/10>
<P>

<HR>
<A NAME="PHYLOGR"><P></A> Ram&oacute;n D&iacute;az-Uriarte, of the
Bioinformatics Unit of the Spanish National Cancer Center (CNIO), Madrid, Spain
(<tt>rdiaz&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ligarto.org</tt>) and
Theodore
Garland, Jr, of the Department of Biology of the University of California,
Riverside
(<tt>garland&nbsp;<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ucr.edu</tt>
have released <strong>PHYLOGR</strong>, version 1.0.7.  This is a package of
programs in the R statistical language (and also available as a Windows
executable) to carry out comparative methods analyses, particularly ones
using the Generalized Least Squares method.  The package is distributed
through the
<a href="http://cran.r-project.org/web/packages/PHYLOGR/index.html">web site of
the Cran-R (Comprehensive R Archive Network) project</a>
at <code>http://cran.r-project.org/web/packages/PHYLOGR/index.html</code>
<P>

<A NAME="SLOUCH">
<HR>
<P>
Jason Pienaar, Thomas F. Hansen, and Steven Hecht Orzack</A>
of (respectively) the Department of Molecular Genetics and Microbiology 
of the University of Florida, Gainesville, Florida
<tt>(jasonpienaar<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ufl.edu)</tt>,
the Center for Ecological and Evolutionary Synthesis at the University of
Oslo, Norway
(<tt>t.f.hansen<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;bio.uio.no</tt>) who
is also of the Department of Biological Sciences at Florida State
University, Tallahassee, Florida
(<tt>thomas.hansen<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;bio.fsu.edu</tt>),
and the Fresh Pond Research Institute, 
173 Harvey Street, Cambridge, Massachusetts 02140
(<tt>orzack<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;freshpond.org</tt>) have
written <strong>SLOUCH</strong>
(Stochastic Linear Ornstein-Uhlenbeck models for Comparative Hypotheses),
version 1.0, an R package for comparative methods analysis using an
Ornstein-Uhlenbeck model.  SLOUCH allows the user to estimate 1) the
evolutionary and optimal regressions between a predictor and a response trait,
and 2) phylogenetic inertia. These quantities are estimated jointly by a
comparative method based upon an Ornstein-Uhlenbeck model of adaptive evolution
in which a single trait adapts to an optimum that is influenced by one or more
continuous, randomly changing predictor variables.
The methods are described in the paper:
Hansen, T. F., J. Pienaar, and S. H. Orzack. 2008. A comparative method for
studying adaptation to a randomly evolving environment. <i>Evolution</i>
<b>62:</b> 1965-1977.
It is available as an R package.  It can be downloaded from
<a href="http://freshpond.org/software/SLOUCH/">its web site</a>
at <code>http://freshpond.org/software/SLOUCH/</code>
<P>

<A NAME="phangorn">
<HR>
<P>
Klaus Schliep</A>, then
at Massey University, Palmerston North, New Zealand
<tt>(K.P.Schliep<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;massey.ac.nz)</tt>
has written <strong>phangorn</strong>
version 1.1-2, an R package to do phylogenetic analysis using parsimony,
distance, likelihood and Hadamard methods.  phangorn can carry out parsimony
methods, the neighbor-joining distance method, maximum likelihood, and both
Hadamard and distance Hadamard methods.  It can also compute tree distances.
It is available as an R package.
phangorn requires that the <a href="software.etc2.html#APE">APE</a> and stats
packages also be installed.
It can be downloaded from <a
href="http://cran.r-project.org/web/packages/phangorn/index.html">its web
site</a>
at <code>http://cran.r-project.org/web/packages/phangorn/index.html</code>
<P>

<A NAME="PHYSIG">
<HR>
<P>
Ted Garland</A>
of the Department of Biology
of the University of California at Riverside
<tt>(tgarland<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;citrus.ucr.edu)</tt>
 distributes <strong>PHYSIG </strong>
(PHYlogenetic SIGnal),
a MATLAB package of modules for comparative methods analysis.  PHYSIG is a 
package of modules in MATLAB that includes tests for phylogenetic signal in 
continuous-character data, tests for the attraction to the mean in an 
Ornstein-Uhlenbeck model, and tests of evolutionary covariation between 
characters by a Generalized Least Squares method.  It can be obtained by 
emailing Garland at the above address (it is not available by web download).
The methods are described in the paper:
Blomberg, S. P., T. Garland, Jr., and A. R. Ives. 2003. Testing for
phylogenetic signal in comparative data: behavioral traits are more labile.
<em>Evolution</em> <b>57:</b> 717-745. 
It is available as a MATLAB  package, which is described at
<a href="http://biology.ucr.edu/people/faculty/Garland/PHYSIG.html">its web
site</a>
at <code>http://biology.ucr.edu/people/faculty/Garland/PHYSIG.html</code>
<P>

<A NAME="Cactus-Pie">
<HR>
<P>
<IMG SRC="icons/tinkertree.gif" ALT="[Cactus-Pie icon]">&nbsp;&nbsp;
Dylan Schwilk</A> of the Department of Biology, Texas Tech University,
Lubbock, Texas
<tt>(dylan.schwilk<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ttu.edu)</tt>
has written <strong>Cactus-Pie</strong>
(Python version of CACTUS: Comparative Analysis of Continuous Traits Using
Statistics),
version 0.3.1, a program for comparative analysis of continuous traits.
Cactus-pie is a version of CACTUS written in Python.   It consists of a front
end program and a number of back-end modules.  They include programs that
calculate independent contrasts, provide the Divergence Order Test (DOT) and
the Synchronized Changes Test (SvS), assign branch lengths to a phylogeny
according to several different possible algorithms, and label clades in a
phylogeny as defined by a taxa list in an auxilary file.
It is available as a Python script which can run on computers that have the
Python language, which is widely available on Linux and Unix systems and
is standard on Mac OS X.  Cactus-Pie can be downloaded from
<a href="http://www.pricklysoft.org/software/cactus-pie.html">its web site</a>
at <code>http://www.pricklysoft.org/software/cactus-pie.html</code>&nbsp;
An earlier version which is a Windows executable (using graphical
libraries that may not work on Windows XP or larer) is also
<a href="http://www.pricklysoft.org/software/cactus.html">available there</a>
at
<code>http://www.pricklysoft.org/software/cactus.html</code>
<P>

<A NAME="pcca">
<HR>
<P>
Liam Revell</A>
of the Department of Biology, University of Massachusetts, Boston
<tt>(lrevell<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;nescent.org)</tt>
has released <strong>pcca</strong>
(Phylogenetic Canonical Correlation Analysis),
Beta version, a program for phylogenetic canonical correlation analysis.  pcca 
uses a phylogenetic tree and measurements for an arbitrary number of 
continuous characters to perform a PGLS transformation of the data and then 
calculate canonical scores, weights, and correlations, and conduct hypothesis 
tests about the canonical correlations.  Canonical correlation analysis (CCA) 
is a procedure using two sets of variables and calculating the linear
combinations of each that have highest correlation with each other.
The program can also estimate a multivariate version of Pagel's lambda
transformation of time, and perform the analyses under that transformation.
It is described in the paper:  Revell, L. J. and A. S. Harrison. 2008. PCCA: A
program for phylogenetic canonical correlation analysis. <i>Bioinformatics</i>
<b>24:</b> 1018-1020.
pcca is available as Windows executables, Linux executables and Mac OS X
universal executables.  It can be downloaded from
<a href="http://anolis.oeb.harvard.edu/~liam/programs/">its web site</a>
at <code>http://anolis.oeb.harvard.edu/~liam/programs/</code>
<P>

<HR>
<A NAME="treescan"><P>
<IMG SRC="icons/treescan.jpg" ALT="[Treescan icon]">&nbsp;David Posada
(<TT>dposada<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>uvigo.es</TT>), of the
Universidad Vigo, Spain, and Taylor Maxwell and 
Alan Templeton, of the Department of Biology of Washington University, Saint
Louis, Missouri (<tt>temple_a<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>biology.wustl.edu</tt>)
have written <strong>TreeScan</strong>, version 1.0.  This is a program to
test the association of continuous quantitative characters with a
tree of haplottypes.  It is described in a paper: Templeton, A. R.,
T. Maxwell, D. Posada, J. H. Steng&aring;d, . Boerwinkle, and C. F. Sing. 2005.
Tree scanning: a method for using haplotype trees in phenotype/genotype
association studies. <em>Genetics</em> <b>169:</b> 441-453.  Treescan is
provided in C source code, and also as a DOS and Windows executable or
a Mac OS X executable.  It is available from
<a href="http://darwin.uvigo.es/software/treescan.html">its web site</a>
at <code>http://darwin.uvigo.es/software/treescan.html</code>
<P>

<HR>
<A NAME="ANCML"><P>
Dolph Schluter</A> (<CODE>schluter<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zoology.ubc.ca</CODE>)
of the Department of Zoology and Biodiversity Research Centre
University of British Columbia, in Vancouver, Canada, has released
<STRONG>ANCML</STRONG>, a program which
estimates ancestor states for a continuous trait, and provides a
"standard error" for the marginal distribution of each estimate. The method is
described in
Schluter, D., T. Price, A. &Oslash;. Mooers and D. Ludwig. 1998. Likelihood of
ancestor states in
adaptive radiation. <EM>Evolution</EM> <B>51:</B> 1699-1711. The method assumes
a Brownian motion model
for the evolution of the trait.  ANCML was written by modifying the program
CONTRAST in <A HREF="software.pars.html#PHYLIP">PHYLIP</A> version
3.5, and it uses similar input conventions.
ANCML is available from
<A HREF="http://www.zoology.ubc.ca/~schluter/software.html">its web page</A>
at <CODE>http://www.zoology.ubc.ca/~schluter/software.html</CODE>.  It
is available as generic C source code, as a DOS executable, as a Mac OS X
executable, and as a SunOS executable.
<P>

<HR>
<A NAME="RIND"><P>
Bill Bruno</A> (<TT>billb<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>lanl.gov</TT>) of the
Theoretical Biology and Biophysics Group T10, Los Alamos Scientific
Laboratory, has produced <STRONG>RIND</STRONG>, (Reconstructed INDependence),
a program which takes a tree supplied by the user, or uses a distance
method of the users choosing (one which can be found in <A
HREF="software.pars.html#PHYLIP">PHYLIP</A>), and computes a maximum likelihood
estimate of the
number of times each residue in aligned protein sequences was replaced
in each position.  The method is described in: Bruno, W. J. 1996,
Modeling residue usage in aligned protein sequences via maximum likelihood
<EM>Molecular Biology and Evolution</EM> <B>13:</B> 1368-1374.
RIND is available from
<A HREF="http://www.t10.lanl.gov/billb/rind/">its web site</A>
at <code>http://www.t10.lanl.gov/billb/rind/</code> as C source code
for a Unix environment, and assumes that
<A HREF="software.pars.html#PHYLIP">PHYLIP</A> is also installed.
<P>

<HR>
<A NAME="TreeSAAP"><P>
Steve Woolley</A>, then of the Department of Computer Science, Brigham Young
University, Provo, Utah, and now at the Department of Biology, Washington
University, St. Louis, Missouri
(<tt>stevenwoolley<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp; wustl.edu</tt>),
together with Justin Johnson, Matthew Smith,
Keith Crandall, and David McClellan of the Department of Integrative
Biology, Brigham Young University (whose email address is
<tt>David_McClellan<!deathtospam>&nbsp; (at)<!deathtospam>&nbsp; byu.edu</tt>)
have released <strong>TreeSAAP</strong> version
3.2, (Selection on Amino Acid Properties),
a program that analyzes the frequency of change in various
properties of amino acids in sequences evolving on well-corroborated
phylogenies supplied by the user.  TreeSAAP is given nucelotide sequences
and a user-defined tree, and optimizes the placement of changes
on the tree.  It then calculates measure of departure of these changes from
models of uniform neutral substitution, with respect to different
properties of the amino acids.  The methods are described in a paper:
Woolley, S., J. Johnson, M. J. Smith, K. A. Crandall, and D. A. McClellan.
2003. TreeSAAP: Selection on Amino Acid Properties using phylogenetic trees.
<em>Bioinformatics</em> <b>19:</b> 671-672.   The program is
a Java application.  It is distributed as Windows executables,
Mac OS X executables, and as Java source code from
<a href="http://dna.cs.byu.edu/treesaap/">its web site</a> at
<code>http://dna.cs.byu.edu/treesaap/</code>
<P>

<HR>
<A NAME="Jevtrace"><P>
<IMG SRC="icons/jevtrace.gif">&nbsp; Marcin Joachimiak</A>
(<tt>marcin<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>compbio.berkeley.edu</tt>)
of the Computational Genomics Research Group in the
Department of Plant and Microbial Biology, University of California, Berkeley,
has released <strong>Jevtrace</strong> version 3.12b.  This is a Java
package to analyze the distribution of amino acid changes across a
phylogeny, and use protein structures to identify 
based on their distribution in the tree and in the protein structure.
The program reads a multiple sequence alignment, a tree, and can also
read a PDB protein structure.  It displays the tree on either a branch
length scale or a sequence similarity scale and allows the user to select
clades of interest.  The residues conserved in these clades and differing
between them are then found and can then be viewed on the three-dimensional
structure of the protein.  Jevtrace is described in
<a href="http://genomebiology.com/2002/3/12/research/0077">electronic
publication</a>:
Joachimiak, M. P. and F. E. Cohen.  2003.
JEvTrace: refinement and variations of the evolutionary trace in JAVA.
<em>Genome Biology</em> <b>3 (12)</b>
<code>http://genomebiology.com/2002/3/12/research/0077</code>.  The Jevtrace
<a href="http://compbio.berkeley.edu/people/marcin/jevtrace/">web page</a> is
available at <code>http://compbio.berkeley.edu/people/marcin/jevtrace/</code>.
The software cannot be directly downloaded from there:
the user should fill out the Academic Software License form and submit it.
Distribution is free for academic institutions (prices for others
are not stated).  The user will be emailed a link to download the software.
<P>

<HR>
<A NAME="DIVERGE"><P> Xun Gu, of the Department of Genetics, Development and
Cell Biology
and the Center for Bioinformatics and Biological Statistics at
Iowa State University, Ames, Iowa (<tt>xgu<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>iastate.edu</tt>) and Kent Vander
Velden, then at that
University, have released <strong>DIVERGE</strong> version 2.0.  DIVERGE
reads protein sequences and either infers a tree by Neighbor-Joining or lets
you read in a tree that you supply.  It then allows you to define two
clades in the tree, and tests whether the pattern of rates of change at
different sites differ in these two clades.  The statistical method used,
which is a likelihood ratio test based in a probabilistic model, is given in
the paper:  Gu, X. 1999. Statistical methods for testing functional divergence
after gene duplication. <em>Molecular Biology and Evolution</em> <b>16:</b>
1664-1674.  The program is downloadable as Windows executables from
<a href="http://xungulab.com/software.html">the Gu laboratory software web
site</a>
at <code>http://xungulab.com/software.html</code>.   An earlier version 1.0.4,
which has Windows and Linux executables, is also available there.
<P>

<HR>
<P>
<A NAME="PICA">Mark Wilkinson,</A>
of the Department of Zoology, The Natural History Museum, London, U.K.
(<CODE>marw<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>nhm.ac.uk</CODE>) has
written <STRONG>PICA</STRONG>, version 4.0, a package of programs for character
weighting
and randomization tests for compatibility analysis for
0/1 or multistate characters.  These carry out a variety of tests for
nonrandomly compatible characters and include methods developed by Sharkey,
Le Quesne, Meacham and Alroy.  They include ability of process data that
reflect the splits method of Bandelt and Dress.
The programs are available as a package of DOS executables, from 
<A
HREF="http://www.nhm.ac.uk/research-curation/projects/software/mwphylogeny.html">his
software
web site</A> at
<CODE>http://www.nhm.ac.uk/research-curation/projects/software/mwphylogeny.html
</CODE>.
<P>

<HR>
<P>
<A NAME="COMPROB">Christopher  Meacham</A>, of the Berkeley Phylogenomics
Group, and also affiliated with the University
Herbarium, University of California, Berkeley
has produced <STRONG>COMPROB</STRONG>, a Pascal
program to compute probabilities that characters would be compatible at random,
thus  telling  us  which  clique  is "most surprising".  He can be contacted as
<CODE>meacham<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>berkeley.edu</CODE> about receiving
a copy.  The program is free.
<P>

<HR>
<A NAME="RAPDistance"><P>
John Armstrong, Adrian Gibbs, R. Peakall and George Weiller,</A>
(<CODE>John.Armstrong<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>anu.edu.au</CODE>) of Mark Gibbs's
group
at the School of Botany and Zoology of the Australian
National University, Canberra, have produced <STRONG>RAPDistance</STRONG>
version 1.04, a package for DOS for computing distance matrices for RAPD
analyses.  Version 2.00 was produced but seems not be currently available.
Version 1.04 has slightly more
functionality but cannot handle data sets as large as version 2.00.
RAPDistance has a comprehensive range of options for creating data 
files, editing them and using application programs to analyse them. 
It can export data sets in format of several other packages.
RAPDistance 1.04  is available free
<A
HREF="http://www.rsbs.anu.edu.au/Products&Services/BiotechnologyTransferUnit/rapdistance.php">
at a web page</A> at
<CODE>http://www.rsbs.anu.edu.au/Products&Services/BiotechnologyTransferUnit/rapdistance.php</code>
<P>

<HR>
<A NAME="MULTICOMP"><P>
Peter Reeves, L. Farnell, and Ruiting Lan</a>, then of the School of
Molecular and Microbial Biosciences at Sydney University, Australia, have
produced <STRONG>MULTICOMP</STRONG>, a program for computing various distances
from sequence
data.  It can also do sequence format conversions, compute various descriptive
statistics on the sequences, and can submit the sequences to two programs
from <A HREF="software.pars.html#PHYLIP">PHYLIP</A>.  It is described in a
paper: Reeves, P. R., L. Farnell and R. Lan.
1994. MULTICOMP: a program for preparing sequence data for phylogenetic
analysis. <em>Computer Applications in the Biosciences (CABIOS)</em> <b>10:</b>
281-284.
It is available as SunOS executables and as Linux executables
<a href="http://www.emi.unsw.edu.au/~lanlab/public/multicomp/download.htm">at a
web site</a> 
at <code>
ttp://www.emi.unsw.edu.au/~lanlab/public/multicomp/download.htm</code>
in the lab of Ruiting Lan
at the University of New South Wales, Australia.  Lan may be emailed to
obtain the C source code, at
(<tt>rlan<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;unsw.edu.au</tt>).
<P>

<HR>
<A NAME="NimbleTree">
<P>
<IMG SRC="icons/NimbleTree.jpg"></A>&nbsp;&nbsp;Stuart Ray of the Division of
Infectious Diseases at the Johns Hopkins University School of Medicine,
Baltimore, Maryland
<tt>(sray<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;jhmi.edu)</tt> have
produced <strong>NimbleTree</strong>
version 2.6, a program that submits data sets to PAUP* or PHYLIP.
NimbleTree reads a variety of different sequence alignment formats, and allows
you to more easily submit the resulting data sets to <a
href="software.pars.html#PHYLIP">PHYLIP</a> or to <a
href="software.pars.html#PAUP">PAUP*</a>.  For PAUP* you need to have that
program already installed on your computer.  Some PHYLIP source code from its
version 3.5p is included in NimbleTree (with my agreement). (I am not quite
sure which methods from PHYLIP are available in this program).
It is available as Windows executables.
It can be downloaded from <code><a
href="http://sray.med.som.jhmi.edu/SCRoftware/nimbletree/">http://sray.med.som.jhmi.edu/SCRoftware/nimbletree/</a></code>
<P>

<HR>
<A NAME="Microsat"><P>
<STRONG>Microsat</STRONG> versiobn 1.5, </A> by Eric Minch, then of
the Department of Human Genetics, Stanford University, Stanford, California
(his email address is now <CODE>eric.minch<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>lionbioscience.com</CODE>)
is a program for calculating distances from
microsatellite data.   It uses the methods developed by David Goldstein et.
al.,
and presented in their papers of 1995 in <EM> Proceedings of the Natonal
Academy
of Sciences USA </EM> 
<B>92:</B> 6720-6727 and <EM> Genetics </EM> <B>139:</B> 463-471.  The distance
is based on
the mean microsatellite array size, implementing the "&Delta;&mu;" distance
that they
defined, which corrects for within-population variability and provides a
distance that is independent of population size.  It also calculates a
number of other microsatellite-based genetic distances.  It is available for
free
from <A HREF="http://hpgl.stanford.edu/projects/microsat/">a page in
Luca Cavalli-Sforza's lab web site</A> at
<CODE>http://hpgl.stanford.edu/projects/microsat/</CODE>.
The program is written in ANSI C. Source code is available there, and
so are executables for DOS and Mac OS.
<P>

<HR>
<A NAME="MSA"><P>
Daniel Dieringer (<tt>daniel.dieringer<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>boku.ac.at</tt>)
and Christian Schl&ouml;tterer of the
University of Vienna have produced <strong>MSA</strong> (MicroSatellite
Analyzer) version 4.05, a program for handling large microsatellite data
sets.  It can calculate many descriptive statistics for these data sets,
can convert the data into a variety of file formats for other programs,
and can also calculate a variety of distance measures.
It is described in a paper: Dieringer, D., and C. Schl&ouml;tterer.  2003.
Microsatellite analyser (MSA): a platform independent analysis tool for large
microsatellite data sets. <em>Molecular Ecology Notes</em> <b>3:</b> 167-169.
MSA is available as source code in C plus executables for Linux,
Windows (and DOS), Mac OS, and Mac OS X from
<A HREF="http://i122server.vu-wien.ac.at/MSA/MSA_download.html">its web
site</A>
at <code>http://i122server.vu-wien.ac.at/MSA/MSA_download.html</code>.
<P>

<HR>
<A NAME="DIPLOMO"> <P>
Georg Weiller</A>, of the Genomic Interactions Group at the Research School of
Biological Sciences, Australian National University,
Canberra, Australia (<CODE>georg.weiller<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>anu.edu.au</CODE>) has produced
<STRONG>DIPLOMO</STRONG> (DIstance PLOt MOnitor) version 1.03.  It compares
different distance measures with each other 
by displaying them as a scatter plot. It then helps one 
instantly identify all individual comparisons within the plot. individual
taxa can be excluded or included in the plots,  DIPLOMO enables you to see
whether different taxa have different mutational characteristics (such as
more having relatively more transitions in some lineages), and whether
different distance measures correlate.  The program takes as input a file
with several different distance matrices.  This file is in a simple format
which can readily be produced by editing distance matrices produced by other
packages.  It is described in
a publication: Weiller, G. F. and A. Gibbs. 1995. DIPLOMO: The tool for a
new type of evolutionary analysis. <EM>CABIOS</EM> <B>11:</B> 535-40.
It runs on DOS or in a Command Tool on Windows. It can be obtained from
<A
HREF="http://www.rsbs.anu.edu.au/Products&Services/BiotechnologyTransferUnit/diplomo.php">its
web site</a>
at
<CODE>http://www.rsbs.anu.edu.au/Products&Services/BiotechnologyTransferUnit/diplomo.php</CODE>.
<P>

<HR>
<A NAME="PTP"><P><IMG SRC="icons/PTP.gif">&nbsp; Oclair Prado
(<tt>oclair<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>cpqd.com.br</tt>) and Fernando Van
Zuben</A> (<tt>vonzuben<!deathtospam>&nbsp; (at)<!deathtospam>&nbsp;
dca.fee.unicamp.br</tt>) of the Department of Computer Science of the
University of Campinas,
Brazil released the <strong>Phylogenetic Tree Project (PTP)</strong> genetic
algorithms toolbox, version 1.0.  It infers phylogenies by maximum likelihood
or a distance matrix method,
using an evolutionary computation strategy which represents a phylogeny by
a genotype, simulates natural selection to choose among phylogenies and
recombination to rearrange phylogenies.  It is a Windows executable,
which can be downloaded <a
href="ftp://ftp.dca.fee.unicamp.br/pub/docs/vonzuben/oclair/PTP.zip">by ftp</a>
from 
file <code>PTP.zip</code> at <code>ftp.dca.fee.unicamp.br</code> in folder
<code>/pub/docs/vonzuben/oclair</code>.
<P>

<HR>
<A NAME="MetaPIGA"><P>
<IMG SRC="icons/metapiga.jpg" ALT="MetaPIGA icon">&nbsp;&nbsp;
Alan R. Lemmon</a>, of the Department of Scientific Computing and the
Department of Biological Sciences at Florida State University, Tallahassee,
Florida (<tt>alemmon<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>evotutor.org</tt>)
and Michel Milinkovich (<tt>mcmilink<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ulb.ac.be</tt>) of the
Unit of Evolutionary Genetics,  Institute of Molecular Biology and Medicine,
Free University of Brussels, Belgium
have written <strong>MetaPIGA 2</strong> (Phylogeny Inference using the MetaGA),
a program searching for maximum likelihood phylogenies using a genetic
algorithm with metapopulations.  This allows different populations in the
genetic algorithm to arrive at different solutions, which can then be
combined by migration and recombination.  It is said to make possibe
effective maximum likelihood inference of phylogenies for data sets with
hundreds of species.   It allows rate variation among sites, different
substitution models, partitioning of data, and LRT, AIC amd BIC tests.
MetaPIGA is described in two papers:
<UL>
<LI> Lemmon, A. R. and M. C.  Milinkovitch. 2001.  The metapopulation genetic
algorithm: An efficient solution for the problem of large phylogeny estimation.
<em>Proceedings of the National Academy of Sciences, USA</em> <b>99:</b>
10516-10521.
<LI> Helaers, R., and M. C. Milinkovich.  2010.  MetaPIGA v2.0: maximum
likelihood large phylogeny estimation using the metapopulation genetic
algorithm and other stochastic heuristics.  <I>BMC Bioinformatics</I>
<B>11:</B> 379.
</UL>
It is available as Java code from
<A HREF="http://www.metapiga.org">its web site</A> at
<code>http://www.metapiga.org</code>.  The earlier version, MetaPIGA 1, is also
available there.
<P>

<A NAME="GARLI">
<HR>
<P>
<IMG SRC="icons/garli.jpg">&nbsp;&nbsp;Derrick Zwickl</A>
of the Department of Ecology and Evolution
of the University of Kansas, Lawrence, Kansas
<tt>(zwickl<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ku.edu)</tt>
 released <strong>GARLI</strong>
(Genetic Algorithm for Rapid Likelihood Inference),
version 1.0, a program using a genetic algorithm to search for maximum
likelihood phylogenies.  GARLI uses a genetic algorithm to perform heuristic
phylogenetic searches under the General Time Reversible (GTR) model of
nucleotide substitution and its submodels, with or without gamma distributed
rate heterogeneity and a proportion of invariant sites.  It can also
analyze protein sequences using amino acid models and codon-based
models.  It can read NEXUS and PHYLIP format
sequence files.  An MPI algorithm is included for use on parallel computing
clusters (the parallel version seeks to perform a more thorough tree search and
does <em>not</em> reduce runtimes.
The program was written when Zwickl was a graduate student at the
Department of Zoology of the University of Texas at Austin.
It is described in his thesis:
Zwickl, D. J. 2006. Genetic algorithm approaches for the phylogenetic analysis
of large biological sequence datasets under the maximum likelihood criterion.
<em>Ph.D. dissertation, The University of Texas at Austin</em>.
It is available as C++ source code, Windows executables and Mac OS X universal
executables.  It can be downloaded from
<a href="http://code.google.com/p/garli/">its Google Code web page</a>
at <code>http://code.google.com/p/garli/</code>.
An older version, called GARLI-PART version 0.97, which can handle
partitioned models, is also available at
<a href="https://www.nescent.org/wg_garli/Partition_testing_version">its Wiki site</a> at the National Evolutionary Synthesis Center at
<code>https://www.nescent.org/wg_garli/Partition_testing_version</code>
An even older version of GARLI, 0.951, is available
<a href="http://www.bio.utexas.edu/faculty/antisense/garli/Garli.html">here</a>
at <code>http://www.bio.utexas.edu/faculty/antisense/garli/Garli.html</code>.
<P>

<HR>
<A NAME="MUST"><P>
<STRONG> MUST</STRONG></A>, version 5.15, a package of sequence management 
programs, was developed by Herv&eacute; Philippe (<tt>herve.philippe (at)
umontreal.ca</tt>), of the Departement de Biochimie
Universit&eacute; de Montr&eacute;al, Qu&eacute;bec.  It 
intended  as complementary to existing phylogeny and alignment programs and can
produce output files in the formats of <A
HREF="software.pars.html#PHYLIP">PHYLIP</A>, <A
HREF="software.pars.html#PAUP">PAUP*</A>, <A
HREF="software.pars.html#Hennig86">Hennig86</A>, and <A
HREF="#ClustalW">CLUSTAL</A>.  It
contains a variety of sequence input, editing, checking, and storage functions,
as well as a sequence editor and a phylogeny plotter.  It also  allows  further
analyses of the results from these phylogeny programs.
The original version of MUST is described in a paper: Philippe, H. 1993.
MUST, a computer package of management utilities for sequences and trees.
<em>Nucleic Acids Research</em> <b>21:</b> 5264-5272.
MUST is available as a Linux/Unix program (including source code that
can be compiled on Mac OS X as well) or an older and not
recently-maintained DOS program (which can also be run in a command
tool window on Windows) from
<a href="http://megasun.bch.umontreal.ca/Software/HPLab/must/must.html">its web page</a> at
<code>http://megasun.bch.umontreal.ca/Software/HPLab/must/must.html</code>
<P>

<HR>
<A NAME="GDE"><P>
Steve Smith</A>, formerly of the Harvard Genome Laboratory, wrote
an  X-Windows interactive sequence editor, <STRONG>GDE</STRONG> (Genetic Data
Environment), version 2.2, which
allows the user to edit sequences and align them by hand, and to select subsets
of  sites  and  sequences  and  call  a  variety of analysis proprams including
<A HREF="#ClustalW">ClustalV</A> and many of the <A
HREF="software.pars.html#PHYLIP">PHYLIP</A> 3.5 programs.  The GDE 2.2 system
will run on
Unix or Linux systems using the X windowing system.  It also includes the
<A HREF="#TreeTool">TreeTool</A> tree-plotting program (see below). GDE has
been described in two
papers:
<UL>
<LI>Smith, S. W., R. Overbeek, C. R. Woese, W. Gilbert, and P. M. Gillevet.
1994.  The genetic data environment an expandable GUI for multiple sequence
analysis.  <em>Computer Applications in the Biosciences</em> <b>10:</b>
671-675. 
<LI>De Oliveira, T., R. Miller, M. Tarin, and S. Cassol. 2003.
An integrated genetic data environment (GDE)-based LINUX interface for analysis
of HIV-1 and other microbial sequences. <em>Bioinformatics</em> <b>19:</b>
153-154.
</UL>
GDE is free and is available as source code in C and as executables at these
places:
<UL>
<LI> A MacOS X version that runs under X windows, MacGDE 2.4 is available at <A HREF="http://macgde.bio.cmich.edu">a web page</A> at
Central Michigan University at <code>http://macgde.bio.cmich.edu</code>,
ported by Tim Littlejohn
(<tt>tim<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>biolateralgroup.com</tt>) of
BioLateral Group
in Sydney, Australia.
and maintained by Eric Linton of the Biology Department at
Central Michigan University, Mt. Pleasant, Michigan 
(<tt>eric.linton<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>cmich.edu</tt>)
<LI>binaries for SunOS, Sun Solaris, and Linux are available at
<A HREF="http://iubio.bio.indiana.edu/soft/molbio/unix/GDE/">a web page</A>
at the IUBIO molecular biology software server
<CODE>http://iubio.bio.indiana.edu/soft/molbio/unix/GDE/</CODE>
<LI> An earlier version of GDE, 1.0, is available at the <A
HREF="ftp://ftp.ebi.ac.uk/pub/software/unix/">EBI Software Archive</A> at
<code>ftp://ftp.ebi.ac.uk/pub/software/unix/</code>
</UL>
GDE is the precursor of the SeqLab interface used with the (now unavailable) <A
HREF="software.old.html#GCG">GCG Wisconsin Package</A> of sequence search and
analysis programs.
<P>

<A NAME="BIRCH">
<HR>
<P>
Brian Fristensky</A>
of the Department of Plant Science
of the University of Manitoba, Winnipeg, Manitoba, Canada
<tt>(frist<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;cc.umanitoba.ca)</tt>
distributes <strong>BIRCH</strong>
(BIological Research Computer Hierarchy),
version 2.1, a sequence management and submission system with molecular
databases.
BIRCH consists of scripts and programs for a wide variety of analyses.  It uses
the <a href="software.etc2.html#GDE">GDE</a> system for sequence editing and
submission to programs.  Some of the phylogeny programs (about 13 programs
or packages for phylogenies) include many from
<a href="software.pars.html#PHYLIP">PHYLIP</a>,
<a href="software.etc1.html#fastDNAml">fastDNAml</a>,
<a href="software.etc1.html#MrBayes">MrBayes</a>,
<a href="software.etc1.html#Phylo_win">Phylo_win</a>, 
<a href="software.etc2.html#FORESTER">ATV</a>,
and <a href="software.etc1.html#ClustalW">ClustalW</a>.  BIRCH when installed
also includes copies of molecular databases; the user is instructed on how
to download these from their distribution sites.
BIRCH is written in C and in Python and Java.  It is distributed in source code
and can be compiled on
Linux and Solaris.  The programs are distributed as binaries.  The
<a href="http://home.cc.umanitoba.ca/~psgendb/">main BIRCH web site</a> is at
<code>http://home.cc.umanitoba.ca/~psgendb/</code>.
It is described in
<UL>
<LI> Fristensky, B. 1999. Building a multiuser sequence analysis
facility using freeware. pp 131-145 in S. Misener and S. A. Krawetz.
<em>Methods in Molecular Biology, vol. 132: Bioinformatics Methods and
Protocols</em>. Humana Press, Totowa, NJ, USA. 
<LI> Fristensky, B. 2007.  BIRCH: A user-oriented, locally customizable
bioinformatics system.  <I>BMC Bioinformatice</i> <b>8:</b> 54.
</UL>
BIRCH can be downloaded from
<ahref="http://home.cc.umanitoba.ca/%7Epsgendb/downloads.html">its download web site</a> at
<code>http://home.cc.umanitoba.ca/%7Epsgendb/downloads.html</CODE>
<P>

<HR>
<A NAME="PHYDIT">
<P>
Jongsik Chun</A> of the School of Biological Sciences,
Seoul National University, Korea (<TT>jchun<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>snu.ac.kr</TT>) has released
<B>PHYDIT</B> version 3.1.  PHYDIT is a sequence alignment environment that
can align either nucleotide or protein sequences manually or using pairwise
methods, and contains some <A HREF="software.pars.html#PHYLIP">PHYLIP</A>
programs to do either neighbor-joining or least-squares phylogenies on the
alignments.  PHYDIT is available as a Windows executable.  It is
available free for noncommercial use.  The user needs to register to
download.  The <A HREF="http://plaza.snu.ac.kr/~jchun/phydit/">
PHYDIT web page</A> is <CODE>http://plaza.snu.ac.kr/~jchun/phydit/</CODE>.
<P>

<HR>
<A NAME="SeqPup"><P>
<IMG SRC="icons/seqpup.gif" ALT="[SeqPup icon]">
<IMG SRC="icons/seqpup-java.jpg" ALT="[SeqPup Java icon]">
Don Gilbert</A> (<CODE>gilbertd<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>indiana.edu</CODE>) of the
Department of Biology of the University of Indiana, has written
<STRONG>SeqPup</STRONG> versions 0.9 and 0.6, a biological sequence editor and
analysis program
usable on MacOS, Windows and Unix systems.  It allows alignment of
sequences by hand and submission of selected parts of selected sequences to
phylogeny programs, as well as to network services such as BLAST.  It is
available in a more
complete earlier version (0.7) written in C++, or a later (0.9) version
written in Java.  The latter will work on all systems that have the
Java 1.1 runtime environment.  It will not work on the Java 1.2 runtime
environment, so the earlier Java 1.1 environment needs to be installed as
well if the later version of Java is present. (I do not know whether
SeqPup works with later versions of Java).
The two versions of SeqPup are described at
<A HREF="http://iubio.bio.indiana.edu/soft/molbio/seqpup/java/seqpup-doc.html">a web page</A>
at
<CODE>http://iubio.bio.indiana.edu/soft/molbio/seqpup/java/seqpup-doc.html</CODE>.
Versions 0.9 and 0.6 can be obtained at
<A HREF="http://iubio.bio.indiana.edu/soft/molbio/seqpup">its download web
page</A>
The earlier version (0.6) is available in C++, and
executables of it are available for MacOS (the PowerMac and 68K platforms), 
Windows, and Unix X windows systems
including Sun Solaris, SGI Irix, Dec (HP/Compaq) Unix, Linux.
The C++ source code 
for the 0.6 version
is available <A HREF="ftp://iubio.bio.indiana.edu/util/dclap/source/">by anonymous
ftp</A> at: <CODE>iubio.bio.indiana.edu</CODE> in
directory <CODE>util/dclap/source/</CODE>.
<P>

<A NAME="CBCAnalyzer">
<HR>

<P>
<IMG SRC="icons/cbcanalyzer.gif">&nbsp;&nbsp;Matthias Wolf, Joachim Friedrich,
Thomas Dandekar, and Tobias M&uuml;ller</A>
of the Department of Bioinformatics
at the the Biocenter, University of W&uuml;rzburg, Germany
<tt>(Matthias.Wolf<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;biozentrum.uni-wuerzburg.de)</tt>
have written <strong>CBCAnalyzer</strong>
versions 1.0.3 and 1.1&beta;, programs for inferring phylogenies based on 
compensatory base changes (CBCs).
The CBCAnalyzer  (CBC = compensatory base change) is a custom written software
toolbox consisting of three parts, CTTransform, CBCDetect, and CBCTree.
CTTransform reads several CT-file formats (ct, RNAviz ct or Mac ct), and
generates a so called "bracket-dot-bracket" format that specifies which sites
are paired in an RNA structure.  This typically is used as input for other
tools such as RNAforester, RNAmovie or MARNA. The latter one creates a multiple
alignment based on primary sequences and secondary structures that now can be
used as input for CBCDetect. The count (distance) matrix of compensating
changes obtained by CBCDetect is used as input for CBCTree that reconstructs a
phylogram by using the BIONJ algorithm.
It is described in the paper:
Wolf, M., J. Friedrich, T. Dandekar and T. M&uuml;ller.  2005.  CBCAnalyzer:
inferring phylogenies based on compensatory base changes in RNA secondary
structures. <em>In Silico Biology</em> <b>5:</b> 0027.
It is available as C++ source code which can be compiled on Linux, and as 
Windows executables.  It can be downloaded from
<a href="http://cbcanalyzer.bioapps.biozentrum.uni-wuerzburg.de/">its web
site</a>
at <code>http://cbcanalyzer.bioapps.biozentrum.uni-wuerzburg.de/</code>
<P>

<HR>
<A NAME="ARB"><P>
Wolfgang Ludwig and Oliver Strunk</A> of the Lehrstuhl f&uuml;r
Mikrobiologie of the Technische Universit&auml;t M&uuml;nchen
(<CODE>wolfgang.ludwig<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>arb-home.de</CODE>)
distribute
<STRONG>ARB</STRONG>, an environment for 16s/18s/23s ribosomal RNA sequence
data.  It provides a windowing environment for building up databases of
RNA sequences, aligning them, and searching, editing, modifying, aligning,
profiling, and constructing trees.  ARB uses its own RNA sequence databases
which are made available to ARB over the Web.  For phylogenies it uses
programs from
<A HREF="software.pars.html#PHYLIP">PHYLIP</A>,
<A HREF="software.etc1.html#RAXML">AXML</A>,
<A HREF="software.etc1.html#PHYML">PHYML</A> and
<A HREF="software.etc1.html#fastDNAml">fastDNAml</A>, as well as its own
ARB Neighbor-Joining program.  ARB is also incorporates a variety of other
sequence analysis software.  It can handle large numbers of sequences and has
sophisticated tree drawing and manipulation.  ARB is distributed as source
code and executables for Linux systems, specifically Ubuntu and SuSE Linux.  
It can also be compiled successfully on SunOS and Mac OS X.
ARB is available from
<A HREF="http://www.arb-home.de/">its web site</A>
at <CODE>http://www.arb-home.de/</CODE>.
A Mac OS X port of ARB is available <A
HREF="http://www.haloarchaea.com/resources/arb/index.html">by Mike
Dyall-Smith</A> of the Max Planck Institute of Biochemistry at
<CODE>http://www.haloarchaea.com/resources/arb/index.html</CODE>.
<P>

<A NAME="DART">
<HR>
<P>
Ian Holmes</A>
of the Department of Bioengineering
of the University of California at Berkeley, Berkeley, Calfornia
<tt>(ihh<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;berkeley.edu)</tt>
has released <strong>DART</strong>
(Dna, Amino and Rna Tests),
a package of programs to do a variety of genomic, alignment, and RNA structure 
inferences.  (DART is not to be confused with NCBI's Domain Architecture
Retrieval Tool, also called DART).  Various programs in the DART package do phylogenetic alignment; 
RNA structure prediction and multiple RNA alignment, stochastic context-free
grammars, inference of evolutionary models, phylo-HMMs and phylo grammars, 
reconstruction of ancestral sequences; and phylogenomics. It contains a number 
of programs including
<ul>
<li> stemloc and EvolDeeds for RNA alignment and ancestral prediction
<li> xrate for phylo-grammars
<li> phylocomposer and phylodirector for analysis and visualization
<li> other statistical alignment programs in the Handel package
<li> simgenome for simulation of genome sequence alignments
</ul>
The programs use statistical algorithms (MCMC, EM) to impute multiple 
alignments, annotations and other unseen evolutionary parameters from sequence 
data. All are based on stochastic grammars or state-machine models of sequence 
mutation and natural selection. Many other people (listed at the DART web 
site) have contributed to the package.
(The package includes &ldquo;EvolDeeds&rdquo;, which I have awarded the John Buettner-Janusch
Award for most sinister name of a program).
A large number of papers on the individual methods in the package
are listed at the DART web site.
It is available as C source code.  It can be downloaded from
<a href="http://biowiki.org/DART">its web site</a>
at <code>http://biowiki.org/DART</code>
<P>

<A NAME="rRNA">
<HR>
<P>
Andrew Smith, Thomas W. H. Lui and Elisabeth Tillier</A>
of the Ontario Cancer Institute and the Department of Medical Biophysics
at the University of Toronto, Canada
<tt>(e.tillier<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;utoronto.ca)</tt>
have released <strong>rRNA phylogeny</strong>, a program package to infer 
phylogenies from ribosomal RNA using a model of substitution that allows for 
compensating substiutions at paired sites.  The program makes use of a model 
(the OTRNA model) of ribosomal RNA substitution that has different rates for 
paired and unpaired sites, that reflect the lower probability of a compensated 
substitution that maintains the pairing.  The model is empirically tabulated 
from rRNA sequences and used in a modified version of programs from
<a href="software.pars.html#PHYLIP">PHYLIP</a> to infer phylogenies. The 
package also allows distances to be computed from the OTRNA model for use in 
distance matrix programs.
The methods are described in the paper:
Smith, A., T. W. H. Lui and E. R. M. Tillier, 2004. Empirical substitution 
models for Ribosomal RNA <em>Molecular Biology and Evolution</em> <b>21:</b> 
419-427.
It is available as C source code, Windows executables and Linux executables.  
It can be downloaded from
<a href="http://www.uhnresearch.ca/labs/tillier/software.htm">its web site</a>
at <code>http://www.uhnresearch.ca/labs/tillier/software.htm</code>
<P>

<A NAME="AMBIORE">
<HR>
<P>
Dick Hwang and Phil Green</A>
of the Department of Genome Sciences
of the University of Washington, Seattle, Washington
<tt>(dhwang<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;u.washington.edu)</tt>
has released <strong>AMBIORE</strong>
(Applications for Mcmc Bayesian Inference Of Rates in Evolution),
version 1.00, which estimates rates of change in nucleotide sequences 
depending on the two neighboring bases.  It implements a flexible and 
computationally efficient Bayesian Markov chain Monte Carlo approach to 
estimating rates in evolution given a sequence alignment and tree topology 
relating the species. The evolutionary model allows substitution rates at a 
site to depend on the two flanking nucleotides, the branch of the phylogenetic 
tree, and position within a sequence.
The methods are described in the paper:
Hwang D. G. and P. Green.  2004. Bayesian Markov chain Monte Carlo sequence 
analysis reveals varying neutral substitution patterns in mammalian evolution. 
<em>Proceedings of the National Academy of Sciences U.S.A.</em> <b>101:</b> 
13994-14001.
It is available as C source code.  It can be downloaded from
<a href="http://www.phrap.org/othersoftware.html">its web site</a>
at <code>http://www.phrap.org/othersoftware.html</code>
<P>

<A NAME="CodeAxe">
<HR>
<P>
Chris Saunders</A>,
then of the Department of Genome Sciences
of the University of Washington, Seattle, Washington (and currently
at Illumina, Inc.)
has written <strong>CodeAxe</strong>
version 1.0, a tool for the phylogenetic analysis of mutation and selection in 
coding sequences.  It provides, for a phylogeny provided by the user, analysis 
of approximate maximum likelihood nucleotide, codon and "extended codon" 
models that allow for neighbor-dependent nucleotide mutation, asymmetric 
exchange-dependent amino acid selection, nonreversible evolution and mixture 
models of mutation and selection classes.
It is described in the paper:
Saunders, C. T. and P. Green. 2007. Insights from modeling protein evolution 
with context-dependent mutation and asymmetric amino acid selection. 
<em>Molecular Biology and Evolution</em> <b>24(12):</b> 2632-2647.
It is available as C++ source code.  It can be downloaded from
<a href="http://www.phrap.org/othersoftware.html">its web site</a>
at <code>http://www.phrap.org/othersoftware.html</code>
<P>

<HR>
<A NAME="BioEdit"><P>
<IMG SRC="icons/bioedit.gif" ALT="[BioEdit icon]">
Tom Hall, then of the Department of Microbiology at North Carolina State
University, and now of Ibis Therapeutics, Carlsbad, Calfornia</A>
(<CODE>thall<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>isisph.com</CODE>) has produced
<STRONG>BioEdit</STRONG>, version 7.0.5.  This is a sequence editor with
many kinds of general molecular biology functions available (alignment,
BLAST searches, plasmid drawing, restriction mapping, sequence machine trace
viewing, etc.).  For our purposes the feature worth mentioning is that
it comes with a number of existing phylogeny programs which can be
automatically run from within BioEdit.  These are: <A HREF="#TreeView">TreeView</A>,
<A HREF="software.etc1.html#fastDNAml">fastDNAml</A>, and six DNA and protein
programs from <A HREF="software.pars.html#PHYLIP">PHYLIP</A>.  It has been
described in the paper:  Hall, T. A. 1999.  BioEdit: a user-friendly biological
sequence alignment editor and analysis program for Windows 95/98/NT. <i>Nucleic
Acids Sympoisum Series</i> <b>41:</b> 95-98.
BioEdit is available as Windows executables. Version 7.0.5 is available
from <A HREF="http://www.mbio.ncsu.edu/BioEdit/bioedit.html">its web site</A> at
<CODE>http://www.mbio.ncsu.edu/BioEdit/bioedit.html</CODE>.
<P>

<HR>
<A NAME="GeneStudioPro"><P>
<IMG SRC="icons/gspro.gif">&nbsp; GeneStudio, Inc.</A> of Suwanee, Georgia
(<tt>info<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>genestudio.com</tt>)
has released <strong>GeneStudio Pro</strong>, version 2.1.2.4, a commercial package for
sequence analysis and sequence format conversion.  (Incidentally, although the
Suwanee River is partly located in Georgia, the town of Suwanee is <em>not</em>
located "way down upon the Suwanee River" but is quite far from it).  Included
is an Alignment Editor that can invoke a variety of phylogeny programs,
including <A HREF="software.etc1.html#fastDNAml">fastDNAml</A>,
some programs from <A HREF="software.pars.html#phylip.html">PHYLIP</A>,
<A HREF="software.etc1.html#TREE-PUZZLE">Tree-puzzle</A>
and <A HREF="software.etc2.html#TreeView">TreeView</A>.  It can also
do many other functions such as sequence format conversion, BLAST searches,
and contig editing.  GeneStudio Pro is for the Windows platform.  A free
trial version is available.  Prices are not given on their web site but are
available on request from a page on their web site, accessed through the trial
version.  For further information 
see the
<A HREF="http://www.genestudio.com/genestudio.htm">GeneStudio Pro web site</A>
at <CODE>http://www.genestudio.com/genestudio.htm</CODE>.
<P>

<HR>
<a name="simplot"><P><img src="icons/simplot.jpg" alt="[Simplot
icon"></a>&nbsp; Stuart Ray, of the Division of Infectious
Diseases of the Department of Medicine at the Johns Hopkins University
School of Medicine, Baltimore, Maryland
(<tt>sray&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>jhmi.edu</tt>) has
produced <strong>Simplot</strong>, version 3.5.1.  It is a Windows program that
serves as a front end to either code
from <a href="software.pars.html#PHYLIP">PHYLIP</a> or to <a
href="software.pars.html#PAUP">PAUP*</a>
and enables you to easily submit jobs to them.
SIMPLOT enables you to select
regions and do other forms of data selection.  It can also carry out the
"bootscanning" method of detecting inconsistencies in trees in different
regions of a sequence, which can be a signal for recombination.
It was first described in the paper:  Lole, K. S., R. C. Bollinger, R. S.
Paranjape, D. Gadkari, S. S. Kulkarni, M. G. Novak, R. Ingersoll, H. W.
Sheppard, and S. C. Ray. 1999. Full-length
human immunodeficiency virus type 1 genomes from subtype C-infected
seroconverters in India, with evidence of intersubtype recombination.
<i>Journal of Virology</i> <b>73(1):</b> 152-160.
It is distributed as a Windows executable through
<a href="http://sray.med.som.jhmi.edu/SCRoftware/simplot/">its web site</a>
at <code>http://sray.med.som.jhmi.edu/SCRoftware/simplot/</code>
<P>

<A NAME="MBEToolbox">
<HR>
<P>
James J. Cai</A>
of the Department of the Department of Veterinary Integrative
Biosciences at the Texas A&amp;M University, College Station, Texas
<tt>(jcai<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;tamu.edu)</tt>
has written <strong>MBEToolbox</strong>
(Molecular Biology and Evolution Toolbox),
version 2.21, A MATLAB package to enable evolutionary biologists to analyze 
and view DNA and protein sequences.  MBEToolbox includes sequence manipulation 
and statistics, evolutionary distance calculations, tree creation, a novel 
window analysis method and a graphical user interface. version 2.0 added new 
functions for phylogenetic analyses by the maximum likelihood method, analysis 
of site-specific evolutionary rates, and algorithms to detect recombination.
It is described in the papers:
<ul>
<li>Cai, J. J., D. K. Smith, X. Xia, and K.-Y. Yuen. 2005. MBEToolbox: a 
Matlab toolbox for sequence data analysis in molecular biology and evolution. 
<em>BMC Bioinformatics</em> <b>6:</b> 64 (22Mar2005)
<li> Cai, J. J., D. K Smith, X. Xia, and K.-Y. Yuen. 2006. MBEToolbox 2.0: An 
enhanced version of a MATLAB toolbox for Molecular Biology and Evolution. 
<em>Evolutionary Bioinformatics Online</em> <b>2:</b> 189-192.
</ul>
It is available as a MATLAB  package, and has versions for download to
Windows, Linux, and Sun Solaris systems.  It requires MATLAB 6.5 or higher; the
parsimony and likelihood analyses also require programs from
<a href="software.pars.html#PHYLIP">PHYLIP</a>.  MBEToolbox can be downloaded
from <a href="http://bioinformatics.org/mbetoolbox/">its web site</a>
at <code>http://bioinformatics.org/mbetoolbox/</code>
<P>

<A NAME="Bosque">
<HR>
<P>
<img src="icons/bosque.jpg" ALT="[Bosque icon]">&nbsp;&nbsp;Salvador Ramirez</A>
of the Departamento de Oceanografia
of the Universidad de Concepcion, Chile
<tt>(sram<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;profc.udec.cl)</tt>
has released <strong>Bosque</strong> Phylogenetic Analysis Software,
version 1.7.42, an integrated set of phylogenetic analysis software into a 
complete graphical user interface.  Bosque is a graphical program that 
integrates console programs from the
<a href="software.pars.html#PHYLIP">PHYLIP</a> package, from
<a href="software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a>, and from
<a href="software.etc1.html#MUSCLE">MUSCLE</a>. 
As a graphical program it also includes sequence, alignment and tree editors 
to ease the manipulation of the data, the execution of the programs and the 
administration of the output result from the phylogenetic programs. It also 
has the option to function as a client-server program where a Bosque server is 
installed, thus allowing the remote execution of the mentioned phylogenetic 
programs.
It is available as Windows executables, Mac OS X executables,
and Linux executables.  It can be downloaded from
<a href="http://bosque.udec.cl">its web site</a>
at <code>http://bosque.udec.cl</code>
<P>

<A NAME="Segminator">
<HR>
<P>
John Archer and David Robertson</A>
of the Faculty of Life Sciences
of the University of Manchester
<tt>(john.archer.jpa<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;googlemail.com)</tt>
and Andrew Rambaut of the Institute of Evolutionary Biology of the University
of Edinburgh
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
have produced <strong>Segminator II</strong>,
a tool for extracting data from reads from viral 
data generated using the 454 Life Science sequencing platform.
Following the mapping and pairwise alignment of reads, data that is immediately
accessible in tabulated format includes:
<ol><li> Obtaining nucleotide and amino acid residue frequencies at sites 
across the template.
<li> Generating a consensus across data set using overlapping windows (while allowing for indels differing from those in relation to the template).
<li> Removal of reads based on hamming distance cutoff thresholds. 
<li> Reads spanning a particular region of the template can be multiply aligned and used to infer phylogenetic trees. and
<li> Identification of variants within viral populations.
</ol>
The user can make a quick phylogenetic scan of clusters within the data using 
the TreeDar feature, or
a detailed phylogenetic analysis of reads spanning a selected region of the
template with the ability to track individual codon states on the tree.
Segminator uses <a href="software.etc1.html#MUSCLE">MUSCLE</A> and
<a href="software.etc1.html#RAXML">RAxML</a> (which the
user must install separately) to produce
maximum likelihood trees for the phylogeny step.
Segminator is described in the paper:
Archer, J., A. Rambaut, B. E. Taillon, R. Harrigan, M. Lewis and D. Robertson.
2010.  The evolutionary analysis of emerging low frequency HIV-1 CXCR4-using
variants through time - an ultra-deep approach. <i>PLoS Computational
Biology</i> <b> 6 (12):</b> e1001022.
It is available as Java executables with associated Windows executables, Linux 
executables and Mac OS X universal executables.  It can be downloaded from
<a href="http://www.bioinf.manchester.ac.uk/segminator/">its web site</a>
at <code>http://www.bioinf.manchester.ac.uk/segminator/</code>
<P>

<HR>
<P>
<IMG SRC="icons/likewind.jpg" ALT="[Likewind icon]">&nbsp;&nbsp;<A NAME="Likewind"> John Archibald and Andrew Roger</A>, of the
Department of Biochemistry and Molecular Biology, Dalhousie University,
Halifax, Nova Scotia, Canada
(<tt>john.archibald<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;dal.ca</tt>
and <tt>aroger<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;is.dal.ca</tt>)
have released <strong>Likewind</strong> version 1.0.1.  This is a set of
three Perl scripts that allow the user to create the necessary
<a href="software.pars.html#PAUP">PAUP*</a> blocks to drive a likelihood
analysis of differences in phylogenies betweend different parts of a
molecule.   The object is to detect recombination, hybridization, or
other sources of conflicting phylogenetic signal.  The utilities also
harvest the output from PAUP* and present it to the user as a log-likelihood
difference plot.  This shows the log-likelihood difference in the window
of adjacent sites between a predefined tree and the best estimate from the
local information.  The scripts need
<a href="software.etc2.html#Seq-Gen">Seq-Gen</a> and PAUP* present in a
Linux or Unix environment.  The method has been described in a paper: 
Archibald, J. M. and A. J. Roger. 2002. Gene conversion and the evolution of
euryarchaeal chaperonins: a maximum likelihood-based method for detecting
conflicting phylogenetic signals. <em>Journal of Molecular Evolution</em>
<b>55:</b> 232-245.  Likewind is available from
<a
href="http://rogerlab.biochemistryandmolecularbiology.dal.ca/Software/Software.htm#likewind">its
web site</a> at
<code>http://rogerlab.biochemistryandmolecularbiology.dal.ca/Software/Software.htm#likewind</code>
<P>

<HR>
<A NAME="FORESTER"><P>Christian Zmasek</A>, currently of the Genomics Institute
of the Novartis Research Foundation in San Diego, California
(<CODE>czmasek<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>gnf.org</CODE>) and Sean Eddy
(<tt>eddy<!deathtospam>&nbsp; (at)<!deathtospam>&nbsp; genetics.wustl.edu</tt>)
of the Department of Genetics, Washington University, St. Louis, Missouri
have released <STRONG>FORESTER</STRONG>, version 1.92, a package of Java
routines for
inferring gene function using the output of high-throughput genome sequencing.
It includes the RIO Resampled Inference of Orthologs method which searches
for orthologs in the PFAM database, the SDI
Speciation-Duplication Inference method, and ATV, a tree viewer.  This can take
a tree in the standard Newick format (and also in an extension that has
additional information), and display it in various forms.   These are
described in papers:
<UL>
<LI>Zmasek, C.M. and S. R. Eddy.  2001. ATV: display and
manipulation of annotated phylogenetic trees. <EM>Bioinformatics</EM>
<B>17:</B>
383-384.
<LI>Zmasek, C. M. and S. R. Eddy. 2001. A simple algorithm to infer gene
duplication and speciation events on a gene tree. <EM>Bioinformatics</EM>
<B>17:</B> 821-828.
<LI>Zmasek, C. M. and S. R. Eddy. 2002.  RIO: Analyzing proteomes
by automated phylogenomics using resampled inference of orthologs. <EM>BMC
Bioinformatics</EM> <B>3:</B> 14.
</UL>
The programs (in Java) are available from
<A HREF="http://www.phylosoft.org/forester/">the Forester web
site</A>
at <CODE>http://www.phylosoft.org/forester/</CODE>.
A RIO web server is available from this group: see the listing
<A HREF="software.serv.html#RIO server">in the list of servers</A> in these pages.
<P>

<HR>
<A NAME="WebPHYLIP"><P>
Louxin Zhang</A> (<CODE>matzix<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>nus.edu.sg</CODE>) of the
Department of Mathematics of the Faculty of Science, 
of the National University of Singapore has produced
<strong>WebPHYLIP</strong>, version 2.0, a web interface
for the <A HREF="software.pars.html#PHYLIP">PHYLIP</A> package, which
can submit jobs to it. It has been described in an article:  Lim, A. and L.
Zhang.  1999.  WebPHYLIP: a web interface to PHYLIP. <i>Bioinformatics</i>
<b>15(12):</b> 1068-1069.  There are versions as Windows executables and
for Linux/Unix as C source code which can be compiled.  It is available at
<a href="http://www.math.nus.edu.sg/~matzlx/webphylip_download/">this download page</a>
at <code>http://www.math.nus.edu.sg/~matzlx/webphylip_download/</code>
above address.
<P>

<A NAME="Geneious">
<HR>
<P>
<IMG SRC="icons/geneious.gif"> &nbsp;&nbsp;Biomatters, Ltd.</A>
in Auckald, New Zealand
<tt>(sales<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;geneious.com)</tt>
has released <strong>Geneious</strong>
version 5.3, a Java package for sequence management, searching, editing and
phylogenies.  Geneious is an integrated bioinformatics tool suite for
manipulating, finding, sharing, and exploring biological data such as DNA
sequences or proteins, phylogenies, 3D structure information, publications,
etc. It features tree-based progressive sequence alignment and phylogenetic
analysis by Neighbor-Joining and UPGMA, bootstapping, drawing
the phylogeny, access to
biological databases, BLAST, protein structure viewing, NCBI, EMBL, PubMed
auto-find, and more. It includes an API for creating your own plugins.
It is available as Java executables.   Geneious is available for free;
that version cannot edit sequences or annotations, cannot run the multiple
alignment program ClustalW from within Geneious, and cannot do some other
tasks.  A full version, Geneious Pro, is available commercially.
Free downloads of Geneious or purchases of Geneious Pro can be done from
<a href="http://www.biomatters.com">the Biomatters web site</a>
 at  <code>http://www.biomatters.com</code>.
Geneious Pro is available from Biomatters, Ltd. for a 12-month license for academic or government
institutions for $395, for a 12-month student license for
$175, or for a 12-month commercial license for $995.   The prices for
perpetual licenses for these classes of users are $795, $395, and $1,995.
Discount prices for multiuser licenses
are also available.  Additional plugins enabling it to run
<a href="software.etc1.html#MrBayes">MrBayes</a>,
<a href="software.pars.html#PAUP">PAUP*</A> (Geneious Pro only),
<a href="software.etc1.html#PHYML">PHYML</A>,
<a href="software.etc1.html#DualBrothers">DualBrothers</A>,
and <a href="software.etc1.html#MAFFT">MAFFT</A>
are available free at the website.
<P>

<A NAME="PISE">
<HR>
<P>
Catherine Letondal</A>
of the Institute Pasteur, Paris, France
<tt>(letondal<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;pasteur.fr)</tt>
 distributes <strong>PISE</strong>
(Pasteur Institute Software Environment),
a tool to generate Web interfaces for Molecular Biology programs.  PISE allows 
the user to generate web page interfaces to many programs, including
<a href="software.pars.html#PHYLIP">PHYLIP</a>,
<a href="software.dist.html#EMBOSS">EMBOSS</a>,
<a href="software.etc1.html#MOLPHY">MOLPHY</a>,
and <a href="software.etc1.html#fastDNAML">fastDNAML</a>,
Web interfaces for these are available in PISE.  The web interfaces allow you to 
rapidly create a web server for running these programs.  PISE is used to 
create the Pasteur Institute web server.  PISE web servers allow the results 
of one program to the "piped" to another program. The web server generated is 
designed to be run on a Unix machine.  It is described in the paper:
Letondal C. 2001. A Web interface generator for molecular biology programs in 
Unix. <em> Bioinformatics</em>, <b>17(1):</b> 73-82.
It is available as Perl scripts.  It can be downloaded from
<a href="http://www.pasteur.fr/recherche/unites/sis/Pise/">its web site</a>
at <code>http://www.pasteur.fr/recherche/unites/sis/Pise/</code>
<P>

<A NAME="Bioinformatics_Toolbox">
<HR>
<P>
MathWorks</A> of  Natick, Massachusetts
have produced <strong>Bioinformatics Toolbox</strong>
version 3.6, a MATLAB toolbox for bioinformatics.  It has many functions for 
sequence analysis and microarray data, including multiple sequence alignment 
and consensus sequences.  For this listing, the relevant ones are that it 
enables you to create and edit phylogenetic trees. You can calculate pairwise 
distances between aligned or unaligned nucleotide or amino acid sequences 
using a broad range of similarity metrics, such as Jukes-Cantor, p-distance, 
alignment-score, or a user-defined distance method. Phylogenetic trees are 
constructed using hierarchical linkage with a variety of techniques, including 
neighbor joining, single and complete linkage, and UPGMA. Bioinformatics 
Toolbox includes tools for weighting and rerooting trees, calculating 
subtrees, and calculating canonical forms of trees. Through the graphical user 
interface, you can prune, reorder, and rename branches; explore 
distances; and read or write Newick-formatted files. You can also use the
annotation tools in MATLAB to create presentation-quality trees.
It is available as a MATLAB  package.  It is available from 
MathWorks, Inc. The price is not available on the website without a login,
but there are Commercial, Academic, and Student prices.  It us believed that
commercial licenses are about $1,000, academic licenses about $90. More
information and sales contact can be obtained from the product
<a href="http://www.mathworks.com/products/bioinfo/">web site </a>
at <code>http://www.mathworks.com/products/bioinfo/</code>
<P>

<A NAME="UGENE">
<HR>
<P>
<IMG SRC="icons/ugene.jpg">&nbsp;The UGENE team</A>
of the Unipro company, Novosibirsk, Russia
<tt>(mfursov<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;unipro.ru)</tt>
has released <strong>UGENE</strong>
version 1.8.0, a free cross-platform genome analysis suite. 
It is a sequence manipulation environment which can call other programs as
plugins, including <a href="software.pars.html#PHYLIP">PHYLIP</a> programs
to compute distances and create trees from distance matrix methods, and
<a href="software.etc1.html#MUSCLE">MUSCLE</A> to align sequences.  It can
also plot trees.
It is available as C++ source code, Windows executables, Linux executables,
Powermac Mac OS X executables and Intel Mac OS X executables.  It can be
downloaded from
<a href="http://ugene.unipro.ru">its web site</a>
at <code>http://ugene.unipro.ru</code>
<P>

<HR>
<A NAME="Phyl-O-Gen">
<P><IMG SRC="icons/phylogen.gif">&nbsp;&nbsp; Andrew Rambaut
of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
has written <strong>Phylogen</strong> version 1.1, a tree simulation
program.  It simulates phylogenies produced by a birth-death process.
It also has a mode that allows for multiple epdisodes of evolution and
multiple mass extinctions.  It is available as generic source code for Unix
and as Mac OS, Mac OS X, and Windows executables.  It is distributed from
<A HREF="http://tree.bio.ed.ac.uk/software/phylogen/">its web page</A>
at <CODE>http://tree.bio.ed.ac.uk/software/phylogen/</CODE>.
Although Rambaut considers it to have superseded his earlier program
<A HREF="software.old.html#Bi-De">Bi-De</A>.  Phylogen is itself now
unsupported software that will be distributed but not developed further.
<P>

<HR>
<A NAME="Genie">
<P>
<IMG SRC="icons/genie.gif" ALT="[Genie icon here]">
Oliver Pybus</A>, of the Department of
Zoology,
University of Oxford, (<CODE>oliver.pybus<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zoo.ox.ac.uk</CODE>
and Andrew Rambaut, of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
have written <STRONG>Genie</STRONG>
(<em>GEN</em>ealogical <em>I</em>nterval <em>E</em>xplorer)
version 3.0, a program for
the inference of demographic history from reconstructed phylogenies.
The methods it implements are described in the papers:
<UL>
<LI>Pybus, O., A. Rambaut
and P. Harvey. 2000. An integrated framework for the inference of viral
population history from reconstructed genealogies. <EM>Genetics</EM>
<B>155:</B> 1429-1437.
<LI> Pybus, O. G., M. A. Charleston, S. Gupta, A. Rambaut, E. C.  Holmes, and
P. H. Harvey. 2001. The epidemic behavior of the Hepatitis C Virus.
<EM>Science</EM> <B>292:</B> 2323-2325.
<LI>Strimmer, K. and O. G. Pybus. 2001. Exploring the demographic history of
DNA sequences using the generalised skyline plot. <EM>Molecular Biology and
Evolution</EM> <B>18:</B> 2298-2305.
</UL>
It is considered by its authors to supersede the methods used in
Rambaut's program <A HREF="software.old.html#End-Epi">End-Epi</A>, which has
been withdrawn
from distribution.  Genie is available from
<A HREF="http://evolve.zoo.ox.ac.uk/evolve/Genie.html">its web site</A> at
<CODE>http://evolve.zoo.ox.ac.uk/evolve/Genie.html</CODE>
as Mac OS, Linux or Windows executables, and as C source code for
Unix.
<P>

<HR>
<P>
<IMG SRC="icons/macrocaic.gif" ALT="[MacroCAIC icon]">
<A NAME="MacroCAIC">Paul-Michael Agapow</A>
(<TT>p.agapow<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ic.ac.uk</TT>)
and Nick Isaac, then of the Department of Biology,
Imperial College, Silwood Park, U.K. (and most recently of the Health
Protection Agency, U.K.)
has released <STRONG>MacroCAIC</STRONG> version 1.0.1, which was developed from
<A HREF="#CAIC">CAIC</A>, by Andy Purvis and Andrew Rambaut.  MacroCAIC
uses phylogenies and data sets of character values to examine correlates of
species richness, i.e. which traits are associated with abnormal speciation.
The phylogeny is used to make these correlations independent. Traits may be
continuous or discrete, and not every trait value needs to be known for every
clade in the phylogeny.  MacroCAIC has been described in a paper: Agapow, P.-M.
and N. J. B. Isaac. 2002. MacroCAIC: correlates of species richness.
<em>Diversity & Distributions</em> <b>8:</b> 41-43.  MacroCAIC is a PowerMac
and 68k Mac binary executable for Mac OS.  It is available from
<A HREF="http://www.bio.ic.ac.uk/evolve/software/macrocaic/index.html">its web
site</A>
at <CODE>http://www.bio.ic.ac.uk/evolve/software/macrocaic/index.html</CODE>.
<P>

<HR>
<P>
<IMG SRC="icons/seq-gen.gif" ALT="[Seq-Gen icon here]">&nbsp;&nbsp;
<A NAME="Seq-Gen">Andrew Rambaut, of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
University of Oxford, (<CODE>andrew.rambaut<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zoo.ox.ac.uk</CODE>) and
Nick Grassly</A> of the Department of Infectious Disease Epidemiology of
Imperial College School of Medicine, St. Mary's Campus, London
(<tt>n.grassly<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ic.ac.uk</tt>)
have written <STRONG>Seq-Gen</STRONG> (Sequence Generator), version 1.3.2,
a program that will simulate the evolution of nucleotide sequences
and protein sequences
along a phylogeny or multiple phylogenies, using common models of the 
nucleotide or protein substitution process. A range
of models of molecular evolution are implemented 
Nucleotide frequencies and other parameters of the model may be given
and site-specific rate heterogeneity may also be incorporated in a number
of ways.  The models available are the Hasegawa, Kishino and Yano (HKY) model,
the Felsenstein F84 model, the general reversible model, the Kimura 2-parameter
model, the Jukes-Cantor model and the Dayhoff PAM, JTT, Blosum62, mtREV,
and WAG amino acid substitution models.  Rate heterogeneity among sites or
among the different positions within a codon can be specified.
Seq-Gen is described in a paper: Rambaut, A. and N. C. Grassly.  1997.
Seq-Gen: an application for the Monte Carlo simulation of DNA sequence
evolution along phylogenetic trees. <em>Computer Applications in the
Biosciences ICABIOS)</em> <b>13:</b> 235-238.
A Mac OS executable is available, as well as source code
files for Unix systems. A Windows executable is available for an earlier
version, 1.3.1.  These are available from
<A HREF= "http://tree.bio.ed.ac.uk/software/seqgen/">its Web
page</A> at
<CODE>http://tree.bio.ed.ac.uk/software/seqgen/</CODE>.
<P>

<A NAME="SGRunner">
<HR>
<P>
<IMG SRC="icons/SGRunner.jpg">&nbsp;&nbsp;Tom Wilcox</A> has written <strong>SG
Runner</strong>
version 2.0.1, a graphical user interface for running <a
href="#Seq-Gen">Seq-Gen</a>.  It is available as Powermac Mac OS X executables.
It can be downloaded from <a
href="http://homepage.mac.com/tpwilcox/FileSharing15.html">his software web
site</a>
at <code>http://homepage.mac.com/tpwilcox/FileSharing15.html</code>
<P>

<HR>
<P>
<IMG SRC="icons/PSeq-Gen.gif" ALT="[PSeq-Gen icon here]">
<A NAME="PSeq-Gen">Nick Grassly</A>
of the Department of Infectious Disease Epidemiology of
Imperial College School of Medicine, St. Mary's Campus, London
(<tt>n.grassly<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ic.ac.uk</tt>)
and Andrew Rambaut, of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
have written <STRONG>PSeq-Gen</STRONG> (Protein-Sequence Generator), version
1.1, which will simulate the evolution of protein sequences along evolutionary
trees.
Three common models of amino acid substitution are implemented (PAM, JTT,
and mtREV), allow for user-defined amino acid frequencies.  Site-specific rate
heterogeneity following a gamma distribution is allowed.  The program can
handle multiple trees and produce multiple data sets.
PSeq-Gen is available from
<A
HREF="http://iubio.bio.indiana.edu/soft/iubionew/molbio/evolution/phylo/PSeq-Gen/main.html">a web site at the IUBIO server</A>
at <CODE>http://iubio.bio.indiana.edu/soft/iubionew/molbio/evolution/phylo/PSeq-Gen/main.html</CODE> as
Unix source code and also as Mac OS executables.  An online manual can also
be viewed at that site.  PSeq-Gen is now largely superseded by <a href="#Seq-Gen">Seq-Gen</a>,
and although it is still available at the IUBIO server it is not being
actively supported by its authors.
<P>

<HR>
<P>
<IMG SRC="icons/treevolve.gif" ALT="[Treevolve icon here]">
<A NAME="Treevolve">Nick Grassly</A>, of the School of
Public Health of Imperial College, St. Mary's Campus, London
(<tt>n.grassly<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;imperial.ac.uk</tt>)
 and
Andrew Rambaut, of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
wrote
<STRONG>Treevolve</STRONG>, version 1.32 and also <STRONG>Ptreevolve</STRONG>,
programs that simulate the evolution of DNA and protein sequences
respectively. The molecular sequences are simulated under coalescent models
with constant population size, or with exponential population size growth.
In addition different levels of recombination can be specified.
In Treevolve, it is also possible to have an island model of population
subdivision.
Treevolve and Ptreevolve are written in ANSI C and should compile on most Unix
systems.  They are also available as Mac OS executables, and
a project file for the MetroWerks Codewarrior compiler is included
in the Macintosh archive.  They can be obtained from
programs viewed, from
<A HREF="http://microbe.bio.indiana.edu:7131/soft/iubionew/molbio/genetic/pop/TreeVolve/">their archives at the IUBIO web site</A>
at <CODE>http://microbe.bio.indiana.edu:7131/soft/iubionew/molbio/genetic/pop/TreeVolve/</CODE>  
<P>

<HR>
<P>
<A NAME="ROSE">Jens Stoye, Dirk Evers and Folker Meyer</A> of the
Research Center for Interdisciplinary Studies on Structure Formation (FSPM) and 
the Technische Fakult&auml;t of the University of Bielefeld, Germany
(<CODE>j.stoye<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>dkfz-heidelberg.de</CODE>,
<CODE>dirk<!deathtospam>&nbsp; (at)<!deathtospam>&nbsp;
TechFak.Uni-Bielefeld.de</CODE>, and <CODE>folker<!deathtospam>&nbsp;
(at)<!deathtospam>&nbsp; TechFak.Uni-Bielefeld.de</CODE>) have released
<STRONG>ROSE</STRONG>, the <i>R</i>andom model <i>O</i>f <i>S</i>equence <i>E</i>volution, version 1.3.
It simulates the evolution of DNA, RNA, or protein sequences on a randomly
generated tree, allowing insertions and deletions and substitution at
different rates at different sites as
well.  It can also use a predefined tree that is input in standard format.
It can report ancestral sequences or sequences at the tips of the tree, and
it also keeps a record of the true multiple sequence alignment for comparison
with the results of multiple sequence alignment programs.
ROSE is described in the paper:
Stoye, J. D. Evers and F. Meyer.  1998.  Rose: generating sequence families.
<EM>Bioinformatics</EM> <B>14:</B> 157-163.
ROSE is available in source code at
<A HREF="http://bibiserv.techfak.uni-bielefeld.de/rose/">its web site</A>
at <CODE>http://bibiserv.techfak.uni-bielefeld.de/rose/</CODE>.
<P>

<A NAME="Simprot">
<HR>
<P>
Andy Pang, Andrew Smith, Paulo Nuin and Elisabeth Tillier</A>
of the Cancer Genomics and Proteomics Division of the Ontario Cancer Institute
and the Department of Medical Biophysics
of the University of Toronto, Canada
<tt>(e.tillier<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;utoronto.ca)</tt>
have released <strong>Simprot</strong>
(<i>Sim</i>ulation of <i>Prot</i>eins),
version 1.03, a program to simulate protein evolution by substitution,
insertion and deletion.  It allows for several models of amino acid
substitution (PAM, JTT and PMB), it allows for gamma distributed sites rates
according to Yang's model, and it implements a parameterised Qian and Goldstein
distribution model for insertion and deletion. 
It is described in the paper:
Pang, A., A. Smith., P. Nuin, and E. R. M. Tillier.  2005. SIMPROT: Using an
empirically determined indel distribution in simulations of protein evolution
<em>BMC Bioinformatics</em> <b>6:</b> 236.
It is available as C source code and Windows executables.  It can be downloaded
from
<a href="http://www.uhnresearch.ca/labs/tillier/software.htm">its web site</a>
at <code>http://www.uhnresearch.ca/labs/tillier/software.htm</code>
<P>

<A NAME="indel-Seq-Gen">
<HR>
<P>
Cory Strope, S. D. Scott and Etsuko Moriyama</A>
then of the Department of Computer Science and School of Biological Sciences
of the University of Nebraska
<tt>(corystrope<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;gmail.com)</tt>
have released <strong>indel-Seq-Gen</strong>
version 2.1.0,  a sequence family simulator incorporating domains, motifs, and
indels.  it simulates a more realistic evolutionary
process of protein sequences including insertions and deletions (indels).
iSG allows the user to simulate multiple subsequences according to
different evolutionary parameters, which is necessary for generating realistic
protein families with multiple domains. It tracks all evolutionary events
including indels and
outputs the "true" multiple alignment of the simulated sequences.  iSG can also
generate a larger sequence space by allowing the use of multiple related root
sequences. It is intended to be used to test the accuracy of multiple alignment
methods, phylogenetic methods, evolutionary hypotheses, ancestral protein
reconstruction methods, and protein family classification methods. 
It is described in the paper:
Strope, C. L., S. D. Scott, and E. N. Moriyama. 2007. indel-Seq-Gen: A new
protein family simulator incorporating domains, motifs, and indels.
<em>Molecular Biology and Evolution</em> <b>24:</b> 640-649.
It is available as C source code, Perl script, Linux executables and
Intel Mac OS X executables.  It can be downloaded from
<a href="http://bioinfolab.unl.edu/~cstrope/iSG">its web site</a>
at <code>http://bioinfolab.unl.edu/~cstrope/iSG</code>
<P>

<HR>
<A NAME="ProSeq"><P>
<IMG SRC="icons/proseq.gif" ALT="[ProSeq icon]">&nbsp;
Dmitry Filatov</A>, of the Department of Plant Sciences of
the University of Oxford, U.K.
(<TT>dmitry.filatov<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>plants.ox.ac.uk</TT>) has
released <STRONG>ProSeq</STRONG> (<i>Pro</i>cessor of <i>Seq</i>uences) 
version 3.  ProSeq is a sequence-editing environment that can do sequence 
alignment editing, translation, detection of polymorphic sites, and
a variety of tests, many of a population-genetic nature, for neutrality
and recombination.  The part of its capabilities that are relevant to this
listing is that it can simulate the evolution of a set of DNA sequences
along a coalescent tree, with or without recombination. Version 3 allows
simultaneous alignment of multiple loci.  An earlier version of
ProSeq is described
in a paper: Filatov, D.A. 2002. ProSeq: A software for preparation and
evolutionary analysis of DNA sequence data sets.
<em>Molecular Ecology Notes</em> <b>2:</B> 621-624.  
ProSeq is available as Windows or Linux executables from
<A HREF="http://dps.plants.ox.ac.uk/sequencing/proseq.htm">its web site</A> at
<CODE>http://dps.plants.ox.ac.uk/sequencing/proseq.htm</CODE>.
<P>

<HR>
<A NAME="MESA"><P><IMG SRC="icons/mesa2.jpg"> &nbsp; Paul Michael Agapow</A>
(<tt>agapow<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;agapow.net</tt>), then
of the Department of Biology, Imperial College, Silwood Park, U.K. (most
recently
of the Health Protection Agency, U.K.) has released
<strong>MESA</strong> (<i>M</i>acro<i>E</i>volutionary analysis and
<i>S</i>imul<i>A</i>tion),
version 1.9.23, a program to simulate evolution of a
group, allowing for a variety of kinds of extinction mechanisms.
It can simulate evolution and describe the diversity of the resulting
groups.  Its actions can be recorded and repeated on multiple trees.
It also has features for editing and manipulating trees.
MESA is available as an executable for Windows and as an executable
for Mac OS X from
<a href="http://www.agapow.net/software/mesa">its web site</a>
at <code>http://www.agapow.net/software/mesa</code>
<P>

<A NAME="DAWG">
<HR>
<P>
<img src="icons/dawg.jpg" ALT="[DAWG icon]">&nbsp;&nbsp;Reed Cartwright</A>
of the Department of Biology and Biochemistry at the University of Houston,
Houston, Texas
<tt>(reed<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;scit.us)</tt>
has written <strong>DAWG</strong> (<i>D</i>NA <i>A</i>ssembly <i>W</i>ith
<i>G</i>aps),
version 1.2, a program to simulate evolution of DNA sequences with 
recombination and gaps.  It is designed to simulate the evolution of 
recombinant DNA sequences in continuous time based on the general time 
reversible model with gamma and invariant rate heterogeneity and a novel 
length-dependent model of gap formation. It accepts phylogenies in Newick 
format and can return the sequence of any node, allowing for the exact 
evolutionary history to be recorded at the discretion of users. Dawg records 
the gap history of every lineage to produce the true alignment in the output. 
Many options are available to allow users to customize their simulations and 
results.  It is described in the paper:
Cartwright, R. 2005.  DNA Assembly with gaps (DAWG): simulating sequence
evolution.  <em>Bioinformatics</em>  <b>21 (Supplement 3):</b> iii31-iii38.
It is available as C source code.  It can be downloaded from
<a href="http://scit.us/projects/dawg/wiki">its web site</a>
at <code>http://scit.us/projects/dawg/wiki</code>
<P>

<A NAME="EvolveAGene3">
<HR>
<P>
Barry Hall</A> of the Bellingham Research Institute in  Bellingham, Washington
<tt>(barryhall<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;zeninternet.com)</tt>
has written <strong>EvolveAGene3</strong>, a program that simulates evolution 
of a protein sequence along a tree.  It generates a bifurcating tree, and 
assigns branch lengths from a distribution whose mean is specified by the 
user.  A protein sequence is evolved along this tree, with deletions and 
insertions of codons and with base subtitutions.  Substitutions that change 
the amino acid are accepted with a specified probability.
This includes having variable regions of selection
intensity, positive or purifying, within the sequence and variation
in intensity of selection over branches.
Output includes the true tree and unaligned coding sequences and protein
sequences as well as the true DNA and the true protein alignments.
It is available as Perl script, Windows executables and Mac OS X universal 
executables. See
<a href="http://web.me.com/barryghall/Software/Software.html">the web site </a>
<code>http://web.me.com/barryghall/Software/Software.html</code>
<P>

<A NAME="EvolSimulator">
<HR>
<P><IMG SRC="icons/ES.jpg" ALT="[EvolSimulator icon]">&nbsp;&nbsp;
Robert Beiko and Robert Charlebois</A>
of the Faculty of Computer Science at Dalhousie University
<tt>(beiko<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;cs.dal.ca)</tt>
have released <strong>EvolSimulator</strong>
version 2.1.0, a program to simulate the evolution of genes and genomes.  
EvolSimulator was created to allow the simulation of complex evolutionary 
regimes, potentially comprising nonstationary and nonuniform processes at the 
sequence level, alongside genome-level processes such as gene duplication, 
gene loss, and lateral gene transfer. Several models of LGT are implemented, 
including random transfers, transfers that favour close relatives in the 
organismal tree, and preferential transfer among members of the same habitat.
It is described in the paper:
Beiko, R. G. and R. L. Charlebois. 2007. A simulation test bed for hypotheses
of genome evolution. <em>Bioinformatics</em> <b>23 (7):</b> 825-31. 
It is available as C++ source code.  It can be downloaded from
<a href="http://bioinformatics.org.au/evolsim">its web site</a>
at <code>http://bioinformatics.org.au/evolsim</code>
<P>

<A NAME="Recodon">
<HR>
<P>
<img src="icons/recodon.jpg" alt="[Recodon icon]">&nbsp;&nbsp;
Miguel Arenas and David Posada</A>
of the Department of Biochemistry, Genetics and Immunology
of the University of Vigo, Spain
<tt>(miguelmmmab<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;gmail.com)</tt>
have released <strong>Recodon</strong>
(coalescent simulation of Coding sequences with Recombination),
version 1.6.0, A simulator of coding sequences based on the coalescent with 
recombination, migration and demographics.  The program simulates coding and 
non-coding sequences using the coalescent and with recombination, migration 
and demographics. It implements all major DNA substitution models of evolution 
and the GY94 codon model including rate heterogenity among codons.
It is described in the paper:
Arenas, M. and D. Posada, 2007. Recodon: Coalescent simulation of coding DNA sequences with recombination, migration and demography. <i>BMC Bioinformatics</i> <b>8:</b> 458.
It is available as C source code, Windows executables, Linux executables and Powermac Mac OS X executables.  It can be downloaded from
<a href="http://darwin.uvigo.es/software/recodon.html">its web site</a>
at <code>http://darwin.uvigo.es/software/recodon.html</code>
<P>

<A NAME="NetRecodon">
<HR>
<P>
<img src="icons/netrecodon.jpg" alt="[NetRecodon icon]">&nbsp;&nbsp;Miguel 
Arenas and David Posada</A>
of the Department of Biochemistry, Genetics and Immunology
of the University of Vigo, Spain
<tt>(miguelmmmab<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;gmail.com)</tt>
have released <strong>NetRecodon</strong>
(coalescent simulation with intracodon Recombination),
version 6.0.0, A simulator of coding sequences based on the coalescent with 
(intra and inter codon) recombination, migration and demographics.  The 
program simulates coding and non-coding sequences using the coalescent  with 
recombination (where breakpoints can occur at both intra and inter codon 
positions), migration and demographics. Indeed, the program implements many DNA 
substitution models of evolution and also GY94 codon models including 
heterogenity of rate of evolution among codons.  This program is more advanced 
than the previous program <a href="#Recodon">Recodon</a>.
One improvement is that it allows recombination within as well
as between codons.  It is described in the paper:
Arenas, M. and D. Posada. 2010. Coalescent simulation of intracodon recombination. <i>Genetics</i> <b>184(2):</b> 429-437.
It is available as C source code, Windows executables, Linux executables and Powermac Mac OS X executables.  It can be downloaded from
<a href="http://darwin.uvigo.es/software/netrecodon.html">its web site</a>
at <code>http://darwin.uvigo.es/software/netrecodon.html</code>
<P>

<HR>
<A NAME="W2H"><P>Martin Senger</A> (<tt>senger<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ebi.ac.uk</tt>) of the European
Bioinformatics Institute, Hinxton, U.K. and Peter
Ernst (<tt>P.Ernst<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>dkfz-heidelberg.de</tt>) of
the Deutsches Krebsforschungszentrum, Heidelberg, Germany distribute
<strong>W2H</strong>, (<i>W</i>eb <i>2</i> <i>H</i>usar) version 4.1.2, a web 
interface for running molecular
sequence analysis programs.  It can invoke a variety of sequence analysis
programs, including the EMBOSS Embassy versions of a number of
<a href="software.pars.html#PHYLIP">PHYLIP</A> 3.5 programs.  W2H is
a set of web pages that has been developed primarily with Unix platforms in
mind.  They can be downloaded and are available from
<a href="http://www.w2h.dkfz-heidelberg.de/"> the W2H web page</a>
at <code>http://www.w2h.dkfz-heidelberg.de/</code>
<P>

<HR>
<A NAME="Phyledit"><P>
<IMG SRC="icons/phyledit.gif">&nbsp; Juan Jos&eacute; de Haro</A>
(<tt>jjdeharo<! death to spam>&nbsp;(at)<! death to spam>&nbsp;gmail.com</tt>)
of "una peque&ntilde;a
ciudad" of the province of Barcelona, Spain, has released
<strong>Phyledit</strong> version 2.0.  This is an interactive data
editing and analysis program 
that uses <A HREF="software.pars.html#PHYLIP">PHYLIP</A> and
<A HREF="software.etc1.html#Treeview">Treeview</A> for analysis and
display of trees.  It uses eight programs from PHYLIP, all concerned with
discrete 0/1 characters.  Phyledit's menus, responses and documentation are
all in Spanish.  It is a Windows executable downloadable available from
<A HREF="http://perso.wanadoo.es/jjdeharo/phyledit/index.htm">its web site</A>
at <CODE>http://perso.wanadoo.es/jjdeharo/phyledit/index.htm</CODE>.  It
is available there either by itself or with the PHYLIP and TREEVIEW
executables as well.
<P>

<HR>

<A NAME="PAST"><P>
<IMG SRC="icons/past.gif">&nbsp; &Oslash;yvind Hammer</A>, of the
Paleontological Museum of the University of Oslo, Norway
(<tt>ohammer<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>nhm.uio.no</tt>), together with
David A.T. Harper of the Geological Museum, Copenhagen, Denmark, and P.D. Ryan,
has written <strong>PAST</strong>
(<i>PA</i>leontological <i>ST</i>atistics), version 2.04, a package which carries out many
kinds of paleontological
data analyses, including stratigraphic and morphometric statistics.  It
also does parsimony analysis, including exhaustive, branch-and-bound and
heuristic algorithms for Wagner, Fitch and Dollo parsimony. It does bootstrap
methods, strict and majority rule consensus trees, and consistency and
retention indices. It calculates three stratigraphic congruency indices with
permutation tests.  It also does many other statistics and curve fitting.
PAST is described in a paper: Hammer, &Oslash;.,
D.A.T. Harper, and P. D. Ryan. 2001. PAST: Paleontological statistics
software package for
education and data analysis. <em>Palaeontologia Electronica</em> <b>4:</b>,
issue 1: <a href="http://palaeo-electronica.org/2001_1/past/issue1_01.htm">
<CODE>http://palaeo-electronica.org/2001_1/past/issue1_01.htm</CODE></a>.
PAST is available from <A HREF="http://folk.uio.no/ohammer/past/index.html">
its web site</A> at <code>http://folk.uio.no/ohammer/past/index.html</code>
as a Windows executable.  Manuals can be read online or downloaded from the
web site.
<P>

<A NAME="GHOSTS">
<HR>
<P>
<img src="icons/ghosts.jpg" alt="[GHOSTS icon]"> &nbsp;&nbsp; Matthew Wills</A> of the Department of Biology
at the the University of Bath,. U.K.
<tt>(bssmaw<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;bath.ac.uk)</tt>
has produced <strong>GHOSTS</strong>
version 2.4, a program for significance tests for RCI, SCI and GER values by
randomization.
It calculates the Relative Completeness Index (RCI), the Stratigraphic 
Consistency Index (SCI), and the Gap Excess Ratio (GER) measures of 
consistency of phylogenies with the stratigraphic record if given one or more 
tree topologies and stratigraphic range data for up to 74 terminal taxa.  It 
can be used with NEXUS tree files and can be used interactively with <a 
href="software.pars.html#MacClade">MacClade</a>.  It can also randomly permute 
the assignment of stratigraphic ranges among taxa, while holding tree 
topologies constant, to yield a distribution of values. It tests whether the 
RCI and SCI values for the original data differ significantly from the random 
distributions.  Its methods are described in the paper:
Wills, M. A. 1999. Congruence between phylogeny and stratigraphy: randomization
tests. <em>Systematic Biology</em> <b>48</b> 559-580.
It is available as source code for use with Chipmunk Basic.  It can be
downloaded from the
<a href="http://www.bath.ac.uk/bio-sci/biodiversity-lab/?page_id=281">Biodiversity Lab resources web page</a>
at <code>http://www.bath.ac.uk/bio-sci/biodiversity-lab/?page_id=281</code>
&nbsp;The program requires a password that can then be obtained from him.
<P>

<A NAME="QDate">
<HR>
<P>
<IMG SRC="icons/qdate.gif" ALT="[QDate icon here]">
Andrew Rambaut, of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
has written <STRONG>QDate</STRONG> version 1.1.1.  QDate
estimates the date of divergence between two pairs of sequences given
that the date of divergence of the members of each pairs is known.
It analyzes the data under three models: (1) a perfectly clocklike model,
(2) a model in which one pair has a different rate of divergence than the
other, and (3) a model in which all branches have different rates.
The method is described in the paper: Rambaut, A., and L. Bromham.  1998.
Estimating divergence dates from molecular sequences. <EM>Molecular
Biology and Evolution</EM> <B>15:</B> 442-448.  QDate is available from
<A HREF="ftp://net.bio.net/iubionew/molbio/evolution/phylo/QDate/main.html">its
web page at the IUBIO server</A> at
<CODE>ftp://net.bio.net/iubionew/molbio/evolution/phylo/QDate/main.html</CODE>.  It is
available
as C source code for Unix or as a Macintosh or Windows executable.
<P>

<HR>
<A NAME="RRTree"><P>
Marc Robinson-Rechavi</A>, of the Department of Ecology and Evolution of the
Universit&eacute; de Lausanne, Lausanne, Switzerland
(<TT>Marc.Robinson-Rechavi<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>unil.ch</TT>) has written
<STRONG>RRTree</STRONG>,
(Relative Rate tests within a Tree), version 1.1.  It carries out relative rate
tests
for equality of evolutionary rates in DNA or protein sequences between
lineages,
taking into account the structure of the tree, which can be input in a number
of
common formats.  In addition sequences are read in.
The methods are described in a papers:
<UL>
<LI>Robinson, M., M. Gouy, C. Gautier, and D. Mouchiroud. 1998. Sensitivity of
the relative-rate test to taxonomic sampling, <EM>Molecular Biology and
Evolution</EM> <B>15:</B> 1091-1098.
<LI>Robinson-Rechavi, M. and D. Huchon. 2000. RRTree: Relative-rate tests
between groups of sequences on a phylogenetic tree. <em>Bioinformatics</em>
<b>16:</b> 296-297. 
</UL>
Marc Robinson-Rechavi suggests that other programs might do better and
maintains the source code and documentation in distribution in case they
might be of interest, but the program is not being further developed.
RRtree is available as C source code and documentation from
<A HREF="http://bioinfo.unil.ch/rrtree/">its web page</A> at
<CODE>http://bioinfo.unil.ch/rrtree/</CODE>
<P>

<HR>
<A NAME="rate-evolution"><P>
Bette Korber</A> of the Theoretical Division,
Los Alamos National Laboratory , Los Alamos, New Mexico
(<TT>btk<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>lanl.gov</TT>), Mark Muldoon
and  and their colleagues
have released <STRONG>programs for rate evolution</STRONG>
which implement the method described by Jeff Thorne, Hirohisa Kishino,
and Ian Painter
of North Carolina State University for making a maximum likelihood
estimate of the phylogeny while allowing rates of evolution to
change along the tree in a "relaxed molecular clock".  Their method
was described in a paper: Thorne, J. L., H. Kishino, and I. S. Painter.
1998.  Estimating the rate of evolution of the rate of molecular
evolution. <EM>Molecular Biology and Evolution</EM> <B>15:</B> 1647-1657.
Korber and her colleagues used it for their paper:
Korber, B., M. Muldoon, J. Theiler, F. Gao, R. Gupta,
A. Lapedes, B. H. Hahn, S. Wolinksy and T. Bhattacharya. 2000. Timing the
ancestor of the HIV-1 pandemic strains. <EM>Science</EM>  <B>288:</B>
1789-1796.  Three programs are available (all as C code for Unix):
<UL>
<LI><B>mlcov</B>, which produces a covariance matrix (Hessian matrix)
of rates among branches which is needed by the other programs,
<LI><B>rate-evolution</B>, which carries out the method of Thorne et. al.
and gives likelihoods and estimates of rates on the different branches
(a version which was used in the Science paper is available as well as
a corrected version written by Muldoon), and
<LI><B>TREEPAINT</B>, which takes a Postscript file from the tree-drawing
program <A HREF="#TreeTool">TreeTool</A> and "colorizes" it by classifying
branches into five categories by their estimated rates of evolution.
</UL>
The programs are available from
<A HREF="http://www.santafe.edu/~btk/science-paper/bette.html">
the web site for the programs from that paper</A> at
<CODE>http://www.santafe.edu/~btk/science-paper/bette.html</CODE>.
Two other programs used for the analyses in the paper are available there too.
<P>

<A NAME="Multidivtime">
<HR>
<P>
Jeff Thorne</A>
of the Department of Genetics
at North Carolina State University, Raleigh, North Carolina
<tt>(thorne<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;statgen.ncsu.edu)</tt>
has written <strong>Multidivtime</strong>, version 09.25.03, a program that 
estimates divergence times from multiple-locus data.  The method starts with a 
given tree supplied by the user.  The Bayesian dating method implemented in 
multidivtime uses a probabilistic model to describe the change in evolutionary 
rate over time and uses a Markov chain Monte Carlo (MCMC) procedure to derive 
the posterior distribution of rates and times. It allows simultaneous use of 
different substitution models for multiple data partitions as well as multiple 
calibration windows, and provides direct credibility intervals for estimated 
divergence times and substitution rates.  The dating method is described in
an earlier paper:  Thorne, J. L., H. Kishino, and I. S. Painter. 1998.
Estimating the rate of evolution of the rate of molecular evolution.
<em>Molecular Biology and Evolution</em> <b>15:</b> 1647-1657.
It is available as C source code which can be compiled on Mac OS X or other
standard Unix systems, and as Windows executables.
Multidivtime is to be used in combination with some of the programs from
Ziheng Yang's <a href="software.etc1.html#PAML">PAML</a> package.  
Multidivtime can be downloaded from
<a href="http://statgen.ncsu.edu/thorne/multidivtime.html">its web site</a>
at <code>http://statgen.ncsu.edu/thorne/multidivtime.html</code>.  An earlier
version is available from Ziheng Yang's lab at
<a href="http://abacus.gene.ucl.ac.uk/software.html">their web site</a> at 
<code>http://abacus.gene.ucl.ac.uk/software.html</code>
where it is called T3, the Thornian Time Traveler.  Yang suggests using the
more recent version at Thorne's site.
<P>

<A NAME="CodonRates">
<HR>
<P>
Tae-Kun Seo</A>
of the Professional Programme for Agricultural Bioinformatics
of the University of Tokyo, Japan
<tt>(seo<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;iu.a.u-tokyo.ac.jp)</tt>
has written <strong>CodonRates</strong> version 1.0,
a program to estimate divergence times and absolute rates of synonymous and 
nonsynonymous substitution with protein-coding genes.  Given sequences and a 
tree, CodonRates can separate the amount of evolution into time and the 
synonymous and nonsynonymous rates without using a molecular clock. Data other 
than sequences, such as fossil constraints and a prior distribution for the 
root time, are required.  It is described in the paper:
Seo T-K, H. Kishino, and J. L. Thorne.  2004. Estimating absolute rates of 
synonymous and nonsynonymous nucleotide substitution in order to characterize 
natural selection and date species divergences. <em>Molecular Biology and 
Evolution</em> <b>21(7):</b> 1201-1213.
It is available as C++ source code and Windows executables.  It can be 
downloaded from
<a href="http://www.iu.a.u-tokyo.ac.jp/~seo/software.htm">its web site</a>
 at  <code>http://www.iu.a.u-tokyo.ac.jp/~seo/software.htm</code>
<P>

<A NAME="PATHd8">
<HR>
<P>
Tom Britton, Cajsa Anderson, David Jacquet, Samuel Lundqvist, and K&aring;re
Bremer</A>
of the Department of Mathematics and the Department of Systematic Botany
of  Stockholm University, Stockholm, Sweden (Anderson is at Uppsala Univerity)
<tt>(tomb<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;math.su.se)</tt>
have released <strong>PATHd8</strong>
(Path-date),
a program to estimate divergence times from a nonclocklike tree and time 
constraints.  PATHd8 reads in a tree which has branch lengths but with no 
requirement that these show a molecular clock.  It also reads in constraints 
on the divergence times.  These are then reconciled using either the PATHd8 
method or the Mean Path Length method.  It is described in the papers:
<UL>
<LI> Britton, T.  2005. Estimating divergence times in phylogenetic trees
without a molecular clock. <em>Systematic Biology</em> <b>54 (3):</b> 500-507. 
<LI> Britton, T., B. Oxelman, A. Vinnersten, and K. Bremer. 2002. Phylogenetic
dating with confidence intervals using mean path lengths. <em>Molecular
Phylogenetics and Evolution</em> <b>24:</b> 58-65.
</UL>
It is available as C source code, Windows executables, Linux executables,
Powermac Mac OS X executables, and Sun SPARC executables.
It can be downloaded from
<a href="http://www.math.su.se/PATHd8/index.html">its web site</a>
at <code>http://www.math.su.se/PATHd8/index.html</code>
<P>

<HR>
<A NAME="branchlength"><P>
Bette Korber</A> of the Theoretical Division,
Los Alamos National Laboratory , Los Alamos, New Mexico
(<TT>btk<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>t10.lanl.gov</TT>), and her
colleagues
have released <STRONG>BRANCHLENGTH</STRONG>, a pair of programs
which allow trees to have their branch lengths to any node
plotted against time.  They take output files from the
<A HREF="software.pars.html#PHYLIP">PHYLIP</A> program DNAML, or from
Gary Olsen's program <A HREF="software.etc1.html#fastDNAml">fastDNAmlM</A>,
or from one of the modified versions of fastDNAml as input.  One program
(BRANCHLENGTHxTIME) plots branch lengths to any specified node against
time, makes best fit lines, and does Monte Carlo resampling statistics. 
The other program (BRANCHLENGTH) computes The sum of branch lengths to any
specified node.  Korber and her colleagues used it for their paper:
Korber, B., M. Muldoon, J. Theiler, F. Gao, R. Gupta,
A. Lapedes, B. H. Hahn, S. Wolinksy and T. Bhattacharya. 2000. Timing the
ancestor of the HIV-1 pandemic strains. <EM>Science</EM>  <B>288:</B>
1789-1796.  The programs are available from
<A HREF="http://www.santafe.edu/~btk/science-paper/bette.html">
the web site for the programs from that paper</A> at
<CODE>http://www.santafe.edu/~btk/science-paper/bette.html</CODE>.
Some further code (scripts and C code) to fully implement the methods
used for branch length plotting in that paper is available from Korber
on request to individuals who have the book Numerical Recipes and thus
have a license to use code derived from it.
<P>

<A NAME="GRate">
<HR>
<P>
Kai M&uuml;ller</A>
of the Institute for Evolution and Biodiversity of the Westf&auml;lische
Wilhelms-Universit&auml;t M&uuml;nster, M&uuml;nster, Germany
<tt>(kaimueller<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;uni-munster.de)</tt>
has written <strong>GRate</strong>
(Group relative RATE tests),
version 1.0, a program for carrying out relative rate tests of evolution in 
specified groups.  GRate runs <a href="software.pars.html">PAUP*</a> and uses 
the sitewise log likelihoods and bootstrapping that it makes available, and 
analyzes these to carry out relative rate tests that compare the rate of 
evolution in two or more clades selected by the 
user.  It is available as Windows executables.  It can be downloaded from
<a href="http://bioinfweb.info/Software/GRate">its web
site</a>
at <code>http://bioinfweb.info/Software/GRate</code>
<P>

<A NAME="NHML">
<HR>
<P>
Nicolas Galtier and Manolo Gouy</A>
of the Institut des Sciences de l' Evolution
of the Universit&eacute; Montpellier 2 - CNRS, Montpellier, France
and the Laboratoire de Biom&eacute;trie et Biologie Evolutive of the
Universiteacute; Claude Bernard - Lyon, France
<tt>(galtier<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;univ-montp2.fr)</tt>
have produced <strong>NHML</strong>
(<i>N</i>on<i>H</i>omogenous <i>M</i>aximum <i>L</I>ikelihood),
version 3, a maximum likelihood phylogeny program using a nonhomogenous model 
of nucleotide change.  It uses a model developed by them which is not 
reversible and thus allows information about the location of the root of the 
tree.  The program can evaluate a user-defined tree, or can start with a "star 
tree" and then resolve it by Subtree Pruning and Regrafting rearrangements to 
infer the phylogeny.  It can use either a model of rates varying among sites 
or covarion-like model which allows rates at each site to independently vary 
along the branches of the tree.
The methods are described in the papers:
<ul>
<li>Galtier N. and M. Gouy. 1998. Inferring pattern and process:
maximum-likelihood implementation of a nonhomogeneous model of DNA sequence
evolution for phylogenetic analysis. <em>Molecular Biology Evolution</em>
<b>15:</b> 871-879.
<li>Galtier N., N. Tourasse N. and M. Gouy. 1999. A nonhyperthermophilic common
ancestor to extant life forms. <em>Science</em>  <b>283:</b> 220-221.
<li>Galtier N. 2001. Maximum-likelihood phylogenetic analysis under a
covarion-like model. <em>Molecular Biology and Evolution</em> <b>18:</b>
866-873.
</ul>
NHML is available as C source code and Linux executables.  It can be downloaded
from
<a href="http://162.38.181.1/nhml/nhml.html">its web site</a>
at <code>http://162.38.181.1/nhml/nhml.html</code>
<P>

<HR>
<A NAME="TreeTool"><P>
Mike Maciukenas</A>, then at the Department of Microbiology
of the  University
of  Illinois, wrote a wonderful X-windows based interactive tree-plotting
program called <STRONG>TreeTool</STRONG>, version 2.02.  It takes as input a 
PHYLIP  tree file,  with  branch
lengths  if  they  are provided, displays the tree in either rooted or unrooted
form on any X-windows screen, and allows the user to modify  the  form  of  the
tree and the placement of nodes and labels.  When the tree is in final form the
user can have it written to a Postscript or PICT file and/or printed to a
Postscript-compatible  printer.  TreeTool is free as a C program for X windows
using the Xview library.   Xview seems to be available mostly on Sun
workstations (it can also be installed on Linux).  TreeTool is available
It will be found
<A href="http://iubio.bio.indiana.edu/soft/molbio/unix/treetool">in the IUBIO software archive</A>
at <code>http://iubio.bio.indiana.edu/soft/molbio/unix/treetool</code>.
It is also included in the <a href="#GDE">GDE</A> sequence analysis
environment mentioned.  A Debian Linux Treetool package for Intel-compatible
processors is available at
<A HREF="http://packages.debian.org/lenny/treetool">its Debian web
page</a>
at <CODE>http://packages.debian.org/lenny/treetool</CODE>.
It has links to the Debian package for Xview.
<P>
<HR>

<A NAME="TreeView"> <P>
<IMG SRC="icons/treeview.gif" ALT="[TreeView icon here]">&nbsp;
<IMG SRC="icons/treeviewx.gif" ALT="[TreeView X icon here]">&nbsp;&nbsp;
Rod Page </A> of the 
Division of Environmental and Evolutionary Biology of the
University of Glasgow, Scotland 
(<CODE>r.page<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>bio.gla.ac.uk</CODE>), has written
<STRONG>TreeView</STRONG> version 1.6.6, a program for displaying and
manipulating trees.
It can draw rooted and unrooted trees, display bootstrap
values, and edit trees by moving branches, collapsing them, and rerooting.
The program reads NEXUS, <A HREF="software.pars.html#PHYLIP">PHYLIP</A>, and <A
HREF="software.pars.html#Hennig86">Hennig86</A> style tree files
(including files produced by <A HREF="#fastDNAml">fastDNAml</A> and <A
HREF="#ClustalW">CLUSTALW</A>), and can save trees in
the same formats so that it can convert trees among these formats.  The Mac and
Windows versions have almost identical interfaces.  They can
support the standard fonts available on Macs and PCs,
and they support the standard PICT and Windows Metafile formats for output,
allowing
tree pictures to be copied into other applications, as well as being saved in
files.
There is a print preview and drag-and-drop facilities.  Currently TreeView can
read trees with up to 1000 taxa.  The program is free, and can be
obtained
<A HREF="http://taxonomy.zoology.gla.ac.uk/rod/treeview.html">from its web
page</A> from <CODE>http://taxonomy.zoology.gla.ac.uk/rod/treeview.html</CODE>.
It comes in Mac OS and Windows executable versions.  There is also
a Unix/Linux version, TreeView X, which does not yet implement all features of 
the program.
Source code is available at
<a href="http://code.google.com/p/treeviewx/">its page at Google Code</a>
at <code>http://code.google.com/p/treeviewx/</code>its page at Google Code</a>.
There is also <A HREF="http://taxonomy.zoology.gla.ac.uk/rod/treeview/help/contents.html">online help</A> including an online manual.
<P>

<HR>
<A NAME="NJplot"><P>
<IMG SRC="icons/NJplot.gif" ALT="[NJplot icon]">&nbsp;
Manolo Gouy </A> of the
Laboratoire de Biom&egrave;trie et Biologie Evolutive of the
University of Lyon, France
(<CODE>mgouy<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>biomserv.univ-lyon1.fr</CODE>), has
produced
<STRONG> NJplot</STRONG>, version 2.3-2,
which plots rooted phylogenies (input in the standard form)
and saves the plots as Postscript (for Mac OS, PICT) files.  It displays
branch lengths and bootstrap information (if present) and allows the user
to swap branches and change the position of the root.
It is described in the paper
Perri&egrave;re, G. and M. Gouy 1996. WWW-Query: an on-line retrieval system
for
biological sequence banks. <EM>Biochimie</EM> <B>78:</B> 364-369.
It is available free as executables for Mac OS X, Mac OS, Windows,
SunOS, Sun Solaris, Linux and DEC/Compaq/HP Alpha.
It can be retrieved using
<A HREF="http://pbil.univ-lyon1.fr/software/njplot.html">its web page</A>
at <CODE>http://pbil.univ-lyon1.fr/software/njplot.html</CODE>.
A Debian Linux package is available from
<A HREF="http://packages.debian.org/unstable/science/njplot">its web page</A>
at <CODE>http://packages.debian.org/unstable/science/njplot</CODE>.
It has links to a number of other Debian packages that are needed to
run it, including Lesstif and the NCBI Vibrant toolkit.  It is
also available in <a href="software.serv.html#PasteurServ">a server</a>
at the Institut Pasteur in Paris.
<P>

<HR>
<A NAME="unrooted"> <P>
<IMG SRC="icons/NJplot.gif" ALT="[unrooted icon]">&nbsp;
Manolo Gouy </A> of the
Laboratoire de Biom&egrave;trie et Biologie Evolutive of the
University of Lyon, France (<CODE>mgouy<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>biomserv.univ-lyon1.fr</CODE>), has
written
<STRONG>unrooted</STRONG>,
which draws unrooted phylogenies
and saves them in Postscript files (for Macintosh, PICT files).
It is available free as executables for Mac OS X, Mac OS,
Windows, SunOS, Sun Solaris, SGI, IBM Unix, Linux and
DEC/Compaq/HP Alpha.  It can be retrieved using
<A HREF="http://pbil.univ-lyon1.fr/software/unrooted.html">its web page</A>
at <CODE>http://pbil.univ-lyon1.fr/software/unrooted.html</CODE>.
<P>

<HR>
<A NAME="DendroMaker"><P>
<IMG SRC="icons/dmicon.gif" ALT="[DendroMaker icon]">&nbsp;
Tadashi Imanishi</A> (<tt>imanishi<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>jbirc.aist.go.jp</tt>)
of the Biological Information Research Center (BIRC) of the National Institute
of Advanced Industrial Science and Technology (AIST), Tokyo, Japan
has produced <STRONG>DendroMaker</STRONG>, version 4.1, Mac OS programs which
can draw trees on the screen and print them in Postscript files or MacPaint
files.  They read
the trees in from tree files produced by the Neighbor-Joining or UPGMA options
of the <A HREF="software.etc1.html#ODEN">Oden</A>
package, and can also read standard Newick-format tree files.
They can them edit them and reroot them in a variety of ways.  They can produce
PICT files of the trees.  DendroMaker is distributed 
as Mac OS PowerMac or 68k Mac executables (each
available in both English and Japanese versions) from its
<A HREF="http://www.cib.nig.ac.jp/dda/timanish/dendromaker/home.html">web
page</A>
at <CODE>http://www.cib.nig.ac.jp/dda/timanish/dendromaker/home.html</CODE>.
<P>

<HR>
<A NAME="TreeDrawDeck"><P>
<IMG SRC="icons/treedrawdeck.gif" ALT="[Tree Draw Deck icon]">&nbsp;
Don Gilbert</A> (<CODE>gilbertd<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>bio.indiana.edu</CODE>) 
of the Department of Biology of the University of Indiana, Bloomington,
Indiana, has written
<STRONG>Tree Draw Deck</STRONG>, a Hypercard deck that draws trees on the
screen from standard Newick-format tree files.  The deck, which runs on
Macintoshes or PowerMacs that have Hypercard Player, is based on
two programs, Drawtree and Drawgram, from the 3.3 version of
<A HREF="software.pars.html#PHYLIP">PHYLIP</A>.  It allows mouse-driven
interactive display
of the trees with selection of options for display, and allows the
resulting plot to to cut into the system clipboard and pasted into
drawing programs such as MacDraw or Canvas.  Its printing and file-saving
options do not at present work, but the clipboard method works.
It is available
<A HREF="ftp://ftp.bio.indiana.edu/molbio/mac">by anonymous ftp</A> from
<CODE>ftp.bio.indiana.edu</CODE> in
directory <CODE>molbio/mac</CODE>.  It is also available by ftp
<A HREF="ftp://ftp.ebi.ac.uk/pub/software/mac">from</A>
<CODE>ftp.ebi.ac.uk</CODE> in directory <CODE>pub/software/mac</CODE>.
<P>

<HR>
<A NAME="Phylodendron"><P>
<IMG SRC="icons/phylodendron.gif" ALT="[Phylodendron icon]">&nbsp;
Don Gilbert</A> (<CODE>gilbertd<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>bio.indiana.edu</CODE>)
of the Department of Biology of the University of Indiana, Bloomington,
Indiana, has written
<STRONG>Phylodendron</STRONG> version 0.8d beta, a Java application for drawing
phylogenetic trees.  It will read tree data in standard Newick
format, then display graphical views of the phylogenetic tree.  Various
options allow you to modify, adorn and edit the tree.   Standard
application functions to save, print, edit and manage preferences are
included.   This program will not estimate nor produce the tree data. 
Phylodendron is written as a Java application.  It is also
distributed with some extra parts for Mac OS.
Phylodendron is an enhancement of the Mac Hypercard program <A
HREF="#TreeDrawDeck">Tree
Draw Deck</A> which was released by Gilbert in 1990, and uses tree drawing
algorithms from <A HREF="software.pars.html#PHYLIP">PHYLIP</A>.
Phylodendron is available from its
<A HREF="http://iubio.bio.indiana.edu/soft/molbio/java/apps/trees/">Web
site</A>
<CODE>http://iubio.bio.indiana.edu/soft/molbio/java/apps/trees/</CODE>
or <A HREF="ftp://iubio.bio.indiana.edu/molbio/java/apps/trees/">by anonymous
ftp</A> from
<CODE>iubio.bio.indiana.edu</CODE> in directory
<CODE>molbio/java/apps/trees/</CODE>.
Source code is also available there.
Phylodendron is also made available as a <a
href="software.serv.html#Phylodendron-server">web server</a>.
<P>

<HR>
<A NAME="PhyloTreeDraw"><P>
Julian Gough (<tt>gough<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>cs.bris.ac.uk</tt>) of the 
Department of Computer Science at the University of Bristol, Bristol, U.K.
has produced a <strong>Phylogenetic Tree Drawing</strong> Perl module that
reads a tree and displays a rooted tree diagram.  The user can specify
width of the diagram and its line thickness, and can make each line
consist of two shadings of different width, and add extra labels to tips.
The routine is in Perl and is available
<a href="http://www.cs.bris.ac.uk/~gough/software/">at its web
site</a>
at <code>http://www.cs.bris.ac.uk/~gough/software/</code>.  Selecting
the "Perl module" link there displays the Perl source on
the browser, and you can then save it to disk.
It is also available at <a
href="http://supfam.mrc-lmb.cam.ac.uk/treedraw/tree.html">his previous web site for it</a> at
<code>http://supfam.mrc-lmb.cam.ac.uk/treedraw/tree.html</code>
<P>

<HR>
<A NAME="TreeJuxtaposer"><P>
<IMG SRC="icons/tj.gif">&nbsp; Tamara Munzer</A> (<tt>tmm<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>cs.ubc.ca</tt>)
and her colleagues at the Graphics, Visualization and Human-Compuyer-Interface
group at the Department of Computer Science of the University of British
Columbia, Vancouver, Canada have produced <strong>TreeJuxtaposer</strong>
version 2.1.  This displays a tree or shows two trees, with the locations
of differences highlighted.  Parts of the trees collapse and expand
as one moves through the trees, so that very large trees can be examined.
TreeJuxtaposer is a Java application which works with GL4Java, the
Java implementation of the OpenGL graphics environment.  TreeJuxtaposer
is described in a paper for the SIGGRAPH 2003 conference:
Munzner, T., F. Guimbretiere, S. Tasiran, L. Zhang, and Y. Zhou. 2003.
TreeJuxtaposer: Scalable tree comparison using focus+context with guaranteed
visibility. <em>ACM Transactions on Graphics</em> <b>22:</b> 453-462.
It was an overall winner in the IEEE Symposium on Information Visualization
InfoVis 2003 Contest on "Visualization and Pair Wise Comparison of Trees".
It can be downloaded from <a
href="http://olduvai.sourceforge.net/tj/index.shtml">its web site</a> at
<code>http://olduvai.sourceforge.net/tj/index.shtml</code>
<P>

<HR>
<A NAME="Archaeopteryx"><P>Christian Zmasek</A>, currently of the Genomics Institute
of the Novartis Research Foundation in San Diego, California
(<CODE>phylosoft<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>gmail.com</CODE>)
has written <strong>Archaeopteryx</strong>, version 0.971&beta;9m, a software
tool for the visualization, analysis, and editing of potentially large and
highly annotated phylogenetic trees.  It is the successor to ATV, a tree viewer
that is part of the <a href="#FORESTER">FORESTER</a> package.  
Archaeopteryx can read trees from a variety of tree file formats.
It can do a number of different shapes of tree display, can have
support values on the tree, allows different colors and display of various
kinds of data such as photographs of the organisms.  It can also display
internal node data by mouse rollover as you move the mouse around the tree.
Archaeopteryx is written in Java, and can be run on Sun Java version 5 and
later.  That version of Java is already installed in recent Windows and Mac OS
X systems, and can also be installed by the user on Linux systems.
Archaeopteryx is available
<a href="http://www.phylosoft.org/archaeopteryx/">at its web site</a> at
<code>http://www.phylosoft.org/archaeopteryx/</code>.  There are also available
there versions that can also be installed as an applet on Java-enabled browsers.
<P>

<HR>
<A NAME="mavric"><P>
<IMG SRC="icons/mavric.gif" ALT="[Mavric]">&nbsp;&nbsp;Richard H. Ree</A> of
the Botany Department of the Field Museum of Natural
History, Chicago, Illinois (<TT>rree<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>fieldmuseum.org</TT>) has written
<STRONG>Mavric</STRONG> (recursively, <i>MA</i>vric <i>V</i>isualizes
<i>RI</i>ck's <i>C</i>ladograms),
version 0.8.3,
a package in the Python language for the manipulation and visualization of
phylogenetic data. It is intended to be flexible and easily customizable to
suit the needs of phylogenetic biologists who use python. 
Mavric is named for its core application Mavric, which aims to provide a
graphical
interface to phylogenies in a manner similar to MacClade. (Rick notes that
&ldquo;the recursion
that defines the name Mavric 
ultimately boils down to its central application: a tool to Manipulate And
Visualize RIck's
Cladograms -- if you followed all this you're pretty far ahead in
understanding recursion&rdquo;).  Currently, its main usefulness is to view and
manipulate
phylogenetic trees. Branches can be moved around, pruned, rotated, have lengths
changed.  Trees can be grafted together, and viewed either rooted or
unrooted.  Postscript tree plots can be produced.
Mavric is available as Python source code from
<A HREF="http://www.bioinformatics.org/mavric/index.html">its web site</A> at
<CODE>http://www.bioinformatics.org/mavric/index.html</CODE>.
<P>

<HR>
<A NAME="TreeEdit"><P>
<IMG SRC="icons/treedit.gif" ALT="[TreeEdit icon]">&nbsp;
Andrew Rambaut, of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
and Mike Charleston (<CODE>mcharles<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>it.usyd.edu.au</CODE>)
of the Sydney University Biological Informatics and Technology Centre,
Sydney, Australia
have produced the <STRONG>TreeEdit</STRONG> Phylogenetic Tree
Editor version 1.0a10.  TreeEdit is an application for organizing,
manipulating and viewing sets of trees.  It is intended as a tool for
preparing sets of trees for use in phylogeny packages.  It can read and write
trees in standard formats (<A HREF="software.pars.html#PHYLIP">PHYLIP</A>,
NEXUS, and <A HREF="software.etc2.html#CAIC">CAIC</A>).  It allows
drag-and-drop editing of branches, cut and paste of trees between windows
(as NEXUS text), rerooting by clicking on branches, editing species labels,
rotating branch order at polytomies, including and excluding species,
and transforming branch lengths (including Sanderson's non-parametric rates
smoothing) and many other interactive features.
TreeEdit is available as an executable for Mac OS 8 or later at
<A HREF="http://tree.bio.ed.ac.uk/software/treeedit/">its web
site</A> at
<CODE>http://tree.bio.ed.ac.uk/software/treeedit/</CODE>
<P>

<HR>
<A NAME="TreeThief"><P>
<IMG SRC="icons/treethief.gif" ALT="[TreeThief icon]">&nbsp;
Andrew Rambaut, of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
wrote <STRONG>TreeThief</STRONG> version 1.0, a program for inputting
phylogenetic trees, with branch lengths, into the computer for use in other
programs.  The tree is input by clicking on each node in turn. A scanned image
of
a tree can be loaded first to act as a template.  TreeThief is distributed
from <A HREF="http://microbe.bio.indiana.edu:7131/soft/iubionew/molbio/evolution/phylo/TreeThief/">its web page</A>
at <CODE>http://microbe.bio.indiana.edu:7131/soft/iubionew/molbio/evolution/phylo/TreeThief/</CODE> as
a Mac OS executable.  It
requires a Macintosh running Mac OS 8 or later.
<P>

<A NAME="TreeSnatcher">
<HR>
<P>
Thomas Laubach and Arndt von Haeseler</A>
of the Bioinformatics Institute of the University of Duesseldorf, Germany
<tt>(thomas.laubach<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;uni-duesseldorf.de)</tt>
and the Center of Integrative BioInformatics Vienna
at the University of Vienna, Austria
<tt>(arndt.von.haeseler<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;univie.ac.at)</tt>
have produced <strong>TreeSnatcher Plus</strong>,
a program to capture phylogenies from images.  TreeSnatcher Plus is a 
GUI-driven 
Java program for the semi-automatic recognition of multifurcating phylogenetic 
trees in pixel images. The program accepts an image file as input and analyzes 
the topology and the metrics of a tree depicted, with user assistance. The 
analysis is carried out in a multiple-stage process using basic algorithms 
from the field of image analysis. It yields a tree in the standard Newick 
format that represents the tree structure, optionally including branch 
lengths. TreeSnatcher Plus can process trees with more than 100 leaves or more 
in a few seconds.   It also allows the user to construct their own trees and
to modify existing trees.  An earlier version, Treesnatcher 1.0, is described 
in the paper:
Laubach, T. and A. von Haeseler. 2007. TreeSnatcher: Coding trees from images.
<em>Bioinformatics</em> <b>23:</b> 3384-3385.
It is available as Java executables, packaged for Windows, for Linux, and
for Mac OS X.  Its web site is available at two locations,
<a href="http://www.cs.uni-duesseldorf.de/AG/BI/Software/treesnatcher/">at the
University of D&uuml;sseldorf</a> at
<code>http://www.cs.uni-duesseldorf.de/AG/BI/Software/treesnatcher/</code>
and <a href="http://www.cibiv.at/software/treesnatcher/">at the University of
Vienna</a> at <code>http://www.cibiv.at/software/treesnatcher/</code>.  The
same versions are available at both sites.
<P>

<HR>
<P>
<IMG SRC="icons/TreeExplorer.gif" ALT="[TreeExplorer Icon]">
<A NAME="TreeExplorer">Koichiro Tamura</A> of the Evolutionary
Genetics Laboratory, Tokyo Metropolitan University,
Tokyo, Japan (<TT>koichiro-tamura<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>c.metro-u.ac.jp</TT>) has written
<STRONG>TreeExplorer</STRONG> version 2.12,
a program for plotting, rearranging, and editing trees, including a
feature to compress a subtree to make essential features more visible.
TreeExplorer is intended to work with <A
HREF="software.pars.html#MEGA">MEGA</A> but can also
be used to read and edit or view trees in other standard formats.
It is distributed as a Windows executable from
<A HREF="http://evolgen.biol.metro-u.ac.jp/TE/TE_man.html">its web site</A>
at <CODE>http://evolgen.biol.metro-u.ac.jp/TE/TE_man.html</CODE> where a
manual for the program and a download link will be found.
<P>

<HR>
<a name="TreeSetViz"><P><IMG SRC="icons/mesquite.gif" ALT="[Mesquite icon]">
A (large) group headed by Katherine St. John at Lehman College, City University
of New York (<tt>stjohn<! death to spam>&nbsp;(at)<! death to
spam>&nbsp;lehman.cuny.edu</tt>), David Hillis and Tandy Warnow of the
University
of Texas, Austin (<tt>DHillis<! death to spam>&nbsp;(at)&nbsp;<! death to
spam>mail.utexas.edu</tt> and <tt>tandy<! death to spam>&nbsp;(at)&nbsp;<!
death to spam>cs.utexas.edu</tt>), and including numerous other people, has
released
<strong>Tree Set Viz</strong>, a program in Java for visualizing distances
between sets of trees in a space.    It computes distances between trees
and uses these to infer points in a two-dimensional space that most closely
approximate those distances.  It is to be used in the
<a href="software.pars.html#mesquite">Mesquite</a> Java project for interacting
with trees.  Tree Set Viz is described in a paper: Hillis, D. M., T. A. Heath,
and K. St. John. 2005.  Analysis and visualization of tree space.
<em>Systematic Biology</em> <b>54:</b> 471-482.  It is available from
<a href="http://comet.lehman.cuny.edu/treeviz/software.html">its web site</a> at
<code>http://comet.lehman.cuny.edu/treeviz/software.html</code>
<P>

<HR>
<A NAME="Treeme"><P><IMG SRC="icons/Treeme.gif" ALT="[Treeme icon]">&nbsp;
Dominik Hepperle</A> of Sequentix, Dorfstrasse 20, D-18249 Klein Raden, Germany
(<tt>info<! death to spam>&nbsp;(at)<! death to spam>&nbsp;do-mix.de</tt>) has
released <strong>TreeMe</strong>, a program for visualisation and annotation
of phylogenies.  It can
read and write trees, allows many kinds of annotation of nodes, rearrangement
and editing of the tree, addition of multiple bootstrap values, collapsing
of nodes, and more.  It is available as a Windows executable from
<a href="http://www.sequentix.de/software_treeme.php">its web site</a> in the company
web sites at <code>http://science.sequentix.de/software_treeme.php</code>.
It is purchased for 499 Euros for the first license, 199 Euros for each
additional license, and 149 Euros for upgrade licenses.
<P>

<HR>
<A NAME="TreeGraph"><P>
<IMG SRC="icons/TreeGraph2.gif" ALT="[TreeGraph 2 icon]">&nbsp;&nbsp; J&ouml;rn M&uuml;ller</A>, of Pipinstra&szlig;e 2,
Bonn,
Germany, Ben St&ouml;ver of the
Nees-Institut f&uuml;r Biodiversit&auml;t der Pflanzen,
Rheinische Friedrich-Wilhelms-Universit&auml;t Bonn, Germany
(<tt>stoever<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;bioinfweb.info</tt>)
and Kai M&uuml;ller, of the group on Evolution and Biodiversity of Plants at
Westf&auml;lische Wilhelms-Universit&auml;t M&uuml;nster, Germany
(<tt>kaimueller<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;uni-muenster.de</tt>)
have written <strong>TreeGraph 2</strong> (version 2.0.45-197 beta),
an editor for phylogenetic trees. It allows automatically combining 
information from different phylogenetic analyses and helps to identify and 
graphically present incongruencies.  It features editing and formatting 
options, such as automatically setting line widths or colors according to the 
value of any of the unlimited number of variables that can be assigned to each 
node or branch. These node/branch data can be imported from spread sheets or 
other trees, be calculated from each other, be kept invisible or set visible 
and then be freely formatted.  Whole clades can be copied from other files and 
be inserted, and can also be manually added.
It can read NEXUS or Newick tree files, or its own TGF format, which uses a
LaTeX-style format that is like those formats, but extended to
indicate support values for nodes as well as
annotations for groups of tips or clades.  The program produces EPS,
SVG, PDF or PNG  graphics file formats.
It is described in the papers: 
<UL>
<LI> St&ouml;ver, B. C., and K. F. M&uuml;ller. 2010. TreeGraph 2: Combining
and visualizing evidence from different phylogenetic analyses.  <i>BMC
Bioinformatics</i> <b>11:</b> 7.
<LI> M&uuml;ller, J. and K. M&uuml;ller. 2004. TreeGraph: automated drawing of
complex
tree figures using an extensible tree description format.
<em>Molecular Ecology Notes</em> <b>4:</b> 786-788.
</UL>
TreeGraph 2 is Java code, available both as Java executables and as Java source
code at
<a href="http://treegraph.bioinfweb.info/">its web page</a>
at <code>http://treegraph.bioinfweb.info/</code>.  An earlier version,
TreeGraph version 1.0 rc4, is available as source code for Unix or Linux,
and as a Windows executable.  It is available from <a href="http://www.math.uni-bonn.de/people/jmueller/extra/treegraph/">its web site</a> at <code>http://www.math.uni-bonn.de/people/jmueller/extra/treegraph/</code>.
<P>

<A NAME="ArboDraw">
<HR>
<P>
<IMG SRC="icons/arbodraw.gif">&nbsp;&nbsp;
Adrian Canutescu and Roland Dunbrack</A> of the Basic Science Division at the
Fox Chase Cancer Center
<tt>(Roland.Dunbrack<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;fccc.edu)</tt>
have released <strong>ArboDraw</strong>
version 1.0, a Windows program for creating, rendering, coloring, annotating,
and saving images of phylogenetic trees.
ArboDraw (from Latin, <em>arbor</em>, tree) is a Windows program for creating
publication-quality images of phylogenetic trees. ArboDraw provides
a combination of features: 
<OL>
<LI> editing of the rendered tree, including independent horizontal and
vertical scaling, full font control, and background and tree coloring;
<LI> branch selection and coloring so that separate parts of the tree can be
colored differently; 
<LI> annotation of sequences from the input FASTA header lines available on
mouseover; labels and annotations can be edited as desired; 
<LI> control of saving the image in png format at desired resolution
for publication; 
<LI> saving of the image in a novel XML format so that editing can be
re-started from a previous editing session; 
<LI> running a fast and accurate sequence alignment program,
<a href="software.etc1#MUSCLE">MUSCLE</a>, for generating trees;
A MUSCLE executable is included with ArboDraw.
<LI> alternatively allowing user input tree data in Newick format.
</OL>
It is available as Windows executables which require the .Net 2.0 framework
(freely downloadable from Microsoft).  Arbodraw itself can be downloaded from <a
href="http://dunbrack.fccc.edu/ArboDraw">its web site</a> at
<code>http://dunbrack.fccc.edu/ArboDraw</code>
<P>

<A NAME="TreeDyn">
<HR>
<P>
Fran&ccedil;ois Chevenet and colleagues,</A>
of the Laboratoire de G&eacute;n&eacute;tique et Evolution des Maladies
Infectieuses
of the Institut de recherche pour le d&eacute;veloppement, UMR CNRS/IRD 2724,
Montpellier, France
<tt>(f.chevenet<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ird.fr)</tt>
have written <strong>TreeDyn</strong>
version 198.3, modules for dynamic graphics and annotation of trees.
TreeDyn can import trees, and place annotations on their nodes, branches, and
clades.  It can compare trees and find parts of trees which are similar to a
highlighted region on a target tree.  Operations in TreDyn can be scripted. It
can handle very large trees and can collapse regions on those trees to
concentrate on certain parts of the tree. TreDyn can handle multiple windows
and multiple trees per window, can save and restore graphical analysis, can
project texts or symbols facing leaf labels or linked to nodes.  It includes a
library of packages dedicated to specific research fields involving trees.
It is described in the paper:
Chevenet, F., C. Brun, A. L. Banuls, B. Jacq and R. Christen.  2006. TreeDyn:
towards dynamic graphics and annotations for analyses of trees. <em>BMC
Bioinformatics</em> <b>7:</b> 439. 
It is available as source code in the graphics-oriented scripting language
Tcl/Tk. There are separate distributions for Windows, for Linux, and for
Mac OS X, for systems that already have Tcl/Tk installed.  It can be
downloaded from
<a href="http://www.treedyn.org/">its web site</a>
 at  <code>http://www.treedyn.org/</code>
<P>

<A NAME="DigTree">
<HR>
<P>
Robin Kramer</A>, then of
the Digital Biology Laboratory, Computer Science Department
of the University of Missouri, Columbia, Missouri
(whose email address was at that time: <tt>kramer<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;mizzou.edu)</tt>
has produced <strong>DigTree</strong>
(Digital Tree).  DigTree is a tool that allows biologist to display large
heirarchical trees in ways that preserve weights such as branch lengths.  It
can also display other sorts of trees.  It is described in the paper:
Kramer, R., V. Olman, Y. Xu, and D. Xu. 2006. DigitalTree: A tool for displaying
biological data in tree structure. <i>Lecture Notes in Computer Science</i>
vol. 3992, pp. 855-862.
It is available as C++ source code and Windows executables.
To compile it, one also needs the spirit++ framework in addition to
a C++ compiler.
TreeDyn can be downloaded from
<a href="http://digbio.missouri.edu/~digtree/">its web site</a>
 at  <code>http://digbio.missouri.edu/~digtree/</code>
<P>

<A NAME="DTdraw">
<HR>
<P>
Olivier Elemento</A>, then
of the LIRMM (Laboratoire d'Informatique de Robotique et de
Micro&eacute;lectronique de Montpellier)
at the Universit&eacute; de Monpellier II, Montpellier, France
and now of the Institute for Computational Biomedicine at Weill Cornell
Medical College, New York Citys
(<TT>ole2001<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;med.cornell.edu</TT>),
has released <strong>DTdraw</strong> (Duplication Tree Draw),
a program to draw trees of gene duplication histories.  a command line tool to 
draw tandem duplication histories.  Tandem duplication trees can be 
reconstructed with the <a href="software.etc1.html#DTscore">DTscore</a>
program. DTdraw takes as input a (rooted) 
duplication history in Newick format, and an order file, i.e. a file which 
contains  the number of leaves and the leaf labels of the tree, in correct 
order. At the moment, DTdraw outputs drawings in two graphical formats : PNG 
and Postscript.
It is available as C source code and Linux executables.  It can be downloaded
from
<a href="http://www.lirmm.fr/~elemento/DTdraw/">its web site</a>
at <code>http://www.lirmm.fr/~elemento/DTdraw/</code>
<P>

<A NAME="TreeMaker">
<HR>
<P>
<IMG SRC="icons/treemaker.gif">&nbsp;&nbsp; Paul Michael Agapow</A>, of the
Department of Biology of Imperial College at Silwood Park, Ascot, Berks U.K.
<tt>(agapow<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;agapow.net)</tt>
has written <strong>TreeMaker</strong>, version 1.3, a program for
interactive construction of taxonomies with species richness data.
It allows the user to create a taxonomy and write out a tree that has
species richness information on it.  It is described in the paper:
Crozier, R. H., L. J. Dunnett, P.-M. Agapow.  2005. Phylogenetic biodiversity
assessment based on systematic nomenclature. <em>Evolutionary Bioinformatics
Online</em> <b>1:</b> 11-36.
It is available as C source code, Windows executables and Mac OS X executables.
It can be downloaded from
<a href="http://www.agapow.net/software/treemaker">its web site</a>
 at  <code>http://www.agapow.net/software/treemaker</code>.  (TreeMaker is
not related to
<a href="http://www.langorigami.com/science/treemaker/treemaker5.php4">this program of the same name</a>)
<P>

<A NAME="Paloverde">
<HR>
<P>
Mike Sanderson</A>
of the Department of Ecology and Evolutionary Biology
at the University of Arizona, Tucson, Arizona
<tt>(sanderm<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;email.arizona.edu)</tt>
has written <strong>Paloverde</strong>
version 1.1,
a program to display a phylogeny in one of several 3-dimensional formats.
Paloverde is best used for trees of 100 to 2,500 tips.  It reads the tree from
a NEXUS tree file and can display it in a number of ways.  These include a
circular format, on a spiral, a cone, a hemisphere or in a three-dimensional
treelike form.  The user can rotate the object and zoom on it, and clades can
be selected to be plotted.  Subclades can also be collapsed to single tips.
It is available as C source code and Powermac Mac OS X executables.  It can
also be compiled from the source code on Linux systems.  It requires the
free OpenGL graphics library.  It can be downloaded from
<a href="http://loco.biosci.arizona.edu/paloverde/paloverde.html">its web
site</a>
at  <code>http://loco.biosci.arizona.edu/paloverde/paloverde.html</code>
<P>

<A NAME="MrEnt">
<HR>
<P>
<IMG SRC="icons/mrent.gif">&nbsp;&nbsp;Alessandro Zuccon and Dario Zuccon</A>
of the Department of Vertebrate Zoology
at the Naturhistoriska riksmuseet, Stockholm, Sweden
have produced <strong>MrEnt</strong>, version 2.2, a program to produce
drawings of phylogenies.  It is intended for drawing high-quality, printable
phylogenetic trees. It is intended as a user-friendly program to recognize
Nexus tree files and display them in the most commonly used tree formats.
Trees can be displayed as rectangular cladograms, slanted cladograms or
phylograms.  The program allows full editing of taxon names, taxon comments,
support values, and branches, addition of other graphic elements such as lines,
shapes, text, brackets).  Drawings can be exported in Portable document format
(PDF), Windows enhanced metafile (EMF), Windows bitmap (BMP), or Portable
Network Graphics (PNG) formats. It also allows multiple trees, layers and
embedded graphics.  It is available as a Windows executable.
It also requires that the Microsoft .NET framework be installed.
It can be downloaded from
<a href="http://www.mrent.org">its web site</a>
at <code>http://www.mrent.org"</code>
<P>

<A NAME="FigTree">
<HR>
<P>
<IMG SRC="icons/figtree.gif" ALT="[FigTree icon]">&nbsp;&nbsp;Andrew
Rambaut</A>
of the Institute of Evolutionary Biology
of the University of Edinburgh, Edinburgh, Scotland
<tt>(a.rambaut<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ed.ac.uk)</tt>
has produced <strong>FigTree</strong>
version 1.3.1, a program to graphically view phylogenies and prepare
publication-ready figures from them.  FigTree is designed as a graphical viewer
of phylogenetic trees and as a program for producing publication-ready figures.
Andrew notes that it was written for his own needs so may not be as polished
and
feature-complete as a commercial program. In particular it is designed to
display summarized and annotated trees produced by
<a href="software.etc1.html#BEAST">BEAST</A>.
It is available as Java executables and Windows executables. It is available 
from
<a href="http://tree.bio.ed.ac.uk/software/figtree/">its web site </a> at
<code>http://tree.bio.ed.ac.uk/software/figtree/</code>
<P>

<A NAME="HyperTree">
<HR>
<P>
<img src="icons/hypertree.gif" alt="[HyperTree icon]">&nbsp;&nbsp;Jonathan
Bingham,
then</A> of Sugen, Inc. (more recently at Jivan Biologics:
<tt>jbingham<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;jivanbio.com</tt>)
wrote <strong>HyperTree</strong> version 12, a phylogenetic tree viewer, with a 
hyperbolic ('fish-eye') view and editing abilities that help in managing very 
large trees.  HyperTree includes many features for navigating large trees, 
such as the ability to visualize large trees with hundreds of nodes or more, 
to rotate and drag the display in cartesian space, to search and select nodes, 
to copy clusters for pasting into other programs, to color-code branches, to 
zoom in and out, and to label branches (eg, common family members).  It can 
view phylogenetic trees and other hierarchical clusterings, such as gene 
expression profile clusters.  The program subsequently came to be owned by
Pfizer, Inc., which has given permission to the Salk Institute for its
distribution at the <tt>kinase.com</tt> web site.
It is described in the paper:
Bingham, J. and S. Sudarsanam.  2000.  Visualizing large hierarchical clusters
in hyperbolic space
<em>Bioinformatics</em> <b>16(7):</b> 660-661.
It is available as Java executables.  It can be downloaded from
<a href="http://kinase.com/tools/HyperTree.html">its web site</a>
at <code>http://kinase.com/tools/HyperTree.html</code>
<P>

<A NAME="Dendroscope">
<HR>
<P>
<IMG SRC="icons/dendroscope.jpg" ALT="[Dendroscope icon]">&nbsp;&nbsp;Daniel
Huson, with contributions from Tobias Dezulian, Markus Franz, Christian Rausch,
Daniel Richter and Regula Rupp</A>
of the Center for Bioinformatics
of the Tuebingen University, Tuebingen, Germany
<tt>(huson<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;informatik.uni-tuebingen.de)</tt>
has released <strong>Dendroscope 2</strong>,
an interactive program for the visualization and navigation
of phylogenetic trees.
Dendroscope is a new program for the interactive visualization and navigation
of phylogenetic trees. The program provides eight standard tree visualizations 
and is optimized to handle trees containing hundreds of 
thousands of taxa. The program allows trees to be edited as well as
for graphics to be exported.  Version 2 adds the capability of
displaying networks as well as trees.
To support the inspection of large trees, Dendroscope offers a 
magnification tool. It is written in Java and and installers are 
provided for Linux/Unix, MacOS X and Windows.
It is described in the paper:
Huson, D. H., T. Dezulian, M. Franz, C. Rausch, D. C. Richter and R. Rupp.
2007.
Dendroscope: an interactive viewer for large phylogenetic trees, submitted
for publication.
Dendroscope can be downloaded from
<a href="http://www-ab.informatik.uni-tuebingen.de/software/dendroscope">its
web site</a>
at <code>http://www-ab.informatik.uni-tuebingen.de/software/dendroscope</code>
for free, although the authors require that you obtain a free license key 
online and cite their paper in any publication using the program.
<P>

<A NAME="PhyloWidget">
<HR>
<P>
<IMG SRC="icons/pw-icon.png" ALT="[PhyloWidget icon]"> Gregory Jordan</A> of
the Goldman Group of the European Bioinformatics Institute
<tt>(phylowidget<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;gmail.com)</tt> or
<tt>greg<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ebi.ac.uk)</tt>
has written <strong>PhyloWidget</strong>
version 1.0, a web-friendly tool for creating, editing, and visualizing 
trees.  PhyloWidget is a phylogenetic tree visualization program that 
"plays nice" with the Internet. It runs directly from the browser window (as a 
Java applet), and trees can be loaded in manually from the user's hard drive, 
or alternatively through a URL-based API.  They can be displayed and edited,
and publication-quality figures produced.
PhyloWidget is aimed at (1) users who want a simple, easy-to-use tree 
visualization tool without having to download software, and (2) phylogenetic 
tree databases who wish to use the URL API to let PhyloWidget visualize the 
database's trees.  It is described in the paper:
Jordan, G. E. and W. P. Piel. 2008.  PhyloWidget: Web-based
visualizations for the
tree of life.  <em>Bioinformatics</em> <b>24:</b> 1641-1642.
It is available as Java source code and Java executables, and a browser-based
Java applet.  It can be downloaded from
<a href="http://www.phylowidget.org/">its web site</a>
at <code>http://www.phylowidget.org/</code>
<P>

<A NAME="TreeToy">
<HR>
<P>
John Brzustowski</A>
of the Department of Biological Sciences
of the University of Alberta, Edmonton, Canada
<tt>(jbrzusto<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ualberta.ca)</tt>
has written <strong>TreeToy</strong>
, a program to view a tree and select subtrees from it.  TreeToy reads a tree
from a file and displays it.  The user can reroot the tree or select subtrees
to write out.  It can also be zoomed and written out in a PICT2 format for
printing.
It is available as Java source code and Java executables.  It can be downloaded
from
<a href="http://www2.biology.ualberta.ca/jbrzusto/TreeToy.php">its web site</a>
at <code>http://www2.biology.ualberta.ca/jbrzusto/TreeToy.php</code>
<P>

<A NAME="Phyutility">
<HR>
<P>
<IMG SRC="icons/phyutility.jpg" ALT="[Phyutility icon]">&nbsp;&nbsp;
Stephen A. Smith and Casey W. Dunn</A>, of the Department of Ecology and
Evolutionary Biology of Brown University, Providence, Rhode Island
(<tt>stephen_a_smith<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;brown.edu</tt>)
have produced <strong>Phyutility</strong>
version 2.2, a utility to modify trees and to perform certain data set 
modifications as well.  Phyutility's tree operations include rerooting, 
pruning, type conversion, consensus trees, measures of leaf stability, of the 
varying attachment locations of lineages, and measures of tree support.  It 
can also do some data operations such as concatenating alignments, parsing 
Genbank files, trimming alignments, and searching for and fetching sequences 
from the Genbank database.  It is described in the paper:
Smith, S. A. and C. W. Dunn. 2008. Phyutility: a phyloinformatics tool for 
trees, alignments, and molecular data. <em>Bioinformatics</em> <b>24:</b> 
715-716.
It is available as Java source code and Java executables.
It needs two additional Java libraries downloadable from elsewhere.
It can be downloaded from
<a href="http://code.google.com/p/phyutility/">its web site</a>
at <code>http://code.google.com/p/phyutility/</code>
<P>

<A NAME="SuiteMSA">
<HR>
<P>
Catherine Anderson</A>
of the Department of Computer Science
of the University of Nebraska, Lincoln
<tt>(cateander27<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;gmail.com)</tt>
has released <strong>SuiteMSA</strong>
(A suite of tools for visual comparision of Multiple Sequence Alignments),
version 1.3.14, Contains tools that allow the visual and quantitative
comparison between multiple sequence alignments using sequence sets as well as structural information. .  This package contains tools that allow the visual and quantitative comparison between MSA using sequence sets as well as structural information. Various color scheme are available including color schemes based on structural information. The Pixel Plot tool allows the display of multiple large alignment in a pixelated display that gives the overall shape of the alignments as well as column wise comparison. 
Included in this package is a sequence simulator, iSGv2.  It can also act as
a GUI and call the alignment
programs <a href="software.etc1.html#ClustalW">ClustalW</a> and <a
href="software.etc1.html#MUSCLE">MUSCLE</a>, and it also has 
as a phylogeny display.  It is described in the paper:
Anderson, C. L., C. L. Strope and E. N. Moriyama. 2011.
SuiteMSA: Visual tools for multiple sequence alignment comparison and molecular
sequence simulation.  <i>BMC Bioinformatics</i> <b>12:</b> 184.
It is available as Perl script, Java executables and Intel Mac OS X executables.  It can be downloaded from
<a href="http://bioinfolab.unl.edu/~canderson/SuiteMSA/">its web site</a>
at <code>http://bioinfolab.unl.edu/~canderson/SuiteMSA/</code>
<P>

<HR>
<P>
David Kidd and Xianhau Liu</A>
of the National Evolutionary Synthesis Center
at Duke University, Durham, North Carolina
<tt>(dk<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;nescent.org)</tt>
have released <strong>GeoPhyloBuilder</strong>
version 1.0.2, an extension for the widely-used geographic information system
program ArcGIS that creates geophylogenies.  Geophylogenies are spatial
networks in which network topology is defined by a  phylogenetic model and the 
geographical position of nodes and branch paths are derived from a 
biogeographical model. Tree depths are assigned to the z-dimension of the 
spatial network facilitating rapid 3D visualization in GIS or other software. 
Geophylogenies can be output in ArcGeodatabase, shapefile and KML (Google 
Earth) format. A copy of ArcGIS is required.
GeoPhyloBuilder is described in the paper:
Kidd, D. M. and X. Liu. 2007. geophylobuilder 1.0: an ArcGIS extension for
creating 'geophylogenies'. <em>Molecular Ecology Notes</em> <b>8 (1):</b> 88-91.
It is available as Windows executables and source code for the Microsoft C++
.NET compiler.  It can be downloaded from
<a href="http://www.nescent.org/informatics/software.php">its web site</a>
at <code>http://www.nescent.org/informatics/software.php</code>
<P>

<A NAME="GenGIS">
<HR>
<P>
<IMG SRC="icons/gengis.gif" ALT="[GenGIS icon]">&nbsp;&nbsp;
Donovan Parks and Robert Beiko</A> of the Faculty of Computer Science
at  Dalhousie University, Halifax, Nova Scotia, Canada
<tt>(beiko<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;cs.dal.ca)</tt>
have released <strong>GenGIS</strong> version 1.08, a Geospatial Information 
System for Genetic Data.  GenGIS allows the integration of digital map data 
alongside information about sequences and sample locations. A wide range of 
data visualizations are supported, and phylogenetic trees or cluster 
relationships can be drawn in two or three dimensions. A Python interface 
provides extensibility and camera controls, while the R statistical package 
allows exposed data to be subjected to statistical analysis.
It is described in the paper:
Parks, D. H., M. Porter, S. Churcher, S. Wang, C. Blouin, J. Whalley, S. Brooks,
and R. G. Beiko.  2009. GenGIS: A geospatial information system for genetic 
data. <i>Genome Research</i>  <b>19:</b> 1896-1904.
It is available as C++ source code, Windows executables and Mac OS X universal 
executables.  It can be downloaded from
<a href="http://kiwi.cs.dal.ca/GenGIS/Main_Page">its web site</a>
at <code>http://kiwi.cs.dal.ca/GenGIS/Main_Page</code>
<P>

<A NAME="Lagrange">
<HR>
<P>
Richard H. Ree and Stephen A. Smith</A> of the Department of Botany
of the Field Museum of Natural History
<tt>(rree<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;fieldmuseum.org)</tt>
has released <strong>Lagrange</strong>
(Likelihood analysis of geographic range evolution),
version 2.0.1, likelihood models for geographic range evolution on 
phylogenetic trees, with methods for inferring rates of dispersal and local 
extinction and ancestral ranges.  Lagrange (likelihood analysis of geographic 
range evolution) is a freely available Python package, co-authored by 
Ree and Smith, implementing dispersal-extinction-cladogenesis models of 
geographic range evolution, methods described in papers by Ree et al.
(<i>Evolution</i>, 2005) and Ree and Smith, 2008.  It can be used to infer 
geographic ancestry (historical ranges) and estimate rates of dispersal and 
local extinction on phylogenetic trees that have species range data arrayed at 
their tips.  A web server that is an analysis configuration tool that prepares 
files to be run on your copy of Lagtange is
<a href="software.serv.html#lagrangeconfigserver">available</a> at: 
<code>http://www.reelab.net/lagrange</code>.
Lagrange is described in the paper:
Ree, R. H. and S. A. Smith. 2008. Maximum likelihood inference of geographic 
range evolution by dispersal, local extinction, and cladogenesis. <i>Systematic
Biology</i> <b>57(1):</b> 4-14.
It is available as C++ source code and Python script.  It can be downloaded from
<a href="http://code.google.com/p/lagrange/">its web site</a>
at <code>http://code.google.com/p/lagrange/</code>
<P>

<A NAME="Phylocom">
<HR>
<P>
<IMG SRC="icons/phylocom.gif" ALT="[Phylocom icon]">&nbsp;&nbsp;Campbell Webb,
David Ackerley, and Steven Kembel</A>
of the Arnold Arboretum of Harvard University,
of the University of California, Berkeley, and the Center for Ecology
and Evolutionary Biology of the University of Oregon
<tt>(cwebb<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;oeb.harvard.edu)</tt>
have written <strong>Phylocom</strong>
(software for the analysis of <i>PHYLO</i>genetic <i>COM</i>munity structure and character
evolution)
version 4.1, a program to measure phylogenetic community structure and carry 
out comparative methods analysys.  It can measure community phylogenetic 
structure (Webb's mean phylogenetic distance, mean nearest phylogenetic 
neighbour distance, NRI and NTI, Faith's phylogenetic diversity, phylogenetic 
distances among samples and taxa) and compare observed patterns to those 
expected under various null models of community assembly and phylogeny 
structure.  It cna also carry out comparative methods for the analysis of 
character evolution and speciation (independent contrasts, lineages through 
time, and trait evolution under different evolutionary models). In addition it 
can manipulate phylogenetic trees and ecological community data (converting 
Newick to Nexus format, pruning and merging phylogenetic trees, and doing data 
randomization using one of several null models).  It can turn species lists 
into phylogenies with branch lengths based on fossil age estimates.
It is available as C++ source code, Windows executables and Mac OS X universal 
executables.  It can be downloaded from
<a href="http://www.phylodiversity.net/phylocom/">its web site</a>
at <code>http://www.phylodiversity.net/phylocom/</code>
<P>

<A NAME="Forest">
<HR>
<P>
Gabriel Valente and Rod Page</A>
of the Department of Software of the Technical University of Catalonia,
Barcelona, Spain and
the Division of Environmental and Evolutionary Biology of the Institute of
Biomedical and Life Sciences at the University of Glasgow, Scotland
<tt>(valiente<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;lsi.upc.edu)</tt>
have produced <strong>Forest</strong>
version 0.1, programs for producing or using edit scripts to convert one tree 
to another in the GML graph representation language.  GML, the Graph Modelling 
Language (not to be confused with the Geographic Markup Language GML) is a 
file format for graphs.  Forest contains programs to take two trees written in 
GML format, and produce an edit script that can convert one into the other.  
It also contains a program to apply the script to a tree.
It is described in the paper:
Page, R. D. M. and G. Valiente. 2005. An edit script for taxonomic
classifications. <em>BMC Bioinformatics</em> <b>6:</b> 208.
It is available as C++ source code. i To compile this the user also needs to
have the Graph Template Library.  Forest can be downloaded from
<a href="http://darwin.zoology.gla.ac.uk/~rpage/forest/">its web site</a>
at <code>http://darwin.zoology.gla.ac.uk/~rpage/forest/</code>
<P>

<HR>
<A NAME="Phylogenetic Investigator"><P>
<IMG SRC="icons/PI.jpg">&nbsp;
Steven Brewer</A> (<CODE>sbrewer<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>bio.umass.edu</CODE>) of the
Biology Department, University of Massachusetts,
Amherst, Massachusetts, and Robert Hafner, formerly of Western Michigan
University have developed <STRONG>Phylogenetic Investigator</STRONG> version
2.01, a teaching program that allows students
to connect together organisms from a data set provided to them, to make
phylogenies and examine them.  The program is
a Mac OS executable.
Phylogenetic Investigator and its manual are
<a href="http://bioquest.org/BQLibrary/library_details.php?product_id=22909">downloadable here</a> at
<code>http://bioquest.org/BQLibrary/library_details.php?product_id=22909</code>.
It is described in two papers:
<UL>
<LI>Brewer, S.D. 1996. A problem-solving Approach to the teaching of evolution.
<em>Bioscene</em> <b>22:</b> 11-17.
<LI>Brewer, S.D. 1996. Towards improved teaching of evolution: The co-evolution
of Phylogenetic Investigator and a conceptual/procedural knowledge base.
<em>BioQUEST Notes</em> <b>6(3):</b> 11-17.
</UL>
Phylogenetic Investigator appears in
BioQUEST Library VI) from the
<A HREF="http://bioquest.org">BioQUEST consortium</A>, 
(whose web page is at <CODE>http://bioquest.org</CODE>),
a nonprofit publisher of interesting biological teaching software.
Although in the past the BioQUEST Library modules were only available on
a CDROM sold by them, they are now freely downloadable.
<P>

<HR>
<A NAME="Phylap"><P></A>
David Joyce, of the Department of Mathematics and Computer Science,
Clark University, Worcester, Massachusetts (<tt>djoyce<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>clarku.edu</tt>)
makes available the <strong>Phylap</strong> program, version 0.2,
a Java applet that
teaches about inferring phylogenies from molecular sequences using some
clustering algorithms.  The program allows the user to choose some parameters
including the number of species, the rate of evolution and how many states
there are at sites in the molecule.  The program then chooses a random tree
and evolves sequences on it, then computes distances between the sequences
and infers the tree by one of several clustering methods.  The user
can see the results at each stage.  The java applet may be downloaded from
<a href="http://babbage.clarku.edu/~djoyce/java/Phyltree/cover.html">its web
site</a>
at <code>http://babbage.clarku.edu/~djoyce/java/Phyltree/cover.html</code>,
and if your browser can run Java programs it can of course also run
the applet when browsing that page.
<P>

<HR>
<A NAME="Dnatree"><P></A>
<IMG SRC="icons/dnatree.gif">&nbsp; I (Joe Felsenstein, Department of
Genome Sciences and Department of Biology, University of Washington, Seattle,
<tt>joe<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>gs.washington.edu</tt>) have
written a program, <strong>Dnatree</strong> version 1.3, for teaching about
molecular phylogenies.  The program
uses a crude character-mode (not GUI) user interface.  It is based on 
several programs from <A HREF="software.pars.html#PHYLIP">PHYLIP</A>.
The user chooses some rate parameters and the number of species.  The
program then simulates a random tree using a birth process and displays it.
The user can then have the program simulate evolution of a stretch of DNA
sequence along this tree.  Then the program displays a different random
tree, showing the number of changes inferred along that tree by a parsimony
criterion.  It allows the user to manually rearrange the tree to try to
minimize the number of changes.  Dnatree is available as C source code
and as executables for Linux, Windows, and Mac OS X from
<A HREF="http://evolution.gs.washington.edu/dnatree/dnatree.html">its web
site</A>
at <CODE>http://evolution.gs.washington.edu/dnatree/dnatree.html</CODE>
Some executables of an earlier version, 1.1, are also available there
for Mac OS, 68k Macintoshes, SunOS, Compaq/Digital Alpha, and IBM AIX.
<P>

<A NAME="SimpleClade">
<HR>
<P>
Dave Dobson</A>
of the Geology Department
of  Guilford College, Greensboro, North Carolina
<tt>(ddobson<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;guilford.edu)</tt>
has written <strong>SimpleClade</strong>, a program that allows simple 
cladistic analysis with a graphical user interface.  SimpleClade takes a data 
file in a common spreadsheet format, with values in the columns which are 
treated as identifiers of discrete states.  It displays an arbitrary tree and 
then allows you to manually rearrange it interactively.  The parsimony score 
is displayed as you do so. Outgroups can be chosen and the directions of the 
state changes are displayed as well as the tree.  The program was written 
primarily as a teaching tool.  It is available as a Windows executable.  It can 
be downloaded from
<a href="http://www2.guilford.edu/geology/simpleclade/">its web site</a>
at <code>http://www2.guilford.edu/geology/simpleclade/</code>
<P>

<HR>
<P>
<A HREF="software.etc2.html#">To top of this page</A>
<P>
<A HREF="software.serv.html">To next section of software pages</A>
<A NAME="bottom"><BR></A>

<div> 
<P>
<font color="#666666">Notices added in compliance with University of Washington
requirements for web sites hosted at the University: &nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.washington.edu/online/privacy" target="_blank"><font
color="#888888">Privacy</font></a>&nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.washington.edu/online/terms"
target="_blank"><font color="#888888">Terms</font></a></font>
</div>
</BODY>
</HTML>
