<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<HTML>

<HEAD>
<TITLE>Phylogeny Programs (continued) </TITLE>
</HEAD>

<BODY BGCOLOR="#CCFFFF">

<A HREF="software.html">To go to top of Software pages</A>
<P>
<A HREF="software.etc2.html#bottom">To previous part of Software pages</A>
<P>
<HR>
<P>
<A NAME="servers">
<CENTER>
<TABLE WIDTH=90%>
<TR ALIGN=CENTER>
<TD BGCOLOR="navy"><STRONG><FONT SIZE=+3 COLOR="cyan">
W</FONT><FONT SIZE=+2 COLOR="cyan">eb or e-mail servers that can analyze data for you</A>
</FONT></STRONG></TD>
</TR>
</TABLE>
</CENTER>
<P>

This list gives only a brief description of each service.
Congratulations to these
groups for providing these free services.
<UL>
<LI><A NAME="Genebee server"><A HREF="http://www.genebee.msu.ru/genebee.html">GeneBee group</A> of the Belozersky Institute, Moscow State University, Russia.  Results
are returned by email.
<UL>
<LI><A HREF="http://www.genebee.msu.ru/services/phtree_reduced.html">Distance matrix analysis</A> from protein or nucleotide sequences using the Treetop program.
<LI><A HREF="http://www.genebee.msu.ru/clustal/">Server</A> for alignment and
neighbor-joining trees using the <A HREF="software.etc1.html#ClustalW">ClustalW</A> program.
</UL>
<LI><A NAME="CBRG server">The Computational Biochemistry Group</A> of the ETH
in Z&uuml;rich, Switzerland has five programs available on 
<A HREF="http://cbrg.inf.ethz.ch/Server/">its Darwin server</A>, which returns
results by email:
<UL>
<LI><A HREF="http://www.cbrg.ethz.ch/services/TreeGen">TreeGen</A>, which
uses a distance matrix method with distance matrix that you supply,
<LI><A HREF="http://www.cbrg.ethz.ch/services/AllAll">AllAll</A>, which
does a multiple alignment of protein sequences and can either compute a
distance matrix or rooted or unrooted phylogenies using a distance method.
<LI><A HREF="http://www.cbrg.ethz.ch/services/PhylogeneticTree">PhylogeneticTree</A> computes either a parsimony tree or a distance method
that bases itself on a matrix obtained from all pairwise alignment scores
among a set of sequences.
</UL>
<LI><A NAME="PHYLIP-servers"><IMG SRC="icons/PHYLIP.gif" ALIGN=BOTTOM
ALT="[PHYLIP icon here]"></A> Five
groups</A> have made available servers using <A HREF="software.pars.html#phylip.html">PHYLIP</A>:
<UL>
<LI> The Institut Pasteur, Paris has
<A HREF="http://bioweb.pasteur.fr/seqanal/phylogeny/phylip-uk.html">
a server</A> for 15 of the PHYLIP 3.6a programs.
<LI> The Bioinformatics Centre at the University of Wageningen, Netherlands,
makes available a <A HREF="http://www.bioinformatics.nl/tools/plottree.html">server for the <TT>DRAWTREE</TT> and <TT>DRAWGRAM</TT> programs</A>.
<LI> <A HREF="http://www.es.embnet.org/Services/">The Spanish EMBNET node</A>
at the Centro Nacional de Biotecnologia, Madrid, makes
the version 3.5c Protpars and Dnapars programs available as a server.
<LI> The Department of Molecular Evolution of the Evolutionary Biology Centre,
Uppsala University, Uppsala, Sweden, makes a <A HREF="http://artedi.ebc.uu.se/programs/">web server</A> of 9 PHYLIP 3.5c programs available.
<LI> The Biology Workbench of the San Diego Supercomputer Center makes
available <a href="http://seqtool.sdsc.edu/CGI/BW.cgi">some of the PHYLIP 3.5c programs as a web server,</a> requiring only a
<a href="http://workbench.sdsc.edu/">free registration</a>.
</UL>
<LI><A NAME="EBI-ClustalW-serv">The European Bioinformatics Institite</A> (EBI)
has available <A HREF="http://www2.ebi.ac.uk/clustalw/">a server</A>
for <A HREF="software.etc1.html#ClustalW">ClustalW </A>.
tree-based multiple alignment runs.
This also produces tree files in Clustal's own format or in PHYLIP format.
<LI><A NAME="Pasteur-Serv">The Institut Pasteur in Paris</A> has made
available its Mobyle server with many phylogeny programs.  It is available
through <A HREF="http://mobyle.pasteur.fr/cgi-bin/portal.py">its web page</A>
at <CODE>http://mobyle.pasteur.fr/cgi-bin/portal.py</CODE>.
At present it includes web servers for programs from
<TABLE><TR><TD>
<UL>
<LI><A HREF="software.etc1.html#BAMBE">BAMBE</A>
<LI><A HREF="software.pars.html#PHYLIP">PHYLIP</A>
<LI><A HREF="software.dist.html#BIONJ">BIONJ</A>
</UL>
</TD>
<TD>
<UL>
<LI><A HREF="software.dist.html#QuickTree">QuickTree</A>
<LI><A HREF="software.dist.html#weighbor">weighbor</A>
<LI><A HREF="software.etc1.html#fastDNAml">fastDNAml</A>
</UL>
</TD>
<TD>
<UL>
<LI><A HREF="software.etc1.html#PHYML">PHYML</A>
<LI><A HREF="software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</A>
<LI><A HREF="software.etc2.html#Seq-Gen">Seq-Gen</A>
</UL>
</TD>
<TD>
<UL>
<LI><A HREF="software.pars.html#LVB">LVB</A>
<LI><A HREF="software.etc1.html#TreeAlign">TreeAlign</A>
<LI><A HREF="software.old.html#TipDate">TipDate</A>
</UL>
</TD>
<TD VALIGN=top>
<UL>
<LI><A HREF="software.etc2.html#NJplot">NJplot</A>
<LI><A HREF="software.dist.html#EMBOSS">EMBOSS Distmat</A>
<LI> Ktreedist
</UL>
</TD>
</TR></TABLE>
To find these expand the "phylogeny" tab on their list of programs.
<LI><A NAME="ROSE-serv">A</A>
<A HREF="http://bibiserv.TechFak.Uni-Bielefeld.DE/rose/">server</A>
for the <A HREF="software.etc2.html#ROSE">ROSE</A> program which
simulates the evolution of DNA, RNA, or proteins sequences with insertions
and deletions is available from the Technische Fakult&auml;t of the University of
Bielefeld, Germany at
<CODE>http://bibiserv.TechFak.Uni-Bielefeld.de/rose/</CODE>.
<LI><A NAME="multalin-serv">The INRA</A> in Toulouse, France, has
<A HREF="http://multalin.toulouse.inra.fr/multalin/">a server</A>
for Florence Corpet's MultAlin multiple sequence alignment method which is similar
to progressive alignment.   It clusters by UPGMA.  A "cluster file" is provided which may contain
information on the phylogeny.
<LI><A NAME="GenePop server">Andrew Marriott and John Wetherall</A> at the
Curtin University of Technology, Australia, have made available
<A HREF="http://genepop.curtin.edu.au/">a web server</A>
running Michel Raymond and Fran&ccedil;ois Rousset's program 
<a href="software.dist.html#Genepop">Genepop</a>, version 3.4.  This can
do many kinds of population genetic analysis -- the relevant feature for
our purposes is the ability to calculate measures of population
differentiation.
<LI><A NAME="RecPars server">Kim Fisker and Thomas Christensen of the
Department of Computer Science, University of Aarhus, Denmark have
made available
<A HREF="http://www.daimi.au.dk/~compbio/recpars/recpars.html">a server</A>
for the program <A HREF="software.etc1.html#RecPars">RecPars</A> which infers trees with recombination for a set of DNA sequences.
<LI><A NAME="RIO server">Christian Zmasek</A> and Sean Eddy have made available at
the Howard Hughes Medical Institute Janelia Farms campus <A HREF="http://rio.janelia.org">a server</A> for their RIO
(Resampled Inference of Orthologs) program from their <A HREF="software.etc2.html#forester">Forester</A> package.  It searches for orthologs of a given
sequence in the PFAM databse.
<LI><a name="Phylodendron-server">Don Gilbert</A> of the Department of Biology, University of Indiana has made
available <a href="http://iubio.bio.indiana.edu/treeapp/">a web server</a> for his <a href="software.etc2.html#Phylodendron">Phylodendron</A> tree-drawing Java package, which uses tree drawing programs from
<a href="software.pars.html#PHYLIP">PHYLIP</A>.
<LI><a name="MatTree server">Rishabh Bhandari</a> of Delhi, India has
<a href="http://www.mattree.org/index.html">a web server</a> for a number
of methods for fitting a tree to distance matrix data by combining nodes whose
variance of distances to other nodes are least.  It also can calculate
distances between trees and patristic distances from trees.
<LI><a name="Datamonkey server"><IMG SRC="icons/datamonkey.gif" ALIGN=BOTTOM ALT="[Datamonkey icon here]">&nbsp;Sergei Kosakovsky Pond</a> and Simon Frost of the
Antiviral Research Center at the University of California, San Diego, have
the <a href="http://www.datamonkey.org"><strong>Datamonkey</strong> Adaptive Evolution Server</a> which carries out several methods
for detecting positively selected sites in DNA sequences.
<LI><a name="Findmodel">The </a> Los Alamos National Laboratory's Hepatitis C
Virus database project has a <a href="http://hcv.lanl.gov/content/sequence/findmodel/findmodel.html">Findmodel web server</a>
that uses <a href="software.dist.html#weighbor">weighbor</a> and
<a href="software.etc1.html#PAML">PAML</a> together with methods from
<a href="software.etc1.html#Modeltest">Modeltest</a> to quickly evaluate
a set of models for sequence evolution.
<LI> <A NAME="Nexplorer">Nexplorer</A> a
<a href="http://www.molevol.org/nexplorer/cgi-bin/index.cgi">server</a>
from Arlin Stoltzfus's group at NIST and
Wei Gang Qiu's group at Hunter College that allows manipulation of NEXUS files
containing molecular data sets and trees to modify them, and to print out
data sets and plot trees with various kinds of information highlighted.
various
<LI> <A NAME="ProtTest server">The</A> <A HREF="http://darwin.uvigo.es/software/prottest_server.html?">ProtTest server</A> from David Posada's
group in Vigo, Spain that makes available their protein substitution
model selection program <a href="software.etc1.html#ProtTest">ProtTest</a>.
<LI> <A NAME="QuickTree_Treemaker">The</A> Hepatitis C Virus database project at Los Alamos National Laboratory, New Mexico
has <a href="http://hcv.lanl.gov/content/sequence/TREEMAKER/TreeMaker.html">a server</a> that converts your sequences to PHYLIP format and then gets
DNA distances using PHYLIP and a Neighbor-Joining tree using <a href="software.dist.html#QuickTree">QuickTree</a>.
<LI> <A NAME="MultiPhyl server"> The
<a href="http://distributed.cs.nuim.ie/multiphyl.php">MultiPhyl and
ModelGenerator server</a>
for the Java-based multiple-computer likelihood program
<a href="software.etc1.html#MultiPhyl">MultiPhyl</a> and for the
model selection program <a href="software.etc1.html#ModelGenerator">ModelGenerator</a>,
from the National University of Ireland, Maynooth.
<LI> <A NAME="MrBayes form server"></A><A HREF="http://www.auburn.edu/~santosr/mrbayesform.htm">MrBayes form server</A> creates simple command blocks for the program <A HREF="software.etc1.html#MrBayes">MrBayes</A>.
sequences from a database and make a tree from them.
<LI> <A NAME="Modeltest server">ModelTest</A> <a href="http://darwin.uvigo.es/software/modeltest_server.html">server</a> runs the <a href="software.etc1.html#Modeltest">ModelTest</a> program for choosing among
models of evolution.
<LI> <a name="Phylemon_server"> <img src="icons/phylemon.jpg"></a> The
<a href="http://phylemon.bioinfo.cipf.es/">Phylemon web server</a> at the Centro de
Investigacion Principe Felipe in Valencia, Spain can carry out analyses with a
wide variety of phylogeny programs.
<LI> <a name="MAFFT_server">
<a href="http://align.bmr.kyushu-u.ac.jp/mafft/online/server/">MAFFT server</a> is a fast tree-alignment server that uses Fast Fourier Transform methods.
Its program <a href="software.etc1.html#MAFFT"> is also in distribution</a>.
<LI><A NAME=SATCHMO_server><A
HREF="http://phylogenomics.berkeley.edu/q/satchmo/">SATCHMO-JS server</A>
finds alignments and trees for protein sequences using HMMM methods.
<LI><A NAME="Phyfi_server"><A HREF="http://cgi-www.daimi.au.dk/cgi-chili/phyfi/go">Phyfi server</A> draws images of phylogenies.
<LI><a name="ConSurf"><a href="http://consurf.tau.ac.il">ConSurf server</a> identifies functionally important regions on the
surface of a protein of known three-dimensional structure,
using the phylogeny of it and its close sequence homologues.
<LI><a name="POWER server"><a href="http://power.nhri.org.tw">POWER</a>
(PhylOgenetic WEb Repeater) is a server and the Institute of Information
Sciences in Taiwan using <a href="software.pars.html#PHYLIP">PHYLIP</A> and <A
HREF="software.etc1.html#Clustal">ClustalW</A> to align sequences and infer
phylogenies.
<LI> A <A NAME="T-REX server"><a href="http://www.trex.uqam.ca/index.php?action=trex&menuD=1&method=5">web server</a> for the
<a href="software.dist.html#T-REX">T-REX</a> program which includes
inference of reticulated trees, and has additional tree
construction methods using <a href="software.pars.html#PHYLIP">PHYLIP</a>
programs is available from the T-REX group at the
Universit&eacute; du Qu&eacute;bec &agrave; Montr&eacute;al.
<LI><A NAME="phylogeny.fr"><IMG SRC="icons/phylogeny-fr.gif" ALT="[phylogeny.fr
icon]">&nbsp; The <a href="http://www.phylogeny.fr/">phylogeny.fr</a> server at
the LIRMM and the IGS
in Montpellier, France uses several alignment programs, some
programs for distances and tree-drawing from <a
href="software.pars.html#PHYLIP">PHYLIP</a>, and the
programs
<table border=0><tr>
<td valign=top>
<ul>
<li> <a href="software.etc1.html#PHYML">PhyML</a>
<li> <a href="software.pars.html#TNT">TNT</A>
<li> <a href="software.dist.html#BIONJ">BIONJ</a>
</ul>
</td>
<td valign=top>
<ul>
<LI> <a href="software.etc1.html#MrBayes">MrBayes</A>
<li> <a href="software.etc1.html#DTscore">DTscore</A>
<li> <a href="software.etc2.html#TreeDyn">TreeDyn</A>
</ul>
</td>
</tr></table>
to provide a web server that is intended as simple and easy to use.
<li><a name="ATGC servers"> A slightly earlier set of servers from the LIRMM 
for some of these programs is <a href="http://www.atgc-montpellier.fr/index.php?type=pg">here</a>:
<UL>
<li> <a href="http://www.atgc-montpellier.fr/phyml/">PHYML</A>
<li> <a href="http://www.atgc-montpellier.fr/fastme/">FastME</A>
<li> <a href="http://www.atgc-montpellier.fr/physic/">PhySIC</A>
<li> <a href="http://www.atgc-montpellier.fr/physic_ist/">PhySIC_IST</A>
</ul>
<LI><a name="RAxML server"> A
<a href="http://phylobench.vital-it.ch/raxml-bb/">server</a> for the fast
likelihood program <a href="software.etc1.html#RAXML">RAxML</A> at the
Swiss Institute of Bioinformatics.
<LI><a name="GRIMM server"> A
<a href="http://nbcr.sdsc.edu/GRIMM/mgr.cgi">server</a> for the MGR
capability of the GRIMM program
to infer distances or phylogenies by minimizing genome rearrangements.
<LI><A NAME="Rosette server"> A <A HREF="http://www.genomics.ceh.ac.uk/GeneSwytch/gnuplot/">server</A>
for the <a href="software.old.html#RPT">Rosette Plotting Tool</A> program that
produces circular tree plots that show the values of characters graphically.
<LI><img src="icons/swami.gif" alt="[swami icon]"> <A NAME="swami"> The <a href="http://www.ngbw.org/">SWAMI</a> "next generation
biology workbench" server at San Diego Supercomputer Center runs <a
href="software.etc1.html#ClustalW">ClustalW</a>, many programs from <a
href="software.pars.html#PHYLIP">PHYLIP</A>, and <a href="software.etc1.html#PHYML">PHYML</a> as
well as doing many kinds of sequence search.  Requires free login.
<LI> <img src="icons/synatree1.gif" alt="[Synatree icon]"> <a name="synatree">
The Malaysian Genomics Resource Center's <a href="http://synasite.mgrc.com.my/synasuite/mainMenu.jsp?modN=10">synatree</a> server allows
you to choose sequences from sequenced genomes and/or upload sequences, and
make Neighbor-Joining, UPGMA, WPGMA or Splitstree trees from them.
<LI> <A NAME="REFGEN"> The web servers <a href="http://projects.exeter.ac.uk/ceem/refgen.html">
REFGEN</a> (REFormat GEne Names for phylogenies)
and <a href="http://projects.exeter.ac.uk/ceem/treenamer.html">
TREENAMER</a> from the Centre for Eukaryotic
Evolutionary Microbiology at the University of Exeter, U.K. replace sequence
reference names in FASTA files with arbitrary identifiers, and replace the
identifiers in treefiles produced from these sequences with the original
species names.
<LI> <A NAME="Icong">
<a href="http://www.ese.u-psud.fr/bases/upresa/pages/devienne/index.html">Icong</a>
server from the Museum of Natural History in Paris computes a measure of congruence between trees that includes a P-value
for the test that the trees are significantly similar.
<LI> <A NAME="PhyloPars">The <a
href="http://www.ibi.vu.nl/programs/phylopars/">PhyloPars</a> server at the
Department of Theoretical Biology of the Vrije Universiteit Amsterdam
estimates ancestral and present day trait values from 
incomplete data using a Brownian Motion statistical model (not parsimony).
<LI> <A NAME="FootPrinter server"> A
<a href="http://genome.cs.mcgill.ca/cgi-bin/FootPrinter3.0/FootPrinterInput2.pl">web server</a> from the School of
Computer Science at McGill University in Montr&eacute;al, Qu&eacute;bec
runs the <a href="software.pars.html#FootPrinter">FootPrinter</a> program
which searches for control sequences near a set of genes using a parsimony
criterion,
<LI> <A NAME="MicroFootPrinter"><a
href="http://bio.cs.washington.edu/MicroFootPrinter.html">MicroFootPrinter</a>
from the Department of Computer Science and Engineering of the University
of Washington
is a web server which is a front end to the <a
href="software.pars.html#FootPrinter">FootPrinter</a> phylogenetic footprinting program.
which will find related prokaryotice genomes containing genes homologous to
the input sequence and run FootPrinter on that data set.
<li> <A NAME="Bioportal web server"> University of Oslo <a href="http://www.bioportal.uio.no">Bioportal web
server</a> makes available (to academic users for a free registration) many different phylogeny
programs including:
<TABLE><TR>
<TD>
<UL>
<LI><A HREF="software.etc1.html#BEAST">BEAST</A>
<LI><A HREF="software.etc2.html#GARLI">GARLI</A>
<LI><A HREF="software.etc1.html#MAFFT">MAFFT</A>
<LI><A HREF="software.etc1.html#Modeltest">ModelTest</A>
</UL>
</TD>
<TD>
<UL>
<LI><A HREF="software.etc1.html#MrBayes">MrBayes</A>
<LI><A HREF="software.etc1.html#MrModeltest">MrModeltest</A>
<LI><A HREF="software.etc1.html#PAML">PAML</A>
</UL>
</TD>
<TD>
<UL>
<LI><A HREF="software.pars.html#PAUP">PAUP*</A>
<LI><A HREF="software.etc1.html#PhyloBayes">PhyloBayes</A>
<LI><A HREF="software.etc1.html#PHYML">PHYML</A>
</UL>
</TD>
<TD>
<UL>
<LI><A HREF="software.etc1.html#ProtTest">ProtTest
<LI><A HREF="software.etc1.html#RAXML">RAxML</A>
<LI><A HREF="software.etc1.html#treefinder">Treefinder</A>
</UL>
</TD>
</TR></TABLE>
<LI> The <a href="http://bibiserv.techfak.uni-bielefeld.de/roci/">Roci
server</a> at the University of Bielefeld, Germany, reconstructs on a given 
tree using conserved intervals the ancestral gene orders for the interior 
nodes of the tree.
<li> The <a href="http://bibiserv.techfak.uni-bielefeld.de/rococo/">Rococo</a>
server at the University of Bielefeld, Germany, reconstructs ancestral gene
clusters at interior nodes of a given tree by minimizing gains and losses.
<li> <A NAME="NetTest"> <img src="icons/nettest.jpg" alt="[NetTest icon]"> The <a href="http://darwin.uvigo.es/software/nettest/">NetTest</a> server at the Department of Biochemistry, 
Genetics and Immunology of the University of Vigo, Spain
characterizes explicit phylogenetic networks into theoretical classes:
Tree, Galled, Tree-Child, and Tree-Sibling.
<li> <a name="Lagrangeconfigserver"> A web server that configures your data
to enable the files to be run in your own copy of the biogeography
program <a href-"software.etc1.html#Lagrange">Lagrange</a> is available in Rick
Ree's lab at the Field Museum of Natural History in Chicago <a
href="http://www.reelab.net/lagrange">here</a>.
<li> <a name="phyloexplorer"> The PhyloExplorer web server that uploads tree 
files and manage the collection, including associating Wikimedia images of 
species with the tips of the trees, calculation of tree shape statistics, and 
identification of relevant trees in the TreeBase database, is available at
<a href="http://www.ncbi.orthomam.univ-montp2.fr/phyloexplorer/">its web site</a> at the Institut des Sciences de l'Evolution at the
Universit&eacute; de Montpellier II, France.
<li> <a name="PhyDesign"> The PhyDesign server from Francesc Lopez-Giraldez and Jeffrey P. Townsend of the Department of Ecology and Evolutionary Biology
of Yale University implements the Townsend phylogenetic informativeness profiling, providing a quantitative prediction of the utility of loci to solve specific phylogenetic questions.  It is available at <a href="http://phydesign.townsend.yale.edu">its web site</a> at the Townsend lab.
<P>
<HR>

<HR>
</UL>
<P>
<HR>

<P>
<A NAME="ftpservers">
<CENTER>
<TABLE WIDTH=60%>
<TR ALIGN=CENTER>
<TD BGCOLOR="navy"><STRONG><FONT SIZE=+3 COLOR="cyan">
T</FONT><FONT SIZE=+2 COLOR="cyan">he ftp servers</A></FONT></STRONG></TD>
</TR>
</TABLE>
</CENTER>
<P>
There are three major collections of older biology software available for
transfer by ftp.  Until about 1998 these collected many of the
freely available phylogeny programs and redistributed them. They have
been updated very little since then, partly because many program
authors could distribute programs through their own web sites. 
They are thus of historical interest and for access to old programs.
They are
<UL>
<LI> the IUBio archive maintained by Don Gilbert at the Biology Department
of the University of Indiana.  Its ftp address is <CODE>ftp.bio.indiana.edu</CODE>.  There is also <A HREF="http://ftp.bio.indiana.edu/software/">a web page</A>
at <CODE>http://ftp.bio.indiana.edu/software/</CODE>.  It includes a
keyword search capability.
<UL>Relevant code will be found in directories:
<LI><CODE>molbio</CODE> in subdirectories
<UL>
<LI><CODE>evolve</CODE> for phylogeny software, 
<LI><CODE>align</CODE> for alignment software
</UL>
<LI> and <CODE>biology</CODE>.
</UL>
and also in machine-specific directories
where binaries and sources specific to those families of machines are to be
found:
<UL>
<LI><CODE>unix</CODE>,
<LI><CODE>ibmpc</CODE>,
<LI><CODE>mac</CODE>,
<LI><CODE>vax</CODE>, and
<LI><CODE>java</CODE>
</UL>
The general <a href="http://ftp.bio.indiana.edu/">IUBIO site</a> can be
found at <code>http://ftp.bio.indiana.edu</code>  It mentions
a <A HREF="http://iubio.bio.indiana.edu/soft/molbio/Listings.html">
descriptive list of the software</A> that was available on the IUBio server
as of 1998 which can be found at <TT>http://iubio.bio.indiana.edu/soft/molbio/Listings.html</TT>.  Unfortunately, the links on that page are not active but
must be copied out by hand.
<P>
<LI> the <A HREF="ftp://ftp.ebi.ac.uk/pub/software">EBI software archive</A> at the European Bioinformatics Institute 
Cambridge, England (<CODE>ftp.ebi.ac.uk</CODE> in directory <CODE>pub/software</CODE>), and
<P>
<LI> the <A HREF="ftp://ftp.pasteur.fr/pub/GenSoft/">Pasteur Institute archive</A> in Paris (<CODE>ftp.pasteur.fr</CODE> in directory <CODE>pub/GenSoft/</CODE>). 
</UL>
These three archives originally contained almost the same programs, but they
diverged some after 1996, and then after 1998 almost entirely ceased being
updated.
<P>
<H3>Mirrors</H3>
<P>
The <tt>molbio</tt> part of the Indiana IUBio
archive is mirrored (exact copies are maintained) in a number of countries
(I do not know how current these mirrors are):
<DL>
<DT><B>Sweden</B></DT>
<DD><A HREF="ftp://ftp.sunet.se/pub/molbio">A mirror of the <CODE>molbio</CODE> section</A> is maintained at <CODE>ftp.sunet.se</CODE>
in directory <CODE>pub/molbio</CODE></DD>
<DT><B>France</B></DT>
<DD><A HREF="ftp://ftp.pasteur.fr/pub/GenSoft/mirrors/IUBio/molbio">A mirror of the molbio section</A> is maintained at
<CODE>ftp.pasteur.fr</CODE> in directory
<CODE>/pub/GenSoft/mirrors/IUBio/molbio</CODE>.
</DD>
<DT><B>Finland</B></DT>
<DD><A HREF="ftp://ftp.funet.fi/pub/sci/molbio/iumolbio">A mirror of the
<CODE>molbio</CODE> section</A> is maintained at
<CODE>ftp.funet.fi</CODE> in directory
<CODE>pub/sci/molbio/iumolbio</CODE></DD>
</DL>
<P>
<HR>
(This is the end of the software listings).
<P>
<HR>
<A HREF="../phylip.html">
<IMG SRC="icons/PHYLIP.gif" ALT="[Phylip icon here]"> ... to the PHYLIP home page</A>
<div> 
<P>
<font color="#666666">Notices added in compliance with University of Washington
requirements for web sites hosted at the University: &nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.washington.edu/online/privacy" target="_blank"><font
color="#888888">Privacy</font></a>&nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.washington.edu/online/terms"
target="_blank"><font color="#888888">Terms</font></a></font>
</div>
</BODY>
</HTML>

