<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<HTML>

<HEAD>
<TITLE>Programs that are and are not listed</TITLE>
</HEAD>

<BODY BGCOLOR="#CCFFFF">

<A HREF="software.html">To go to top of Software pages</A>
<P>
<CENTER>
<TABLE WIDTH=60%>
<TR ALIGN=CENTER>
<TD BGCOLOR="navy"><STRONG><FONT SIZE=+3 COLOR="cyan">
P</FONT><FONT SIZE=+2 COLOR="cyan">rograms that are and are not listed</FONT></STRONG></TD>
</TR>
</TABLE>
</CENTER>

<P>
Setting the boundaries of a listing such as this is a difficult task, but
there have to be some boundaries.  I have decided that this list will
<EM>not</EM> include:

<UL>
<LI><STRONG>Data conversion utilities</STRONG>.  These are important
and useful programs that convert your data from the format needed by
one program to the format needed by another.  As they do not themselves
perform any data analysis, I have reluctantly decided not to list them.
One, <A HREF="software.old.html#Tonex">Tonex</A> was formerly listed
here and has been removed from the list.
<LI><STRONG>Clustering programs</STRONG> that are not intended to
infer phylogenies.  There is a large and active literature on clustering
methods, with many programs in distribution.  However making clusters and
inferring phylogenies are not the same thing (though closely related).
I have tried to decide which packages had some material on inferring
phylogenies, and include only those.
<LI><STRONG>Tree alignment programs that do not infer trees.</STRONG>
Programs or servers that use tree-based alignment of sequences, but do
not allow you to output the resulting tree, fall slightly outside of
this listing, as any tree that they have is not visible to the outside
world.   This means that some <A HREF="software.etc1.html#ClustalW">
ClustalW</A> servers are not listed here (those that do not allow you
access to the tree).
<LI><STRONG>Programs for detecting evidence of recombination that are
not using trees intensively enough.</STRONG>  For example, there are
useful and helpful tests that look at pairs of sites but not at
anything involving trees more directly.  This is a particularly
difficult boundary to define but there has to be one here somewhere.
<LI><STRONG>Sequence editors that do not submit jobs to phylogeny programs.</STRONG>
I have listed a number of sequence-editing environments that allow one to
automatically invoke phylogeny programs on selected sets of sequences or
sets of sites.  Sequence editors that do not have these job submission
capabilities lie outside the domain of this listing.
<LI><STRONG>Coalescent programs that do not infer trees.</STRONG>  This
is a tough one.  There are now a number of programs that add up likelihoods
or Bayesian posteriors over different possible coalescent trees of genes
within species.  Our own lab is distributing one such package.  This
category of programs is growing rapidly.  Many of them infer population
parameters or genetic parameters, or times of most recent common ancestry
of mutants.  Wonderful and important stuff, but as it does not focus on
between-species phylogeny, I am going to exclude most of these programs.
The exceptions will be those that report inferences of the coalescent
tree itself, or those that infer the between-species phylogeny.
This is going to be a hard boundary to define as coalescent programs
proliferate.
</UL>
<P>
I will continue to agonize over these boundaries, but for now this
is the policy.  If you feel that these policies are wrong, you can
always (a) argue with me, or (b) make your own phylogeny programs
web pages.
<P>
<hr>
<div> 
<font color="#666666">Notices added in compliance with University of Washington
requirements for web sites hosted at the University: &nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.washington.edu/online/privacy" target="_blank"><font
color="#888888">Privacy</font></a>&nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.washington.edu/online/terms"
target="_blank"><font color="#888888">Terms</font></a></font>
</div>
</BODY>
</HTML>

