<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<HTML>

<HEAD>
<TITLE>Old Phylogeny Programs (ones no longer distributed)</TITLE>
</HEAD>

<BODY BGCOLOR="#CCFFFF">

<A HREF="software.html">To go to top of Software page</A>
<P>
<CENTER>
<TABLE WIDTH=60%>
<TR ALIGN=CENTER>
<TD BGCOLOR="navy"><STRONG><FONT SIZE=+3 COLOR="cyan">
O</FONT><FONT SIZE=+2 COLOR="cyan">ld Phylogeny Programs<BR> (ones no longer distributed)</FONT></STRONG></TD>
</TR>
</TABLE>
</CENTER>

<P>

The programs described in this web page were formerly listed in the
Phylogeny Programs web pages but are ones which are no longer
distributed.  The entries describing them have been moved here.  <STRONG>The
web links and ftp addresses in these listings typically do not work
any longer</STRONG>.  It may be possible to track down the authors of these
packages and get information directly from them.  I have added comments at
the top of some entries as to what I know about its status.
<P>

<HR>
<P>
<i>(This program is one that is still in distribution.  It has been
removed from the list, not because there is anything wrong with it,
but because I have reconsidered where the boundary of this listing is.
As a data-conversion utility it lies outside it.)</i>
<P>
<IMG SRC="icons/tonex.gif" ALT="[Tonex icon]">
<A NAME="Tonex">Mauro J. Cavalcanti</A> (<CODE>maurobio<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>geocities.com</CODE>), o
f
the Departamento de Biologia Geral, Universidade Santa Ursula, Rio de Janeiro,
Brazil, has written <STRONG>Tonex</STRONG>, a data set translation program for
converting <A HREF="software.pars.html#Hennig86">Hennig86</A> data sets into
NEXUS format.  It is available as MSDOS and Windows executables and as Turbo
Pascal 7.0 source code at its
<A HREF="http://www.geocities.com/RainForest/Vines/8695/software.html#Cladistics
">
web site</A> at
<CODE>http://www.geocities.com/RainForest/Vines/8695/software.html#Cladistics
</CODE>.
<P>

<HR>
<P>
<i>Not sure this one has been in distribution any time in the last 10 years.</i>
<P>
<A NAME="PHYSYS">J. S.  Farris  and  Mary  Mickevich</A>  earlier  released  a
package  of
phylogeny programs, <STRONG>PHYSYS</STRONG>, which, at about $5,000, was extremely expensive (in
my opinion, which is certainly a biased one).  I  am  not  sure  whether,  from
whom, or under what conditions it is still available.
<P>

<HR>
<P>
<i>Kevin Nixon has withdrawn ClaDOS: it is replaced by his more recent program
<A HREF="software.pars.html#WINCLADA">WINCLADA</A> which incorporates the
features of ClaDOS and of another program, DADA.</i>
<P>
<A NAME="ClaDOS"><STRONG>ClaDOS</STRONG></A>, an interactive program which allow
s rearrangement of trees  and
their  evaluation,  mapping of characters into them, and more, is available for
DOS systems from Kevin Nixon, L. H. Bailey Hortorium, Cornell  University,  467
Mann  Library,  Ithaca,  New York  14853.  Rumor has it that the cost is in the
vicinity of $55 US.
<P>

<HR>
<P>
<i>I do not know whether these programs are being distributed right now, and
if so, from where. Zharkikh is now at Myriad Genetics in Salt Lake City, Utah.</i>
<P>
<A NAME="Zharkikh unix">Andrey Zharkikh</A> (<CODE>zharkikh<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>hgc6.sph.uth.tmc.edu
</CODE>),
(then) of the Genetics Centers at the University of Texas at Houston,
has written a series of <STRONG>Unix programs</STRONG> to carry out various phylogeny methods.
They are easily compiled on standard Unix C compilers.
They include programs for
<UL>
<LI> Translating, reformatting and printing aligned sequences
<LI> Calculation of evolutionary distances among sequences
<LI> Inferring phylogenetic trees and bootstrapping, by parsimony or neighbor-joining distance methods
<LI> Drawing the tree structure to a PostScript printer
</UL>
These are available by
<A HREF="http://hgc6.sph.uth.tmc.edu:8080/bootstrap.dir/index.html">World Wide Web</A> from
<CODE>http://hgc6.sph.uth.tmc.edu:8080/bootstrap.dir/index.html</CODE> or by
anonymous  ftp  from  <CODE>hgc6.sph.uth.tmc.edu</CODE>  in directory <CODE>pub/zharkikh/bootstrap</CODE>.
<P>

<HR>
<P>
<i>Lake is of course still at UCLA.  I do not think this program is currently publicly distributed.</i>
<P>
<A NAME="Evomony">James Lake</A> (<CODE>lake<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>uclaue.mbi.ucla.edu</CODE>)
distributes  <STRONG>Evomony</STRONG>,  a  program   for   using   the
"evolutionary parsimony" (invariants) method for inferring phylogenies from DNA
or RNA sequences.  It runs on 286 or higher DOS  systems  with  at  least  500k
bytes of memory. A Macintosh version was also contemplated.  I do not know what
the current distribution arrangements are.  Lake's  address  is  Department  of
Biology, University of California, Los Angeles, California  90024.
<P>

<HR>
<P>
<i>There is no sign that Fujitsu still distributes this package, which is
a version of a phylogeny program called IDEN that is used at the National
Institute of Genetics, Japan (but is not in general distribution).</i>
<P>
<A NAME="SINCAIDEN">Fujitsu Ltd.</A> ("a $21 billion [now it's 37.7 billion]
global  leader  in  advanced  computer,
telecommunications,  and  electronic devices") sells a Fujitsu S
family  workstation  complete  with  a   program,   <STRONG>SINCAIDEN</STRONG>,   which   allows
"experimental  researchers,  even  those  unfamiliar  with  such analyses, [to]
easily create phylogenetic trees in their own laboratories."  The program  also
allows  searches  of the major nucleic acid sequence and protein databases (the
ad I saw does not make it clear whether these databases are provided  with  the
workstation).   The  methods  available  are  UPGMA, neighbor-joining, Farris's
(Distance Wagner)  and  the  modified  Farris  distance  matrix  methods.   The
workstation  is  SPARC  compatible  and  runs SunOS.  The SINCAIDEN program was
developed by the group at the National Institute of Genetics, Japan  under  Dr.
Takashi  Gojobori.   Fujitsu  Ltd. may be contacted at 21-8, Nishi-Shinbashi 3-
chome, Minato-ku, Tokyo 105, Japan (phone 81-3-3437-5111 ext. 2831,  fax  81-3-
5472-4354),  or  in  the  U.S. at Fujitsu America Inc., 3055 Orchard Drive, San
Jose, California 95134-2017 (phone 1-408-432-1300  ext.  5168,  fax  1-408-434-
1045).
There is <A HREF="http://sinca.fqs.co.jp/InfoSINCA/">a web page in Japanese</A>
at <CODE>http://sinca.fqs.co.jp/InfoSINCA/</CODE>.
Several years ago the price of SINCAIDEN (with workstation) was $28,000.
<P>

<HR>
<P>
<i>This teaching program seems no longer to be in distribution (it is not clear
that its author decided this).</i>
<P>
<IMG SRC="icons/systack.gif" ALT="[Systack icon here]">
<A NAME="Systack"> Arnold G. Kluge</A> (<CODE>akluge<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>umich.edu</CODE>), of the
Department of Biology of the University of Michigan, has written
<STRONG>Systack</STRONG>, a teaching program designed to teach the
principles of synapomorphy/homology analysis in the context of chordate
phylogeny.  It implements a hierarchical filing system in the form of a
phylogeny, with character information available on chordates and with users
able to add new characters.  It is a Hypercard stack for Macintosh computers,
and is available free for noncommercial use.
<A HREF="http://www.ummz.lsa.umich.edu/herps/systack.html">A Web site</A> is
available at <CODE>http://www.ummz.lsa.umich.edu/herps/systack.html</CODE> to
download it.
<P>

<HR>
<P>
<i>This program has been withdrawn because it is now contained within
their later system <A HREF="software.dist.html#vCEBL">vCEBL</A>.</i>
<P>
<A NAME="supgma">Allen Rodrigo</A> of the Computational and Evolutionary
Biology Laboratory, School of Biological Sciences, University of Auckland,
Auckland, New Zealand (<TT>a.rodrigo<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>auckland.ac.nz</TT>) distributes
<STRONG>sUPGMA</STRONG> (Serial sample UPGMA).  It reconstructs
evolutionary histories/genealogies under the assumption of a molecular
clock when sequences are obtained serially in time.  Input is a
distance matrix.  sUPGMA can also carry out the ordinary UPGMA,
WGPMA and Neighbor-Joining methods.  sUPGMA is a
Java applet.  It is available for downloading
from <A HREF="http://www.cebl.auckland.ac.nz/">
its web site</A> at
<CODE>http://www.cebl.auckland.ac.nz/"></CODE>
or for use as <A HREF="software.serv.html#supgma-serv">a server</A>.
sUPGMA runs under Netscape Communicator 4.7 or Microsoft Internet
Explorer 5 on PCs running Windows 95/98, and Microsoft Internet Explorer
4.5 on Macintoshes.  It can also be run in standalone mode if you have
a Java runtime engine.
<P>

<HR>
<P>
<i>Andrew Rambaut has retired SPOT as he considers it obsolete in view
of the features of <A HREF="software.pars.html#PAUP">PAUP*</A></i>.
<P>
<IMG SRC="icons/spot.gif" ALT="[SPOT icon here]">
<A NAME="SPOT">Andrew Rambaut </A> of the Department of
Zoology, University of Oxford, (<CODE>andrew.rambaut<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zoo.ox.ac.uk</CODE>) and
Nick Grassly, currently of the Zoologisches Institut, Universit&auml;t
M&uuml;nchen
(<CODE>grassly<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zi.biologie.uni-muenchen.de</CODE>),
have written <STRONG>SPOT</STRONG> (Sequence Parameters Of Trees).
SPOT is a program that will calculate the likelihood of a given tree topology
for a set of aligned nucleotide sequences. For each topology, SPOT will
estimate the maximum likelihood values of branch lengths and other parameters
of the model of nucleotide evolution that has been chosen. Such parameters
include the ratio of transitions to transversions (TS/TV ratio) and relative
rates of substitution at different codon positions. Branch lengths can also
be constrained to assume a molecular clock hypothesis. Multiple datasets
and multiple trees can be analysed which is useful for performing Monte
Carlo simulations of hypothesis (parametric bootstraps). Although SPOT does
not estimate tree topology, an accompanying program, SPOTSHELL, will iterate
between fastDNAml and SPOT until the maximum likelihood parameters and topology
has been found (or at least something close to it).
SPOT is available as C source code for Unix workstations, or as Macintosh sources and
executables.
It can be obtained from the
<A HREF="http://evolve.zoo.ox.ac.uk/Spot/Spot.html">SPOT Web page</A>
at <CODE>http://evolve.zoo.ox.ac.uk/Spot/Spot.html</CODE>.
<P>

<HR>
<P>
<i>Lyons-Weiler has moved to the University of Pittsburgh.  I suspect that
RASA will be made available there in the near future</i>.
<P>
<IMG SRC="icons/rasa.gif" ALT="[RASA icon here]">
<A NAME="RASA">James Lyons-Weiler</A> of the Biological Sciences Department
of the University of Massachusetts, Lowell
(<CODE>James_LyonsWeiler<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>UML.EDU</CODE>) has written <STRONG>RASA</STRONG>,
version 2.5, software for Macintoshes that will perform "Relative Apparent
Synapomorphy Analysis", a test for the presence of phylogenetic signal
in any type of discrete character data matrix (morphological or
molecular).  The RASA program carries out the test and plots the results.
RASA is menu-driven.  The test compares
the observed and null rates of increase in cladistic similarity 
among pairs of taxa predicted by an increase in the phenetic 
similarity among taxon pairs.
The test is described in a paper:
Lyons-Weiler, J., G.A. Hoelzer, and R.J. Tausch. 1996. Relative Apparent
Synapomorphy Analysis (RASA) I: the statistical measurement of
phylogenetic signal.  <i>Molecular Biology and Evolution</i> <B>13:</B>
749-757, the taxon variance plot tool in RASA was described in the
paper: Lyons-Weiler, J., and G.A. Hoelzer. 1997.  Escaping from the
Felsenstein Zone by detecting long branches in phylogenetic data.
<i>Molecular Phylogenetics and Evolution</i> <B>8:</B> 375-384, and
outgroup selection issues were discussed in
Lyons-Weiler, J., G. A. Hoelzer and R. J. Tausch. 1998.
Optimal outgroup analysis. <i>Biological Journal of the Linnean
Society</i> <B>64:</B> 493-511.
The programs are available as Macintosh executables from
<A HREF="http://bio.uml.edu/LW/RASA.html">their web page</A> at 
<CODE>http://bio.uml.edu/LW/RASA.html</CODE>.
<P>

<HR>
<P>
<i>There does not seem to be any web site for either this program or
its author.</i>
<P>
<img SRC="icons/tgard2.gif" ALT="[Tree Gardener icon here]">
<a NAME="Tgard">Tiago Ramos</a> of the Museu de Zoologia, 
Universidade de Sao Paulo, Sao Paulo, Brazil
(<code>tcramos<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ibm.net</code>)
has developed <strong>Tree Gardener</strong> version 2.2.1, a shell to run

<a HREF="#Hennig86">Hennig86</a> interactively on Windows systems.
The program allows the user to edit data files, submit jobs, including
successive weighting runs, rerooting, and consensus trees.  It displays the
resulting trees and allows the user to edit them.  It is freely available
provided that the user has a registered copy of Hennig86.
Tree Gardener is available from the Digital Taxonomy
<a HREF="http://www.geocities.com/RainForest/Vines/8695/software.html#Cladistics">
web site</a> at
<code>http://www.geocities.com/RainForest/Vines/8695/software.html#Cladistics</code>.
<P>

<HR>
<i>Black is still at Colorado State but seems to have ceased ftp distribution of BIOSYS-2</i>
<P>
<A NAME="BIOSYS2">William C. Black</A>
of the Department of Microbiology, Colorado State University
(<CODE>wcb4<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>lamar.colostate.edu
</CODE>) makes available <STRONG>BIOSYS-2</STRONG>.  This is a modified
version of David Swofford and Richard B. Selander's 1981 program BIOSYS-1,
adding some features.  Swofford's program was originally distributed by the
Illinois Natural History Survey but has not been in distribution for
some years -- this is the only version in distribution.  Although
in many respects it has been superseded by other population genetics
packages (such as the ones that follow this listing) it computes
gene frequencies, linkage disequilibria, and many other population
genetics analyses on electrophoretic genotypes.  For our purposes,
it is most relevant to note that it can compute the genetic distances
of Nei (unbiased distance), Rogers, and Cavalli-Sforza.  It can
also carry out UPGMA, WPGMA, Complete Linkage, and Single Linkage
clustering of populations (inferrring clocklike phylogenies), or infer
nonclocklike phylogenies by Farris's
Distance Wagner method.  Black has added the capabilities of
making bootstrap replicates of the genetic distances and writing them
out in <A HREF="software.pars.html">PHYLIP</A> format.
BIOSYS-2 is available as a DOS executable with documentation and
FORTRAN source code.  It is distributed <A HREF="ftp://lamar.colostate.edu/pub/wcb4">by ftp</A> from
<CODE>lamar.colostate.edu</CODE> in directory <CODE>pub/wcb4</CODE>.
<P>

<HR>
<i>Rambaut has discontinued distribution of End-Epi, as its features are
replaced by the latest versions of Pybus and Rambaut's program <A HREF="software.etc2.html#Genie">Genie</A></i>
<P>
<IMG SRC="icons/end-epi.gif" ALT="[End-Epi icon here]">
<A NAME="End-Epi">Andrew Rambaut</A> of the Department of Zoology,
University of Oxford, (<CODE>andrew.rambaut<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zoo.ox.ac.uk</CODE>)
has written <STRONG>End-Epi</STRONG> (Endemic-Epidemic) version 1.0, 
a program to examine trees to assess relative cladogenesis (whether
there is evidence that one clade has speciated more than another),
and make lineages-through-time plots, with the objective of discovering
whether the rate of speciation has been constant through time ("endemic")
or has been higher initially ("epidemic").
Rambaut considers that the methods in End-Epi have since been superseded
by those in his and Oliver Pybus's program <A HREF="#Genie">Genie</A>.
End-Epi is available free for Macintoshes with system 7.0 or later
from its
<A HREF= "http://evolve.zoo.ox.ac.uk/software/End-Epi/End-Epi.html">Web page</A> at
<CODE>http://evolve.zoo.ox.ac.uk/software/End-Epi/End-Epi.html</CODE>
or <A HREF="ftp://evolve.zoo.ox.ac.uk/packages">by ftp</A> from
<CODE>evolve.zoo.ox.ac.uk</CODE> in directory <CODE>packages</CODE> as 
file <CODE>End-Epi10.hqx</CODE>.  However the Macintosh executable will
not work with operating systems later than version 7.5, unless you
switch off Modern Memory Manager in the control panel first.
<P>

<HR>
<P>
<i>George is still at the University of Michigan but the web page distributing this program has disappeared</i>
<P>
<A NAME="QUARTET2">George Estabrook</A> of the Department of
Biology, University of Michigan, Ann Arbor, Michigan
(<TT>Estabrook<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>umich.edu</TT>) has written
<STRONG>QUARTET2</STRONG>, which
calculates measures of difference between phylogenies
based on quartets (subtrees of four tips).
The methods are described in a paper:
Estabrook, G. F.  1992.
Evaluating undirected positional congruence of individual taxa between two
estimates of the phylogenetic tree for a group of taxa.
<i>Systematic Biology</i> <B>41:</B> 172-177.
QUARTET2 is available as a DOS executable from his
<A HREF="http://www-personal.umich.edu/~gfred/">web page of computer programs</A>
at <CODE>http://www-personal.umich.edu/~gfred/</CODE>.
<P>

<HR>
<P>
<i>George is still at the University of Michigan but the web page distributing this program has disappeared</i>
<P>
<A NAME="MEAWILK">George Estabrook</A> of the Department of Biology of the
University of Michigan, Ann Arbor, Michigan (<TT>gfred<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>umich.edu</TT>)
distributes <STRONG>MEAWILK</STRONG> (MEAcham and WILKinson criteria)
which uses a randomization test to evaluates support from character data for
hypothesized monophyletic
groups  It uses criteria published by Christopher Meacham (1994. Phylogenetic
relationships at the basal radiation of angiosperms: Further study by probability
of character compatibility.  <i>Systematic Zoology</i> <B>19:</B>
506-522)
by Mark Wilkinson (1998.  Split support and split conflict randomization tests
in phylogenetic inference. <i>Systematic Biology</i> <B>47:</B> 673-695).  MEAWILK is
a DOS executable distributed from <A HREF="http://www-personal.umich.edu/~gfred/">Estabrook's programs web site</A> at
<CODE>http://www-personal.umich.edu/~gfred/</CODE>.
<P>

<HR>
<P>
<i>MARKOV seems to no longer be available as a program.  A web server
service for it was announced, but there is no response when I try it.</i>
<P>
<A NAME="MARKOV">The program </A><STRONG>MARKOV</STRONG>, by Graziano Pesole
of the University of Milan (<tt>graziano.pesole<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>unimi.it</tt>)
and Professor Cecilia Saccone of the University of Bari, Italy,
computes  a  distance  measure  between  pairs  of
nucleotide sequences.  It also constructs phylogenies from these and summarizes
the 4x4 substitution matrices between the pairs of species.   It  uses  a  more
general  model of substitution than used in PHYLIP, the Stationary Markov Model
published by Lanave et al. in <i>Journal of Molecular Evolution</i> <b>20:</b>
86-93 in 1984.  This is also
described in the paper by Saccone et. al. in Methods in Enzymology volume  183,
pages 570-583, 1990.  It is often known as the General Time Reversible model.
Bootstrapping is used to analyze the statistical error of
the results.  Output files from CLUSTAL and  PILEUP,  as  well  as  some  other
formats,  can  be used for input, and analysis can be confined to certain codon
positions in coding sequences.  The program is written in ANSI C and  runs  on
Unix  systems.  A <a href="http://bigarea.area.ba.cnr.it:8000/EmbIT/coda_markov.html">Web page server</a> is also available at
<code>http://bigarea.area.ba.cnr.it:8000/EmbIT/coda_markov.html</code>.
<P>

<HR>
<P>
<i>Salisbury says that as he is no longer in academia he no longer has time to
distribute this program. His recent email address is <tt>b.salisbury<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>genaissance.com</tt></i>
<P>
<IMG SRC="icons/secant.gif" ALT="[SECANT icon]">
<A NAME="SECANT">Benjamin Salisbury</A> (<TT>ben<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>aya.yale.edu</TT>),
of the Department of Ecology and Evolutionary Biology, Yale University
has released <STRONG>SECANT</STRONG> version 2.2,
based on an earlier program, CLINCH, by Kent Fiala, now of SAS Institute.
SECANT was previously known as CLINCH2.
It is  probably  the  most sophisticated
compatibility analysis (clique analysis) program, capable of handling
unordered multiple states.  It can also group characters by Salisbury's own
Strongest Evidence criterion.
It and its criteria are described in a paper: Salisbury, B. A. 1999. Strongest
evidence in compatibility: clique and tree evaluation using apparent
phylogenetic signal. <i>Taxon</i>  <B>48:</B> 755-766.
SECANT is available as a Windows95/NT executable,
and its source code is described as available on request.
It is available from
<A HREF="http://jkim.eeb.yale.edu/salisbur/">its web page</A>
at <CODE>http://jkim.eeb.yale.edu/salisbur/</CODE>.
<P>
                                                                                
<HR>
<P>
<i>Bailey Kessing has moved to Maryland and for the moment there seems to
be no web or ftp distribution of this program</i>
<P>
<IMG SRC="icons/sequencer.gif" ALT="[Sequencer icon here]">
<A NAME="Sequencer">Bailey Kessing</A> of the Smithsonian Tropical
Research Institute, Panama (<TT>BaileyKessing<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>email.com</TT>)
has
released <STRONG>Sequencer</STRONG> on 6.1.0, a multipurpose program for
manipulating
and analysing data.  It can calculate many kinds of statistics, but for
the purposes of this listing, its important features are the ability to
calculate variety of distances from DNA and protein sequence data.
Distances for DNA sequence data include Jukes-Cantor, Kimura's 2 parameter
model, and computation of synonymous and nonsynonymous substitutions
using Kimura's 2 parameter model.  It can also bootstrap distance
calculations.  It is available as Macintosh executables (both PowerPC and
non-PowerPC executables are available) from
<A HREF="http://nmg.si.edu/Sequencer.html"> its web site</A> at
<CODE>http://nmg.si.edu/Sequencer.html</CODE>.
<P>

<HR>
<P>
<i>Black is still at Colorado State University but seems to have discontinued distribution of this package</i>
<P>
<A NAME="RAPD-PCR">William C. Black</A>
of the Department of Microbiology, Colorado State University
(<CODE>wcb4<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>lamar.colostate.edu</CODE>)
has produced <STRONG>PROGRAMS FOR ANALYSIS OF RAPD-PCR DATA</STRONG>.
There are 7 programs.  The ones relevant to this listing are
<UL>
<LI>RAPDPLOT which computes distances between RAPD patterns using
either Nei and Li's distance or a percent difference score,
<LI>RAPDBOOT which makes these distance matrices in while bootstrap
sampling the data set, and
<LI>RAPDBIOS which converts the RAPD data into a gene frequency
dataset for analysis by <A HREF="software.dist.html#BIOSYS2">BIOSYS-2</A>.
(Black's site is also the distribution site for BIOSYS-2).
</UL>
Other programs compute F<SUB>ST</SUB> and linkage disequilibria from
RAPD data.
The programs are in FORTRAN source code with DOS executables.
They are distributed <A HREF="ftp://lamar.colostate.edu/pub/wcb4">by ftp</A> from <CODE>lamar.colostate.edu</CODE>
in directory <CODE>pub/wcb4</CODE> as file <CODE>RAPDS.ZIP</CODE>.
<P>

<HR>
<P>
<i>This program seems to not be distributed any longer.  A recent email
address for Salisbury is <tt>b.salisbury<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>genaissance.com</tt>.  The
Kim lab is now at the University of Pennsylvania but does not have this
program available at their web site.</i>
<P>
<A NAME="DNASEP">
<P>
Ben Salisbury</A> (<TT>ben<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>aya.yale.edu</TT>) of the
Department of Ecology and Evolutionary Biology, Yale University, New
Haven, Connecticut, has released <STRONG>DNASEP</STRONG>, which uses
(with permission) some code from my
<A HREF="software.pars.html#PHYLIP">PHYLIP</A> program DNAPARS to carry out
Salisbury's criterion of Strongest Evidence Parsimony.  The criterion
is described in a paper: Salisbury, B. A. 1999. Strongest evidence:
maximum apparent phylogenetic signal as a new cladistic
optimality criterion. <i>Cladistics</i> <B>15:</B> 137-149.
DNASEP is available as a Windows95 executable from
<A HREF="http://jkim.eeb.yale.edu/salisbur/">Salisbury's web site</A> at <CODE>
http://jkim.eeb.yale.edu/salisbur/</CODE>.  It has been partially
superseded by a later program of Salisbury's,
<A HREF="software.pars.html#SEPAL">SEPAL</A>, which has more functions.
<P>

<HR>
<A NAME="SEPAL">
<P>
<i>This program seems to not be distributed any longer.  A recent email
address for Salisbury is <tt>b.salisbury<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>genaissance.com</tt>.  The
Kim lab is now at the University of Pennsylvania but does not have this
program available at their web site.</i>
<P>
<IMG SRC="icons/sepal.gif" ALT="[SEPAL icon]">
Ben Salisbury</A> (<TT>ben<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>aya.yale.edu</TT>) of the
Department of Ecology and Evolutionary Biology, Yale University, New
Haven, Connecticut, has written <STRONG>SEPAL</STRONG>, version 1.01, which 
can search for the trees that maximize the Strongest
Evidence criterion, Apparent Phylogenetic Signal. 
It can also do Iterative SE, parsimony, and parsimony
jackknifing. It can also calculate decay values (Bremer supports) for either
parsimony or Strongest Evidence.  It also has some options for removing
characters that are particularly noisy.  The criterion
is described in a paper: Salisbury, B. A. 1999. Strongest evidence:
maximum apparent phylogenetic signal as a new cladistic
optimality criterion. <i>Cladistics</i> <B>15:</B> 137-149.
SEPAL is available as a Windows95 executable from
<A HREF="http://jkim.eeb.yale.edu/salisbur/">Salisbury's web site</A> at <CODE>
http://jkim.eeb.yale.edu/salisbur/</CODE>.
<P>

<HR>
<P>
<i>Belyi is no longer at Penn State and his program is not in distribution.
His email address currently seems to
be <tt>sambauser<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>katehok.ac93.org</tt>.</i>
<P>
<A NAME="TreePack">Igor Belyi</A>  (<CODE>Igor_Belyi<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>transarc.com</CODE>)
has developed <STRONG>TreePack</STRONG>,
a minimum evolution program for Unix workstations.
TreePack can be obtained <A HREF="ftp://ftp.cse.psu.edu/pub/belyi">by ftp</A>
from <CODE>ftp.cse.psu.edu</CODE> in directory <CODE>pub/belyi</CODE>.
It is available as Unix source code in C.
<P>

<HR>
<P>
<i>The ftp distribution of QR2 seems to have become nonfunctional.  Gascuel
is still at this email address, and the Institut Pasteur server for this
program is still functional.</i>
<P>
<A NAME="QR2">Olivier Gascuel and Denise Levy</A>
(<CODE>gascuel<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>lirmm.fr</CODE>) at the
the Laboratoire d'Informatique, de Robotique et de Micro-Electronique de
Montpellier (LIRMM) of the Universite de Montpellier II, France
have produced
<STRONG>QR2</STRONG> version 1.0, a program which approximates a dissimilarity
(or distance) matrix by a tree.  The method is described in a paper:
Gascuel, O. and D. Levy. 1996. A reduction algorithm for approximating a
(nonmetric) dissimilarity by a tree distance. <i>Journal of
Classification</i> <B>13:</B> 129-155.  The program is available
in C++ source code
<A HREF="ftp://lirmm.lirmm.fr/pub/genome/phylo">by ftp</A> from <CODE>lirmm.lirmm.fr</CODE> in directory
<CODE>pub/genome/phylo</CODE>.  It is also available as
a server <A HREF="software.serv.html#Pasteur-Serv">from the Institut Pasteur</A>.
<P>

<HR>
<P>
<i>Posada is in Vigo at the email address indicated, but his software
site there no longer includes this program.</i>
<P>
<A NAME="MATRIX">David Posada</A> (<CODE>dposada<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>uvigo.es</CODE>)
of the University of Vigo, Spain
has produced <STRONG>MATRIX</STRONG> version 1.5, a program to calculate a matrix of pairwise
distances (treating gaps as a fifth state by
default): for absolute, uncorrected, JC69 and K80 distances from a set of
aligned DNA sequences in PHYLIP sequential or NEXUS format.
It is available as a Macintosh executable from
<A HREF="http://bioag.byu.edu/zoology/crandall_lab/programs.htm">
its web site</A> at
<CODE>http://bioag.byu.edu/zoology/crandall_lab/programs.htm</CODE>.
<P>

<HR>
<P>
<i>Salminen is currently at the National Public Health Institute in
Helsinki, Finland.  His email address is <tt>mika.salminen<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ktl.fi</tt>,
but he does not seem have the Bootscanning Package currently available
anywhere. </i>
<P>
<A NAME="bootscan">Mika Salminen and Wayne Cobb</A> (<CODE>msalminen<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>hiv.hjf.org</CODE> and <CODE>wcobb<!deathtospam>&nbsp; (at)<!deathtospam>&nbsp; reed.hjf.org</CODE>), of the Henry M. Jackson
Foundations for the Advancement of Military Medicine, Walter Reed Army
Institite of Research, Bethesda, Maryland, have released the
<STRONG>Bootscanning Package</STRONG>, version 1.0beta.  This is a series
of shell scripts and programs that analyze DNA sequences for evidence of
recombination.  It breaks the sequence into separate pieces that are
analyzed for the bootstrap support of various groups, and it looks for
evidence of significant conflict among trees for different parts of the
sequence.  The programs are currently available only as Sun executables.
They require <A HREF="software.etc2.html#GDE">GDE 2.2a</A> and
<A HREF="software.pars.html#PHYLIP">PHYLIP</A> version 3.4 to work.
They are available <A HREF="http://www.ktl.fi/hiv/mirrors/pub/programs">by anonymous ftp</A> from from
<CODE>http://www.ktl.fi</CODE> in directory <CODE>/hiv/mirrors/pub/programs</CODE>.
<P>

<HR>
<P>
<i>Zharkikh and Rzhetsky are respectively, with Myriad Genetics in
Utah, and Columbia University Genome Center.  VOSTORG does not
seem to be distributed anywhere.</i>
<P>
<A NAME="VOSTORG">Andrey A.  Zharkikh, Andrey Rzhetsky, and co-workers</A> in the
Institute of Cytology and Genetics, Siberian Branch of the Russian Academy of
Sciences, Novosibirsk, Russia, have produced <STRONG>VOSTORG</STRONG>, a package of
programs for alignment (both manual and automatic) and inferring phylogenies by
distance methods and parsimony for molecular sequences.
(Zharkikh and Rzhetsky are currently in the US; their e-mail addresses are
<CODE>zharkikh<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>myriad.com</CODE> and <CODE>andrey<!deathtospam>&nbsp; (at)<!deathtospam>&nbsp; genome2.cpmc.columbia.edu</CODE>).  VOSTORG runs on under DOS on
PC-compatibles and includes some rather fancy graphics (for DOS).
It is available from
<A HREF="http://molevol.bionet.nsc.ru/vs.htm">its Web page in Russia</A>
from <CODE>http://molevol.bionet.nsc.ru/vs.htm</CODE>.
The programs are described in a paper: Zharkikh, A. A., A-Yu. Rzhetsky,
P. S. Morosov, T. L. Sitnikova, and J. S. Krushkal. 1991. VOSTORG: a package of
microcomputer programs for sequence analysis and construction of phylogenetic
trees. <i>Gene</i> <B>101:</B> 251-254.
<P>

<HR>
<P>
<i>Fitch is still at this address but the programs do not seem to be currently distributed.</i>
<P>
<A NAME="FITCH">Walter Fitch</A> (<CODE>wfitch<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>uci.edu</CODE>), of the Department
of Ecology and Evolutionary Biology, of the University of California at
Irvine, has available by anonymous ftp at <CODE>daedalus.bio.uci.edu</CODE> in
directory <CODE>pub/outgoing/evoprog</CODE> <STRONG> about 20 programs</STRONG>
which carry out various kinds of phylogeny estimation and related tasks.
They are available in source code in FORTRAN 77,
(except for a few which are in C) and also as Sun SPARC executables and as
DOS executables.  They include:
<UL>
<LI> <STRONG>ANCESTOR</STRONG> which searches for most parsimonious trees
for nucleotide or protein sequences,
<LI> <STRONG>EVOLVES</STRONG> which carries out the original Fitch-Margoliash
distance matrix method,
<LI> <STRONG>WTDPARS</STRONG>, programs for weighted parsimony analysis
according to the methods he has introduced in the papers by P. L. Williams and
W. M. Fitch, in pages 453-470 of the <i>Nobel Symposium on the Heirarchy of
Life</i>, edited by B. Fernholm, K. Bremer, and H. Jornval, Elsevier, 1989,
and the paper by P. L. Williams and W. M. Fitch, in
<i>Advances in Enzymology</i>, volume 183, pages 615-625, 1990.
</UL>
There are also many programs that convert sequences among various formats,
generate all possible trees, shuffle sequences, align sequences, and do
various other functions.
The programs are available
<A HREF="ftp://daedalus.bio.uci.edu/pub/outgoing/evoprog">by anonymous ftp</A>
from <CODE>daedalus.bio.uci.edu</CODE> in directory
<CODE>pub/outgoing/evoprog</CODE>.
<P>
There is also <STRONG> TDRAW</STRONG> which draws a tree in Postscript.  This
program is in C, and is not available as a DOS executable. 
<A HREF="ftp://daedalus.bio.uci.edu/pub/outgoing/tdraw">It is available</A>
in directory <CODE>pub/outgoing/tdraw</CODE>.
<P>

<HR>
<P>
<i>Kay Nieselt-Struwe is currently at the Zentrum f&uuml;r Bioinformatik T&uuml;bingen, Germany, but she does not make this program available from there.</i>
<P>
<A NAME="STATGEOM">Kay Nieselt-Struwe</A> (<CODE>kns<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>phy.auckland.ac.nz</CODE>) of the
Department of Physics of the University of Auckland, New Zealand has released
version 1.0 of <STRONG>STATGEOM</STRONG>. 
It carries out computation of the statistical geometry in distance and in
sequence space of a set of aligned DNA/RNA, amino acid or binary sequences.
The user can decide to
either compute the overall tree-likeness of the whole set, or a
certain subset, or given a tree of the sequences compute the
reliability of certain edges in the tree.
Postscript files of the graphs of the
statistical geometry are automatically generated.
A sequence reformatting utility allow various sequence formats to be
read in.
STATGEOM is written in ANSI C; source code with documentation and a Sun SPARC executable are
available <A HREF="ftp://cage.mpibpc.gwdg.de/pub/kniesel">by anonymous ftp</A> at <CODE>cage.mpibpc.gwdg.de</CODE> (or <CODE>134.76.209.64</CODE>) in directory
<CODE>pub/kniesel</CODE>.
The method of statistical geometry was originally published in:
Eigen, M., Winkler-Oswatitsch, R. and Dress, A. 1988.
Statistical geometry in sequence space: a method of comparative
sequence analysis. <i>Proc. Natl. Acad. Sci. USA</i> <B>85:</B> 5913-5917.
<p>

<HR>
<P>
<i>There is no sign that this program is still available</i>
<P>
<A NAME="PA"> Hang-Kwang Luh, John Gittleman, and Mark Kot</A> of the University
of Tennessee at Knoxville have made available <STRONG>PA</STRONG>, a
package of Macintosh programs that implement the phylogenetic autocorrelation
comparative method introduced by Gittleman and Kot (<i> Systematic Zoology </i>, 1990).  It is free and is available <A HREF="ftp://ftp.math.utk.edu/pub/luh">
by anonymous ftp</A> from <CODE>ftp.math.utk.edu</CODE> in directory <CODE>pub/luh</CODE>.
<P>

<HR>
<P>
<i>Blomberg has moved to Sydney, and I do not know whether this program
is currently distributed anywhere.</i>
<P>
<A NAME="Fels-rand">Simon Blomberg</A>, of the School of Botany and Zoology
of the Australian National University, Canberra
(<TT>S.Blomberg<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>anu.edu.au</TT>) has announced the
beta-release of a small comparative method program, <STRONG>Fels-Rand</STRONG>
version 0.91beta
<i>[look folks, don't blame me, I had nothing to do with naming this program]</i>.
It is designed to analyse data when the phylogeny is only poorly known, as
when there is one or several polytomies. The program is said to be
inspired by a 1994 paper in <i>Systematic Biology</i> by Jonathan Losos.
It retains known tree topology and randomises the unknown parts of the tree,
unlike some other programs, which randomize the whole tree. The statistics
are calculated on independent contrasts from fully (randomly) resolved trees.
Fels-rand is written in XLISP-STAT, and runs in the XLISP-STAT environment
(in other words you first must get and install XLISP-STAT on your computer
to run the code, which is written in the XLISP-STAT language).
XLISP-STAT is available for Macintosh, Windows, and Unix.
Fels-rand is available from <A HREF="http://dingo.cc.uq.edu.au/~ansblomb/">
Blomberg's home page</A> at <CODE>http://dingo.cc.uq.edu.au/~ansblomb/</CODE>.  Its README file can be found
<A HREF="http://life.biology.mcmaster.ca/~brian/evoldir/Other/ComparativeMethod.software">
in a newsgroup posting</A> at
<CODE>http://life.biology.mcmaster.ca/~brian/evoldir/Other/ComparativeMethod.software</CODE>.
<P>

<HR>
<P>
<i>Dopazo is now at the Centro Nacional de Investigaciones Ocol&oacute;gicas (<tt>jdopazo<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>cnio.es</tt>.  He appears not to have ABLE in distribution at this time.</i>
<P>
<A NAME="ABLE">Joaquin Dopazo</A> of the R&amp;D Department of TDI (TDI-EMBNet),
Spain, (<CODE>dopazo<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>samba.cnb.uam.es</CODE>) has written
a  program  <STRONG>ABLE</STRONG> (Analysis of Branch Length Errors) which
implements the method described by Adell and Dopazo in <i>J. Mol. Evol.</i> <B>38:</B> 305-309
(1994).   This  is a parametric bootstrap test of constancy in evolutionary
rates.  The idea of the test is to simulate a large number of a data sets
under the model of rate constancy and then to examine the distribution of
the branch lengths. After, a tree is reconstructed 
without the constraint of rate constancy it can be checked whether the 
observed branch length values fall within the expected distribution.
The program is intended for use with the PHYLIP programs FITCH and KITSCH.
It is available as a DOS executable from
<A HREF="http://www.cnb.uam.es/~bioinfo/Software/Ximo/www1.html">
Dopazo's software web page</A> at
<CODE>http://www.cnb.uam.es/~bioinfo/Software/Ximo/www1.html</CODE>
or <A HREF="ftp://ftp.cnb.uam.es/pub/cnb/molevol">by anonymous ftp</A> at: <CODE>ftp.cnb.uam.es</CODE>
in directory <CODE>pub/cnb/molevol</CODE>.
<P>

<HR>
<P>
<i>Dopazo is now at the Centro Nacional de Investigaciones Ocol&oacute;gicas (<tt>jdopazo<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>cnio.es</tt>.  He appears not to have SOTA in distribution at this time. There is a never-completed web page there for it but it is not
distributed from that page.  A modified version is available for clustering
genes for gene expression.</i>
<P>
<A NAME="SOTA">Joaquin Dopazo</A> and J. M. Carazo (<CODE>
jd19662<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ggr.co.uk</CODE>
and <CODE>carazo<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>embnet.cnb.uam.es</CODE>) have produced <STRONG>
SOTA</STRONG>, a package to carry out the Self Organizing Tree Algorithm.
It is based on Kohonen's unsupervised 
neural network of self-organizing maps and on Fritzke's growthing cell 
structures algorithm to construct phylogenetic trees from biological 
molecular sequence data.   It is described in a paper:
Dopazo, J. and J. M. Carazo. 1997. Phylogenetic reconstruction using an 
unsupervised growing neural network that adopts the topology
of a phylogenetic tree. <i>Journal of Molecular Evolution</i>
<B>44a:</B> 226-233.  SOTA can use sequence data, distance matrix data,
or dipeptide frequencies from proteins.
SOTA is available as source code in C for Unix,
as executables for SGI workstations, and also with a Windows program
called Drawer that draws the resulting trees.  The package with
documentation is available <A HREF="ftp://ftp.cnb.uam.es/cnb/sota">by anonymous ftp</A> from
<CODE>ftp.cnb.uam.es</CODE> in directory <CODE>cnb/sota</CODE>.
<P>

<HR>
<P>
<i>Ken is at the address below but as far as I can tell does not make RSVP
available any longer.</i>
<A NAME="RSVP"><P>
Ken Rice</A> (<CODE>ken_a_rice<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>gsk.com</CODE>) of
GlaxoSmithKline Beecham, Upper Merion, Pennsylvania (and adjunct faculty at the
University of Pennsylvania)
has  produced  <STRONG>RSVP</STRONG> (restriction site variability program)
which calculates several measures of genetic variability based  on  restriction
map  data.   It  also  produces  Jukes-Cantor  corrected distance matrices with
standard errors from collections  of  restriction  maps.   C  source  code  for
Version 2.08 of RSVP is available free <A HREF="ftp://phylogeny.harvard.edu/pub/rice">by anonymous ftp</A> from:
<CODE>phylogeny.harvard.edu</CODE> in directory <CODE>pub/rice</CODE>.
It runs under Unix.
<P>

<HR>
<P>
<i>Nick Grassly</A> is currently at the Department of Infectious Disease
Epidemiology of Imperial College School of Medicine, St. Mary's Campus, London
(<tt>n.grassly<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ic.ac.uk</tt>). This program seems not to be
currently available there.</i>
<P>
<IMG SRC="icons/seqevolve.gif" ALT="[SEQEVOLVE icon here]">
<A NAME="SEQEVOLVE">Nick Grassly</A> (formerly of the Department of Zoology,
University of Oxford, most recently of the Department of Biology, Imperial
College, London) 
has written <STRONG>SEQEVOLVE</STRONG>, a program that takes
standard (Newick) formatted treefiles and evolves sequences along them following a
stochastic process with the expected number and type of substitutions calculated
according to a model of molecular evolution. A variety of nucleotide substitution
models are implemented: Jukes and Cantor (1969), Kimura (1980), Felsenstein
(1981), Hasegawa <i>et al</i>, (1985), and the DNAML model from PHYLIP
(Felsenstein, 1995).
a PowerMacintosh and Macintosh executable is available, as well as source code
files for Unix systems.  SEQEVOLVE does not allow for rate heterogeneity among
sites or among codon positions as his more recent program
<A HREF="software.etc2.html#Seq-Gen">Seq-Gen</A> does.
SEQEVOLVE is available
<A HREF="ftp://evolve.zoo.ox.ac.uk/packages/grassly/Seqevolve">by ftp</A> from <CODE>evolve.zoo.ox.ac.uk</CODE> in directory
<CODE>packages/grassly/Seqevolve</CODE> as files <CODE>seqevolve-mac.hqx</CODE> or <CODE>seqevolve.tar.Z</CODE>.
<P>

<HR>
<P>
<i>John is now at University of California, San Diego, and has not continued
to distribute this program.</i>
<P>
<A NAME="TheSiminator">John Huelsenbeck</A> (<CODE>johnh<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>brahms.biology.rochester.edu</CODE>) of the Department of Biology of the University of Rochester
has written <STRONG>TheSiminator</STRONG>, version 2.0, a 
program that simulates the evolution of nucleotide sequences along a given
tree or trees.  It allows for gamma-distributed rate variation among sites,
and the Hasegawa-Kishino-Yano 1985 model of nucleotide substitution.
It is distributed as C source code and as a Macintosh executable, with
examples of input files.  It can be fetched from
<A HREF="http://brahms.biology.rochester.edu/software.html">the Huelsenbeck laboratory software web page</A> at
<CODE>http://brahms.biology.rochester.edu/software.html</CODE>.  (The C
source code without the Macintosh executable is also available from the
Slatkin Lab's
<A HREF="http://ib.berkeley.edu/labs/slatkin/software.html">software Web page</A> at
<CODE>http://ib.berkeley.edu//labs/slatkin/software.html</CODE>.)
<P>

<HR>
<P>
<i>John is now at University of California, San Diego, and has not continued
to distribute this program.</i>
<P>
<IMG SRC="icons/stratcon.gif" ALT="[StratCon icon here]">
<A NAME="StratCon">John Huelsenbeck</A> (<CODE>johnh<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>brahms.biology.rochester.edu</CODE>)
of the Department of Biology of the University of Rochester
has written <STRONG>StratCon</STRONG>, a program to test the
consistency of a tree with stratigraphy of the species.  It uses a permutation
test described in the paper Huelsenbeck, J.  1994.  Measuring and testing
the fit of the stratigraphic record to phylogenetic trees.  <i>Paleobiology</i> <B>20:</B> 470-483.
The program is available as a Macintosh executable.  It can be
fetched from the Slatkin Lab's
<A HREF="http://ib.berkeley.edu/labs/slatkin/software.html">software Web page</A> at
<CODE>http://ib.berkeley.edu//labs/slatkin/software.html</CODE>.
<P>

<HR>
<P>
<i>Sherbakov seems to be at Irkutsk State University now.  I do not know
of any distribution of this program.</i>
<A NAME="UO"><P>
Dmitri Yu. Sherbakov</A> of the
Laboratory of Molecular Systematics, Limnological Institute,
Russian Academy of Sciences, Irkutsk <TT>(dysh<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>sherb.lin.irk.ru)</TT>
manually. It gets species names from a sequence file in
Sequential <A HREF="software.pars.html#PHYLIP">PHYLIP</A> format with up tp 150
sequences, then allows you to build multiple trees by clicking on species
names.  It allows multifurcations.
UO is distributed as C sources and Linux binaries from
<A HREF="http://sherb.lin.irk.ru/uo.html">its web page</A> at
<CODE>http://sherb.lin.irk.ru/uo.html</CODE>.  It requires X windows and
the XForms library.
<P>

<HR>
<P>
<i>Pagel has subsumed Continuous into the package <a href="software.etc2.html#BayesTraits">BayesTraits</a> which he and Andrew Meade have produced.
and there is now no separate distribution of Continuous.  The web address has
also changed -- the new one is available in the listing for that package.</i>
<A NAME="Continuous"><P>
Mark Pagel, of the School of Biological Sciences of the University of Reading,
U.K. (<tt>m.pagel<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;reading.ac.uk</tt>) has written
<strong>Continuous</strong>, version 1.  This is a program to fit a
generalized least squares (GLS) model to continuous character data on a
given tree.  The method infers several parameters that allow testing of
whether the data shows any phylogenetic signal, and whether change is
punctuational.  It can also test random walk versus directional change models,
and infer ancestrat states.  The method is described in two papers:
<ul>
<li>Pagel, M. 1997. Inferring evolutionary processes from phylogenies.
<i>Zoologica Scripta</i> <b>26:</b> 331-348.
<li>Pagel, M. 1999.  Inferring the historical patterns of biological evolution.
<i>Nature</i> <b>401:</b> 877-884.
</ul>
Continuous is a Mac OS executable.  It is downloadable from
<a href="http://www.rubic.rdg.ac.uk/meade/Mark/Continues.html">its web page</a>
at <code>http://www.rubic.rdg.ac.uk/meade/Mark/Continues.html</code>
<P>

<HR>
<A NAME="TurboTree">
<P>
<i>TurboTree seems to have been withdrawn some time ago from the Massey
University programs in distribution. Incidentally, it was described in a paper
in CABIOS in 1987.</i>
<P>
David Penny</A> (Institute of Molecular Biosciences, Massey University,
Palmerston North, New Zealand) has been offering for free distribution two
DOS programs, one a fast parsimony program, <STRONG>TurboTree</STRONG>.  There is also another,
Great Deluge, an approximate
search for the most parsimonious tree by a quasi-random method.  He tells me
that funding exigiencies are such that he may soon have to start charging for
these.  His electronic mail address is <CODE>dpenny &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>massey.ac.nz</CODE>.
<P>

<HR>
<A NAME="CIPRES"><P>
<i>The CIPRES platform available in this early version has ceased to be
distributed, as a newer and more comprehensive platform is under active
development.</i>
<p>
<IMG SRC="icons/CIPRES.gif">&nbsp;The <strong>CIPRES</strong> project (<i>C</i>yber <i>I</i>nfrastructure for <i>P</i>hylogenetic
<i>RES</i>earch), an NSF-funded consortium of many institutions centered on
the University of
New Mexico, the University of Texas at Austin, and the San Diego
Supercomputer Center and coordinated by Bernard Moret (<tt>moret<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>cs.unm.edu</tt>), Tandy Warnow (<tt>tandy<!deathtospam>&nbsp; (at)<!deathtospam>&nbsp; cs.utexas.edu</tt>), and Mark Miller
(<tt>mmiller<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>scsc.edu</tt>)</A>
has released version 0.2.0.0 of their software platform for analyzing molecular
sequences from large numbers of species.  This is written in Java and includes
a copy of <A HREF="software.etc1.html#MrBayes">MrBayes</A>.  It allows the user
to read a data set in NEXUS format, and a provisional tree, and carry out
the DCM3 divide-and-conquer algorithm for dividing the data set into
overlapping subsets of sequences.  Runs of MrBayes on these can then be
launched on the same machine or on other machines.  The intent is to
enable very large data sets of up to 7,000 sequences to be analyzed by
combining the results of the MrBayes runs.  There is also an intention to
soon include support for running <A HREF="software.pars.html#PAUP">PAUP*</A>.
The CIPRES software is
available for WindowsXP, Mac OS X, and Linux.  It can be downloaded from
<A HREF="ttp://www.phylo.org/sub_sections/software.htm">the CIPRES software web page</A>
at <CODE>ttp://www.phylo.org/sub_sections/software.htm</CODE>
<P>

<HR>
<A NAME="TOPAL"><P>
<i>TOPAL versions 1 and 2 have been superseded by <a
href="software.etc1.html#TOPALi">TOPALi</A> and they are no longer in
distribution.</i>
<P>
<IMG SRC="icons/topal.gif" ALT="[TOPAL icon]">
&nbsp;Gr&aacute;inne McGuire</A>, currently of Taylor Fry Consulting Actuaries,
Sydney, Australia (<tt>grainne<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>taylorfry.com.au</tt>)
and Frank Wright of Biomathematics and Statistics Scotland, in
Dundee (<TT>frank<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>bioss.sari.ac.uk</TT>), have released <STRONG>TOPAL</STRONG>,
which checks for evidence of past
recombination events, by looking for changes in the inferred
phylogenetic tree <i>TOP</i>ology between adjacent regions of a multiple
sequence <i>AL</i>ignment. Their method detects recombinations by sliding a
window along a sequence alignment, and measuring the discrepancy between
the trees suggested by the first and second halves of the window, using
distance matrix methods. TOPAL version 2 includes a statistical test,
based on parametric bootstrapping, and an improved statistic which
reduces the effect of among-site rate heterogeneity on the results.  The
original method is described in the paper: McGuire, G., F. Wright, and
M. J. Prentice. 1997. A graphical method for detecting recombination in
phylogenetic data sets. <i>Molecular Biology and Evolution</i> <B>14:</B>
1125-1131.
The TOPAL 2.0 program, and recent changes to the method, are described
in a paper: McGuire, G. and F. Wright. 2000. TOPAL 2.0 : improved
detection of mosaic sequences within multiple alignments.
<i>Bioinformatics</i> <B>16</B>: 130-134. 
TOPAL is a set of Unix Bourne shell scripts and C code, plus four
programs in C from my PHYLIP
package. These are available from the TOPAL web site
These are available from the TOPAL
<A HREF="http://www.bioss.sari.ac.uk/~frank/Genetics"> web site</A>
at <CODE>http://www.bioss.sari.ac.uk/~frank/Genetics</CODE>.
A Windows interface to TOPAL, written by Iain Milne in Java, called <strong>TOPALi</strong>, is available at
<A HREF="http://www.bioss.ac.uk/~iainm/topali/">the TOPALi web site</A> at
<CODE>http://www.bioss.ac.uk/~iainm/topali/</CODE>.  It is described in
the paper:  Milne, I., F. Wright, G. Rowe, D. F. Marshal, D. Husmeier and
G. McGuire. 2004. TOPALi: Software for automatic identification of recombinant sequences within DNA multiple alignments. <i>Bioinformatics</i> <b>20:</b>
1806-1807.
<P>

<HR>
<A NAME="homoplasytest"><P>
<i>The homoplasy test software is no longer available -- the web site that
distributed it was Maynard Smith's personal web site, and after his death
in 2004 that site was finally taken down.  Although I do not know of any
current distribution, the test itself is implemented in the <a
href="software.dist.html#START">S.T.A.R.T.</a>
package.  The web site of recollections of Maynard Smith mentioned below is
still available.  Noel Smith is at the Centre for the Study of Evolution
of the University of Sussex (<tt>Noel<!deathtospam>&nbsp;(at)
<!deathtospam>&nbsp;sussex.ac.uk</TT>) and also at the Veterinary Laboratories
Agency Weybridge.  I do not know of any distribution of this program from any
web site there.  The page of tributes to John Maynard Smith is temporarily
down, but Noel Smith hopes to put it back up in the future.
</i>
<P>
John Maynard Smith and Noel Smith</A>
of the School of Biological Sciences of the University of Sussex
(<CODE>noelsmith<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>yahoo.com</CODE>) released programs to
carry out their <STRONG>homoplasy test</STRONG> for recombination in
sequences.  The test is described in a paper:
Maynard Smith, J. and N. H. Smith. 1998. Detecting recombination from gene
trees.  <i>Molecular Biology and Evolution</i> <B>15:</B> 590-599. 
The programs are distributed in QBASIC for DOS and must be run using
QBASIC.  They are available from Maynard Smith's
<A HREF="http://www.biols.susx.ac.uk/Home/John_Maynard_Smith/">web site</A>
at <CODE>http://www.biols.susx.ac.uk/Home/John_Maynard_Smith/</CODE>.
(John Maynard Smith, a true leader in evolutionary biology, died in April,
2004.  <A HREF="http://www.lifesci.sussex.ac.uk/CSE/members/jms/jms-display.php">Here</A> is a nice page of recollections of him, often
amusing, by other biologists. His home page is still available.)
<P>

<HR>
<A NAME="GCG">
<P>
<i>As indicated below, Accelrys, Inc. has discontinued distribution
and support for GCG (in
spite of a petition from users asking it not to) as of June, 2008</i>
<P>
Genetics Computer Group</A> ("GCG"),
a subsidiary of Accelrys, Inc., produces the
<STRONG>GCG Wisconsin Package</STRONG>, version 10.3,
a leading package of sequence search and analysis programs, together with
updates of the leading sequence databases.  Included are programs for
tree-based multiple sequence alignment, calculation of distances, and
estimating phylogenies by the neighbor-joining and UPGMA distance matrix
methods:
<UL>
<li><STRONG>PileUp</STRONG> creates a multiple sequence alignment of up to 
500 sequences using the method of Feng and Doolittle, similar to the 
<A HREF="software.etc1.html#ClustalW">ClustalW</A> method of Higgins and Sharp.
However Pileup uses a UPGMA clustering instead of Neighbor-Joining clustering,
and does not allow as much flexibility in substitution matrices as ClustalW.
A dendrogram illustrating sequence similarity is also created. 
<li><STRONG>Distances</STRONG> writes a matrix of the pairwise evolutionary 
distances between aligned sequences.  To correct for multiple substitutions 
several methods may be chosen:  for nucleic acid sequences, Kimura's 
two-parameter method, the Tajima and Nei method, and the Jin 
and Nei method; for protein sequences, the Kimura method, and 
for either type of sequence the Jukes-Cantor method.  
<li><STRONG>GrowTree</STRONG> creates a phylogenetic tree using the neighbor-joining method
or UPGMA.
</UL>
The Wisconsin Package contains front ends in X windows or as a web interface
(SeqWeb, sold separately by Accelrys).  The GCG Wisconsin Package includes
PAUPSearch and PAUPDisplay, which provide a front
end to <A HREF="software.pars.html#PAUP">PAUP*</A>.
The GCG Wisconsin Package runs on Compaq (Digital) Unix workstations
(under Tru64 Unix 4.0E or later),
Silicon Graphics workstations (under IRIX 6.5 or later), or on
Sun SPARC-based workstations (under Solaris 2.6 or later), IBM AIX
systems (version 5.1 or later), or Intel-compatible Red Hat Linux systems
(version 7.1 and later).
The programs are described on the
<A HREF="http://accelrys.com/products/additional-products.html">Accelrys
Additional Products</A> web page at
<CODE>http://accelrys.com/products/additional-products.html</CODE>.
There it is announced that distribution and support of the GCG package will
cease in June 2008.  It is not made clear how to order it before then.
Prices are no
longer given on the web site (you are asked to contact them).  The most
recently posted prices (several years ago) were $5,000 (plus $3,000 per
year thereafter) for an academic installation ($18,000 and $6,000 for a
nonacademic installation).  These have probably changed since then.
<P>

<HR>
<i>The Saitou lab in Mishima seems to have removed this program from distribution.</i>
<P>
<A NAME="TreeTree"><P>
Naruya Saitou</A> of the Laboratory for
Evolutionary Genetics, National Institute of Genetics,
Japan (<CODE>nsaitou<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>genes.njg.ac.jp</CODE>) has
produced <STRONG>TreeTree</STRONG>,
a set of programs for neighbor-joining distance matrix analysis with
bootstrapping.   Macintosh executables are provided, and
documentation and Pascal or C source code is provided in the package.
The package consists of three main programs:  NJ, a standard neighbor-joining
program, NJorg, which makes an unrooted neighbor-joining tree, and
bootNJ, which bootstraps the analysis, given a data file with multiple
distance matrices, one for each bootstrap replicate.
It can be downloaded <A HREF="ftp://ftp.nig.ac.jp/pub/mac/TreeTree">by
anonymous ftp</A> from <CODE>ftp.nig.ac.jp</CODE>
in directory <CODE>pub/mac/TreeTree</CODE>. 
<P>

<HR>
<i>Jaap Buntjer is still at Keygene, but for a while there was no sign that this
program is currently distributed.  But now it is available again at
http://www.plantbreeding.wur.nl/UK/software_PhylTools.html  and will
shortly be back in our listing.</i>
<P>

<A NAME="Phyltools"><P>
<IMG SRC="icons/phyltools.gif">&nbsp; Jaap Buntjer</A> (<tt>jaap.buntjer (at)
keygene.com</tt>), when he was at the Laboratory of Plant Breeding, Wageningen
University, Wageningen, Netherlands wrote <strong>Phyltools</strong>, which
is distributed by the Laboratory.  It is a package of utilities for
0/1 binary, interval, and quantitative data (including morphological data,
RFLPs, RAPDs, and AFLPs) that can calculate distance matrices, carry out
bootstrapping and some permutation methods, and do a number of conversions
between different formats.
It is intended to work with the <A HREF="software.pars.html#PHYLIP">PHYLIP</A>
package and can read and write PHYLIP file formats.  Phyltools is a
Windows executable.  It can be downloaded from
<A HREF="http://www.dpw.wau.nl/pv/PUB/pt/">its web site</A>
at <code>http://www.dpw.wau.nl/pv/PUB/pt/</code>
<P>

<HR>
<i>This is no longer available, as far as I know.  I do not know whether Ken
is at the same email address.</i>
<P>
<A NAME="AMP"><P>
Ken Rice</A> (<CODE>ken_a_rice<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>gsk.com</CODE>) of 
GlaxoSmithKline Beecham, Upper Merion, Pennsylvania (and adjunct faculty at the
University of Pennsylvania)
has  produced  <STRONG>AMP</STRONG> (Accepted Mutation Parsimony), a program which calculates 
stepmatrices for protein parsimony analysis, for use in <STRONG>PAUP*</STRONG>
and <STRONG>MacClade</STRONG>.  It uses transition probabilities under
models of protein evolution to calculate these stepmatrices.
It is available as C source code for Unix, from its
<A HREF="http://www.cis.upenn.edu/~krice/">web site</A> at
<CODE>http://www.cis.upenn.edu/~krice/</CODE>.
<P>

<HR>
<i>This product seems to have been discontinued by BioRad.  All that they
now have for it is some old manuals, in their Discontinued Software
section of their web pages.</i>
<P>
<A NAME="FingerprintingII"><P>
BioRad,</A> division of Sadtler USA, Inc.,
<CODE>(Sadtler_USA_Sales<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>bio-rad.com)</CODE>
distributes
<STRONG>Fingerprinting II Informatix Software</STRONG>, a package for quantitative
RFLP and Fingerprinting analysis.  The package is available for
PC and Macintosh, and includes average linkage clustering of the gel
patterns.  The <A HREF="http://www.bio-rad.com">web page
</A>for this software is at <CODE>http://www.bio-rad.com</CODE>.  
(To find the page for the programs you have to choose the links there
for "Life Sciences Research", "Software", and then "Fingerprinting II
Informatix Software".
BioRad's headquarter is at
1000 Alfred Nobel Drive,
Hercules, California 94547, and their phone number is (510) 724-7000.
Other contact information is available on their web page.  One
mention of the price elsewhere is that it is from $5,170 up to $20,119.
<P>

<HR>
<A NAME="GeneContent">
<P>
<i>This program is no longer listed at the Gu lab software site.  Gu is still
at the address listed and distributes other programs at that site.</i>
<P>
Xun Gu,</A> of the Department of Genetics, Development and Cell Biology
and the Center for Bioinformatics and Biological Statistics atIowa State
University, Ames, Iowa (<tt>xgu<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>iastate.edu</tt>) together with Wei
Huang, Dongping Xu,
and Hongmei Zhang has produced
<strong>GeneContent</strong>, version 1.0.2,
a program to compute distances between whole genomes from their gene contents.
It uses the presences and absences of gene families (or an extended
version which also notes the presence of a gene family only as a single copy).
It also uses these distances to compute a Neighbor-Joining tree of genomes.
It is described in a paper: Gu, X., W. Huang, D. Xu, and H. Zhang. 2005.
GeneContent: software
for whole-genome phylogenetic analysis. <i>Bioinformatics</i> <b>21:</b>
1713-1714.
<P>

<HR>
<i>Jermiin is at the University of Sydney, and at the University of Rome "Tor
Vergata" as well right now.  The link for this program at his software page in Sydney 
seems to be broken, for now.</i>
<P>

<A NAME="TreeCons"><P>
Lars Jermiin</A> of the School of Biological Sciences
of the University of Sydney, Australia (<TT>lars.jermiin<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>usyd.edu.au</TT>)
and Olena Anpilogova
have produced <STRONG>TreeCons</STRONG> version 1.0.  It
generates a weighted consensus tree from trees obtained by maximum likelihood analysis, 
generates relative likelihood support on edges in this and other user-specified trees, 
and does the Kishino-Hasegawa test with any level of significance.   It
reads output files and tree files produced by some of the programs in
<A HREF="software.pars.html#PHYLIP">PHYLIP</A>,
<A HREF="software.etc1.html#MOLPHY">MOLPHY</A>,
<A HREF="software.etc1.html#fastDNAml">fastDNAml</A>
and <A HREF="software.etc1.html#TrExMl">TrExMl</A>.
The output file from TreeCons is in a format that then is fed back into
PHYLIP's program Consense.  A number of weighting schemes to compute tree
weights from their likelihoods are allowed.
The weighting schemes and the underlying theory are described in a paper:
Jermiin L. S., G. J. Olsen, K. L. Mengersen, and S. Easteal. 1997.
Majority-rule consensus of
phylogenetic trees obtained by maximum likelihood analysis.
<EM>Molecular Biology and Evolution</EM> <B>14:</B> 1296-1302. 
TreeCons is distributed as C source code.  It is available, with documentation
and sample input and output, from
<A HREF="http://jcsmr.anu.edu.au/dmm/humgen/lars/treeconssub.htm">
its web site</A> at
<CODE>http://jcsmr.anu.edu.au/dmm/humgen/lars/treeconssub.htm</CODE>.
<P>

<HR>
<i>Marty Wolf is still at the same address, but TrExML no longer seems
to be distributed.</i>
<P>
<A NAME="TrExMl"><P>
Marty J. Wolf</A> of Bemidji State University, Minnesota
(<TT>mjwolf<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>cs.bemidjistate.edu</TT>) has written, and he and
Lars Sommer Jermiin distribute, <STRONG>TrExMl</STRONG>,
which searches tree space for DNA sequence data to find not only the maximum
likelihood tree but also trees of other topologies which are nearly as good.
TrExMl can also carry out bootstrapping of the sequences before doing the
analysis.
It is described in a paper: Wolf M. J., S. Easteal, M. Kahn, B. D. McKay, and
L. S. Jermiin.  2000.  TrExML: A maximum likelihood program for extensive
tree-space exploration.  <EM>Bioinformatics</EM> <B>16:</B> 383-394.
TrExMl is described in
<A HREF="http://whitetail.bemidji.msus.edu/trexml/trexml.man.html">
its web page</A> at
<TT>http://whitetail.bemidji.msus.edu/trexml/trexml.man.html</TT>.
It is distributed from there as C source code.  One use will be along with
Lars Sommer Jermiin's program <A HREF="software.old.html#TreeCons">TreeCons</A>
which computes a weighted average of trees according to their likelihood values.
<P>

<HR>
<i>Accelrys Inc. now does not offer DS Gene for sale, but instead
points to other of its products that contain analogous features.</i>
<P>

<A NAME="Accelrys Gene"><P>
Accelrys, Inc.</A>, 10188 Telesis Court, Suite 100, San Diego, California 92121,
USA (Phone: +1 858 799 5000, Fax: +1 858 799 5100) sells
<strong>Accelrys Gene</strong> version 1.5, a sequence analysis package
for Windows with the same functionality as the Macintosh package (formerly owned
by them)
<a href="MacVector">MacVector</A>.  It can do sequence search, alignment using
<a href="software.etc1.html#ClustalW">ClustalW</a>, and UPGMA or Neighbor-Joining
distance matrix methods with a variety of distance measures and with
bootstrap analysis available.  It also has
many other features including primer design, gene finding, motif searching,
protein secondary structure and hydrophobicity prediction, and prediction of
restriction digests.  It runs on Windows systems and is mentioned on the
<a href="http://accelrys.com/products/additional-products.html">web page on legacy
products</a>
at <code>http://accelrys.com/products/additional-products.html</code>.  It cannot
be ordered through that web page, but one could contact the company and ask
how to order it.  It is described as "no longer being developed as a stand-alone
product", and its functionality is described as being incorporated into two other
Accelrys products.  However neither of these is described as having the above
phylogeny functions.  The last time I knew the price, its
price for academic use was $2,500, and for commercial use $5,000.
<P>

<HR>
<P>
<i>Lars Jermiin has moved on to the CSIRO.  This program is not one of the
ones in distribution at his
software web site, which is currently still at the University of Sydney.</i>
<P>
<A NAME="K2WuLi">Lars Sommer Jermiin</A> of the School of Biological Sciences
of the University of Sydney, Australia (<TT>lars.jermiin<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>usyd.edu.au</TT>)
(formerly of the John Curtin School of
Medical Research of the University of Canberra, Australia) has released
<STRONG>K2WuLi</STRONG> version 1.0, a program to calculate the Kimura 2-parameter
distance among DNA sequences, to compute its standard deviation, to carry out
the relative rate test of Wu and Li (Wu, C.-I. and W.-H. Li. 1985. Evidence for
higher rates of nucleotide substitution in rodents than in man.
<i>Proceedings of the National Academy of Sciences, USA</i> <B>82:</B>
1741-1745) , in the form suggested by Muse and Weir (Muse, S. W. and  B. S.
Weir. 1992. Testing for equality of evolutionary rates. <i>Genetics</i>
<B>132:</B> 269-276).
The program is available as a DOS executable with Turbo Pascal course code
as well from
<A HREF="http://jcsmr.anu.edu.au/dmm/humgen/lars/k2wulitop.htm">its web page</A>
at <CODE>http://jcsmr.anu.edu.au/dmm/humgen/lars/k2wulitop.htm</CODE>.
<P>

<HR>
<i>Clare Constantine is now at the University of Melbourne's School of
Population Health, but there is no sign of distribution of GeneStrut there or
anywhere else. </i>
<P>
<A NAME="GeneStrut"><P>
Clare Constantine</A> of the Walter and Eliza Hall Institute of Medical
Research, Melbourne, Australia (<CODE>constantine<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>wehi.edu.au</CODE>)
and colleagues at the Division of Veterinary and Biomedical Sciences at
Murdoch University, Perth, Australia have written
<STRONG>GeneStrut</STRONG>, a Macintosh program which computes a range of
standard measures for the analysis of genetic
structure from discrete genetic data. The input data are multilocus genotypes.
It can calculate genotypic and allelic frequencies,
statistics for Hardy-Weinberg disequilibrium, 
genetic diversity within populations,
genetic identities between populations, and indices of population
structure (F-statistics).   For our purposes the important feature is
that it can also calculate Nei's genetic distance between populations, with
standard deviations.  It is described in the paper:
Constantine C. C., R. P. Hobbs and A. J. Lymbery. 1994.
FORTRAN programs for analysing population structure from
multilocus genotype data. <i>Journal of Heredity</i> <B>85:</B> 336-337.
It is available as a Macintosh executable, at
<A HREF="http://numbat.murdoch.edu.au/vetschl/imgad/GenStrut.htm">its web site</A>
at <CODE>http://numbat.murdoch.edu.au/vetschl/imgad/GenStrut.htm</CODE>.
<P>

<HR>
<P>
<i>This seems to no longer be available at the Pasteur Institute sites</i>
<P>
<A NAME="VeryfastDNAml"><P>
Denis Beaumont</A>
(<CODE>beaumont<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>transpac.atlas.fr</CODE>) has made a parallelized version
of <A HREF="software.etc1.html#fastDNAml">fastDNAml</A> called
<STRONG>VeryfastDNAml</STRONG>.  It is parallelized with the TreadMarks
distributed shared memory system, which is a not-quite-free environment for
parallelization that runs on many workstation-class machines.  The
C source code of VeryfastDNAml is available <A HREF="ftp://ftp.pasteur.fr/pub/GenSoft/unix/evolution/FastDNAml">by ftp</A> from the
Institut Pasteur server <CODE>ftp.pasteur.fr</CODE> in directory
<CODE>/pub/GenSoft/unix/evolution/FastDNAml</CODE> as file
<CODE>fastDNAml-tmk.tar.gz</CODE>.  There is a
<A HREF="http://bioweb.pasteur.fr/seqanal/soft-pasteur.html#veryfastdnaml">
web page access</A> to this ftp distribution at
<CODE>http://bioweb.pasteur.fr/seqanal/soft-pasteur.html#veryfastdnaml</CODE>,
which includes a link to the TreadMarks project.
<p>

<A NAME="Codeml3X">
<HR>
<P>
<i>This does not seem to be in distribution anywhere now. Hoffmann is at
Texas A&amp;M University now.  Opazo is at the Instituto Ecologia y
Evoluci&oacute;n
of the Universidad Austral de Chile. His email address is
<tt>jopazo<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;gmail.com</tt></i>
<P>
Federico Hoffmann and Juan Opazo
of the School of Biological Sciences of the University of Nebraska, Lincoln, Nebraska
<TT>(federico<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;biokubuntu.com</TT> and <TT>jopazo<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;biokubuntu.com</TT>)
have written <strong>Codeml3X</strong>, a script that runs Codeml three 
times.  It runs CODEML from <a href="software.etc1.html#PAML">PAML</A> three 
times in a row, with three different starting omega values. The script will 
create a directory and three subdirectories where the results of each run will 
be saved, and it will also create a text file with the likelihood scores of 
each tree for each run. 
It is available as Perl script.  It can be downloaded from
<a href="http://www.biokubuntu.com/enlaces.html">its web site</a>
at <code>http://www.biokubuntu.com/enlaces.html</code>
<P>

<HR>
<P>
<i>Hadtree, Prepare, and Trees seem not to be distributed from Penny's
web site (or anywhere else).  Penny continues to be at Massey University.</i>
<P>
<A NAME="Hadtree">David Penny</A> of the Institute of Molecular Biosciences,
Massey University, Palmerston North, New Zealand (<CODE>dpenny<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>massey.ac.nz</CODE>),
has made available through his Farside Institute three programs, <STRONG>Hadtree</STRONG>,
<STRONG>Prepare</STRONG>, and <STRONG>Trees</STRONG>.  These run on DOS
systems, and compute bipartition spectra by Hadamard transformations (conjugations and the
distance Hadamard), character weighting, distance transformations (including
LogDet), base composition tests, resampling schemes, and tree selection.
The programs are available from the
<A HREF="http://imbs.massey.ac.nz/Research/MolEvol/Farside/programs.htm">
Farside Institute downloads page</A> at
<CODE>http://imbs.massey.ac.nz/Research/MolEvol/Farside/programs.htm</CODE>.
<P>

<HR>
<A NAME="reticulate">
<P>
<I>I cannot locate Jakobsen or find any evidence that this program is still
in distribution.</i>
<P>
Ingrid Jakobsen</A> (currently of the Advanced
Computational Modelling Centre, University of Queensland, Australia,
<tt>ibj<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>maths.uq.edu.au</tt> and Simon Easteal
of Australian National University,
Canberra, have released <STRONG>reticulate</STRONG>.
It is a compatibility matrix program for DNA sequences that has features
designed to test for evidence of reticulate evolution (such as recombination).
The program computes and displays a pairwise compatibility matrix
for all pairs of sites.  It can randomize the order of sites and compute
the fraction of compatible sites in a region for the randomizations, to
test whether there is a pattern suggesting reticulation.
The program is distributed as C source code for Unix and X Windows, though
there are some limited ways of running it without X Windows.  It is
described in the paper:
Jakobsen, I. B. and S. Easteal. 1996.
A program for calculating and displaying compatibility matrices as an aid in determining reticulate evolution
in molecular sequences. <EM>CABIOS</EM> <B>12:</B> 291-295. It is
available from
<A HREF="http://acmc.uq.edu.au/DETYA/people/ibj/Retic/">Ingrid Jakobsen's software web site</A>
at <CODE>http://acmc.uq.edu.au/DETYA/people/ibj/Retic/</CODE>.
<P>

<HR>
<A NAME="LARD">
<P>
<I>Andrew is now at the University of Edinburgh.  His software site there
no longer distributes LARD</i>
<P>
<IMG SRC="icons/lard.gif" ALT="[LARD icon here]">
Andrew Rambaut</A> of the Department of Zoology, University
of Oxford, England (<CODE>andrew.rambaut<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zoo.ox.ac.uk</CODE>) has
produced <STRONG>LARD</STRONG> (Likelihood Analysis of Recombination in DNA)
version 2.2, a program to detect the presence of recombination in a set
of sequences.  LARD looks at the set of sequences to discover which are the
most plausible parents of a potentially recombinant sequence, and performs
a likelihood ratio test for each possible breakpoint position of whether the
three-species tree differs on the two sides of the breakpoint.  LARD is
described as an extension of a method suggested by John Maynard Smith:
Maynard Smith, J.  1992.  Analysing the mosaic structure of genes.
<EM>Journal of Molecular Evolution</EM> <B>34:</B> 126-129.  It is
described in a paper: Holmes, E. C., M. Worobey, and A. Rambaut.  1999.
Phylogenetic evidence for recombination in dengue virus.  <EM>Molecular
Biology and Evolution</EM> <B>16:</B> 405-409.  LARD is available
as C source code and as a Macintosh executable from
<A HREF="http://evolve.zoo.ox.ac.uk/software.html?name=Lard">its web site</A>
at <CODE>http://evolve.zoo.ox.ac.uk/software.html?name=Lard</CODE>.
<P>

<HR>
<P>
<i>The Mark Gibbs / Adrian Gibbs lab is still around (their web site is
<a href="http://biojanus.anu.edu.au/">here</a>) but they seem not to distribute
SiScan any more</i>
<P>
<A NAME="siscan"><P>
Adrian Gibbs</A> (<TT>Adrian.Gibbs<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>anu.edu.au</TT>) of the
Department of Botany and Zoology of the Australian National University,
Canberra, has written <B>SiScan</B>, version 2.0, a program that scans
3 or 4 DNA sequences
for evidence of recombination.  Two of the sequences are the putative parent
sequences, one the putative recombinant, and one an outgroup.  The program uses
a Monte Carlo randomization procedure to test for recombination signal.
The program can
be downloaded as an archived Windows executable (that's what I assume it is,
the web site doesn't say) from <A HREF="http://www.anu.edu.au/BoZo/software/index.html">the department software distribution web site</A> at <TT>
http://www.anu.edu.au/BoZo/software/index.html</TT>.  SiScan is
described in a paper: Gibbs, M. J., J. S. Armstrong, and A. J. Gibbs.  2000.
Sister-Scanning: a Monte Carlo procedure for assessing signals in recombinant
sequences.  <EM>Bioinformatics</EM> <B>16:</B> 573-582.
</TT>
<P>

<HR>
<P>
<i>This program does not seem to be currently distributed either at Worobey's 
department (Ecology and Evolutionary Biology at the University of Arizona), or 
Rambaut's department (the Institute of Evolution at the University of Edinburgh,
U.K.).</i>
<P>
<A NAME="PIST"><P></A><IMG SRC="icons/pist.gif" ALT="[PIST icon]">&nbsp;Michael
Worobey, of the Department of Ecology and Evolution, University of Arizona,
Tucson, Arizona (<tt>worobey<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>email.arizona.edu</tt>) and Andrew
Rambaut, of the Department of Zoology, University of Oxford
(<tt>andrew.rambaut<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zoo.ox.ac.uk</tt>) have written <strong>PIST</strong>
(<em>P</em>hylogenetic <em>I</em>nformative <em>S</em>ites <em>T</em>est)
version 1.0, a program to perform this test.  The program simulates multiple
data sets up a given tree, and then computes, for each of these and for an
original data set, a statistic which is the proportion of sites that have
two states and fit the tree perfectly.  This statistic will be inflated in
the original data if there are recombination events in its genealogy.
The program is available as a Mac OS executable and as source code for
Unix (which can also be compiled on Windows or on Mac OS X).  It is
distributed from
<A HREF="http://evolve.zoo.ox.ac.uk/software.html?id=pist">its web page</A>
at <CODE>http://evolve.zoo.ox.ac.uk/software.html?id=pist</CODE>
<P>

<HR>
<A NAME="TreeMos">
<i>The website is still there, but currently the download is listed as &ldquo;unavailable at this
time&rdquo;</i>.
<P>
Jonathan Moore and Robin Allaby</A> of the Warwick HRI
at the University of Warwick, UK
<tt>(jonathan.moore<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;warwick.ac.uk)</tt>
has released <strong>TreeMos</strong>
version 1.0, a package
for search and visualisation of phylogenetic mosaicism,
which identifies anomalous nearest-neighbour 
relationships of segments in multiple multiple alignments.  It allows the user 
to search for phylogenetic mosaicism in a group of DNA or protein sequence 
multiple alignments or genome sequences.  TreeMos uses a sliding window and 
local alignment and tree-building algorithms
(<a href="software.etc1.html#ClustalW">ClustalW</a>)
to identify sequence segments 
whose nearest neighbour is anomalous to that identified using the whole 
alignment, and visualizes that relationship where the anomalous neighbour may 
come from anywhere in the data set.  TreeMos can import a group of alignments 
in FASTA format, identify instances of phylogentic mosaicism within and 
between alignments, and display graphical representations of the results in a 
web browser.  The methods are described in the paper:
Allaby, R.G. and M. Woodwark.  2007.  Phylogenetic analysis reveals extensive
phylogenetic mosaicism in the Human GPCR superfamily. <em>Evolutionary
Bioinformatics</em> <b>3:</b> 155-168.
It is available as a Perl script and Intel Mac OS X executables.
It can be run as a command line program, but also requires a local Apache
installation for its GUI functionc.  TreeMos can be downloaded from
<a href="http://www2.warwick.ac.uk/fac/sci/whri/research/archaeobotany/downloads/">its web site</a>
at <code>http://www2.warwick.ac.uk/fac/sci/whri/research/archaeobotany/downloads/</code>
<P>

<HR>
<i>This program seems mostly to have been superseded by Nielsen and Jody Hey's
later programs IMA and IMa2 and it is not currently in distribution. Nielsen
is now in the Department of Integrative Biology at the
University of California, Berkeley.</i>
<P>
<A NAME="mdiv"><P></A> Rasmus Nielsen, of the Bionformatics Centre at the
University of Copenhagen, Denmark (<tt>rasmus&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>binf.ku.dk</tt>)
has written <strong>MDIV</strong>, a program that will simultaneously estimate
the divergence time and the migration rates between two populations.  It can
use either an infinite-sites model or an HKY sequence evolution model.  It can
test whether the evidence supports historical divergence, migration between
the populations, or both, and make maximum likelihood estimates and likelihood
surfaces for the parameters.  It assumes equal population sizes in the
two populations and in their ancestors.  It is decsribed in a paper:
Nielsen, R. and J. W. Wakeley. 2001. Distinguishing migration from isolation:
an MCMC approach. <em>Genetics</em> <b>158:</b> 885-896.  It is distributed
as a Windows executable from
<a href="http://www.binf.ku.dk/users/rasmus/webpage/programs.html#MDIV">Nielsen's programs web site</a>
at <code>http://www.binf.ku.dk/users/rasmus/webpage/programs.html#MDIV</code>
<P>

<HR>
<A NAME="MCS">
<P>
<i>Johan Nylander's website is now <a
href="http://www.abc.se/~nylander/">here</a></i>
but he is not distributing MCS currently.
<P>
Johan Nylander (<tt>Johan.Nylander<!deathtospam> 
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>abc.se</tt>)
has written <strong>MCS</strong> version
1.0, a program that reads the output of boostrap or jackknife analyses in
<a href="software.pars.html#PAUP">PAUP*</a> and computes the Mean Character
Support statistic from them.  It is available as a Windows or Mac OS X
executable or as source code from 
<a href="http://www.ebc.uu.se/systzoo/staff/nylander.html">Nylander's
software download site</a> at
<code>http://www.ebc.uu.se/systzoo/staff/nylander.html</code>in Sweden
<P>

<HR>
<A NAME="PHYCON"><P>
<i>McInerney is still in Maynooth but is no longer distributing this
program, probably because programs like PAML can now bootstrap.  And
why did I write that this "allows bootstrapping within PAML", when it actually
used MOLPHY?</i>
<P>
James McInerney</A> of the Department of Biology of the
National University of Ireland, Maynooth, County Kildare, Ireland
(<TT>james.o.mcinerney<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>may.ie</TT>) and also of the Department of Zoology
of the Natural History Museum, London, U.K. (<TT>j.mcinerney<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>nhm.ac.uk</TT>)
has written <STRONG>PHYCON</STRONG>, a program which takes as input
bootstrapped molecular data sets, as produced by
<A HREF="software.pars.html#PHYLIP">PHYLIP</A> and feeds them to
<A HREF="software.etc1.html#MOLPHY">MOLPHY</A> programs.  This allows bootstrapping
within PAML.  The program is available as C source code for Unix; it will not
work on a Windows system or under Mac OS (though it can under Mac OS X).
Source code and documentation are available from
<A HREF="http://www.bioinf.org/vibe/software/phycon/phycon.html">its web site</A>
at <CODE>http://www.bioinf.org/vibe/software/phycon/phycon.html</CODE>.
<P>

<HR>
<P>
<I>Dopazo is now the head of the department of Bioinformatics and Genomics at
the Centro de Investigaci&oacute;n Pr&iacute;ncipe Felipe in Valencia, Spain.   He does not
seem to have WET available for distribution.</I>
<P>
<IMG SRC="icons/wet.gif" ALT="[WET icon]">
<A NAME="WET">Joaquin Dopazo</A> of
the Bioinformatics department of GlaxoWellcome SA, Spain
(<CODE>jd19662<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>glaxowellcome.co.uk</CODE>) has written
<STRONG>WET</STRONG> (Windows Easy Tree), version 1.3, which is an easy-to-use program
for inferring phylogenies from sequence data by distance matrix methods.
The main goal in the development of WET was to make a really user friendly
program able to interact with other phylogenetic packages.
WET can import files of a number of different formats.  It
calculates distances by a number of different methods and constructs
phylogenetic trees using neighbor-joining, UPGMA and WPGMA procedures.  It is a
Windows executable.  It is
available from <A HREF="http://www.cnb.uam.es/~dopazo/software/wet.html">
its web site</A> at <CODE>http://www.cnb.uam.es/~dopazo/software/wet.html</CODE>.
<P>

<HR>
<A NAME="TAAR"><P>
<I>Jiang is still at the same address but does not seem to distribute TAAR
from there.  He has a link to the old TAAR distribution (at McMaster
University) but it is not available there.</i>
<P>
Feng Liu and Tao Jiang</A> (currently of the Department of Computer Science and Engineering at the
University of California, Riverside, <CODE>jiang<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>cs.ucr.edu</CODE>),
have written <STRONG>TAAR</STRONG> (Tree Alignment And Reconstruction),
version 1.0, which constructs multiple
sequence alignment and phylogenies based on the idea of tree alignment.
It is a graphical environment capable of "approximately optimal"
parsimony-based tree alignment.  It can also infer trees by parsimony.
It can handle DNA or protein data.
It is available as C source code for Unix with X Windows (X11R5) and MOTIF 1.2.
It is also available in a version for Linux with Lesstif.
These are distributed through
<A HREF="http://www.cas.mcmaster.ca/~jiang/taar/">its home page
</A> at <CODE>http://www.cas.mcmaster.ca/~jiang/taar/</CODE>.
<P>

<HR>
<P>
<A NAME="Ctree">
<i>David States is currently in the Brown Foundation Institute of Molecular
Medicine at the University of Texas at Houston, but CTREE seems not to be in
distribution there.</i>
<P>
David States</A> (<CODE>dstates<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>umich.edu</CODE>)
of the Department of Human Genetics and Bioinformatics at the University
of Michigan, Ann Arbor, Michigan has released
<STRONG>Ctree</STRONG> version 1.0., a tree alignment program that
uses a Hidden Markov Model method of representing the ambiguities in
alignments of groups of sequences.
The Ctree program is based on a neighbor joining algorithm in which
sequences and groups of sequences are represented by Hidden Markov
Models.  HMMs are aligned using a Smith/Waterman dynamic programming
algorithm to find the best local alignment.  At
each step in building the MSA alignment tree, the highest scoring pair
of HMMs are merged into a new HMM.  In this sense it is similar to the
progressive alignment algorithm used in <A HREF="#ClustalW">ClustalW</A>, but
the use of an HMM to represent clusters retains more information about the
ambiguities than the Clustal algorithm does.  It is possible to write the
tree to a dendrogram output file.
<P>
Ctree is available <A HREF="ftp://www.ibc.wustl.edu/pub/ctree">by anonymous ftp</A>
from <CODE>www.ibc.wustl.edu</CODE> in
directory <CODE>pub/ctree</CODE>.  It is available as C source code and also as
executables for Sun Solaris, SGI, Linux, and Windows (in version 1.1).  Ctree is
also useable as a server but that version seems not to give trees as output.
<P>

<A NAME="bootscore">
<HR>
<I>(Sukumaran has apparently withdrawn bootscore.  He says that his program 
SumTrees, which is part of his <a href="software.pars.html#DendroPy">DendroPy</a> Python package, is its replacement.)</i>
of the 
<P>
Jeet Sukumaran</A>
of the Division of Herpetology of the University of Kansas Natural History Museum and Biodiversity
Research Center at the University of Kansas, Lawrence, Kansas
<tt>(jeetsukumaran<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;frogweb.org)</tt>
distributes <strong>bootscore</strong>
version 3.11, a program to compute bootstrap support from boostrap replicate 
trees and place them on a consensus tree.  A platform-independent Python 
script maps non-parametric bootstrap support for clades onto a 
phylogenetic tree. It outputs a NEXUS/Newick treefile with the topology of the 
given tree and with clade support indicated by node labels or branch lengths. 
In its default bipartition-counting mode, it identifies all distinct 
bipartitions in the tree to be evaluated, and then scans through a file of 
bootstrap replicates to identify the percentage or proportional frequency of 
occurance of each of those bipartitions in each of the bootstrap trees. It can 
also operate in a clade-counting mode, in which it identifies all distinct 
monophyletic groups in the tree being assessed, and then counts the number of 
bootstrap trees in which that particular monophyletic group is recovered.
It is available as a Python script.  It can be downloaded from
<a href="http://bootscore.sourceforge.net/">its web site</a>
at <code>http://bootscore.sourceforge.net/</code>
<P>

<A NAME="Ultrametric Check">
<HR>
<I>Purvis is still in the same location, but there seems to be no
distribution of this helper application currently.</i>
<P>
Andrew Purvis's laboratory</A>
of the Department of Biological Sciences
at  Imperial College, Silwood Park, U.K.
<tt>(a.purvis<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;imperial.ac.uk)</tt>
have produced <strong>Ultrametric Check</strong>, a utility to check the
clockness of a tree.  It calculates the total root-to-tip distance for 
all tips in a phylogeny. It reads <a href="software.etc2.html#CAIC">CAIC</a> format 
<tt>.Phyl</tt> and <tt>.Blen</tt> files and works out whether the tips line up 
(whether the tree is ultrametric).
It is available as Mac OS 9 executables.  It can be downloaded from
<a href="http://www.bio.ic.ac.uk/evolve/software/index.html#ultracheck">its web site</a>
at <code>http://www.bio.ic.ac.uk/evolve/software/index.html#ultracheck</code>
<P>

<HR>
<i>Neither Em&iacute;lia Martins nor Ted Garland currently distributes
CMAP.</i>
<P>
<A NAME="CMAP"><P>
Em&iacute;lia Martins</A> (<CODE>emartins<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>indiana.edu</CODE>), of the
Department of Biology of the University of Indiana, Bloomington, Indiana,
has written <STRONG>CMAP</STRONG>, the
Comparative Method Analysis Package, for comparative methods analysis.  
This package was developed when she and Ted Garland were conducting the
simulation study described in the paper: 
Martins, E. P. and T. Garland, Jr. 1991. Phylogenetic analyses of the correlated evolution of continuous characters: a simulation study. <EM>Evolution</EM>
<B>45:</B> 534-557.
It can be used to estimate the correlation between two continuous characters
measured in different species while taking phylogenetic information into account. Methods for doing so include several versions of Felsenstein's (1985) independent contrasts, and the sum-of-squared-changes parsimony algorithm. 
The programs in CMAP are described by Martins as "slow" and "unfriendly".
The executables are available only for DOS machines. She is no longer
developing this package, and is now concentrating her efforts on
her other package <A HREF="#COMPARE">COMPARE</A>, which will be able to
do everything that CMAP can.  CMAP is available
from <A HREF="http://compare.bio.indiana.edu/ftp/">its download area </A> at <CODE>http://compare.bio.indiana.edu/ftp/</CODE>.
<P>

<HR>
<i>Lindenfors no longer distributes CoSta.</i>
<P>
<IMG SRC="icons/CoSta.Gif" ALT="[CoSta icon here]">

<A NAME="CoSta">Patrik Lindenfors</A> (<CODE>Patrik.Lindenfors<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zoologi.su.se)</CODE>, of the Department of Zoology, Stockholm University Sweden, and
the Department of Biology, University of Virginia, Charlottesville (<tt>Patrik.Lindenfors<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>virginia.edu</tt>), has written
<STRONG>CoSta</STRONG> version 1.03, a DOS program which carries out the Contingent
States Test for the correlation of changes in two characters along a tree, which is described in the paper: Sill&eacute;n-Tullberg, B. 1993.
The effect of biased inclusion of taxa on the correlation between discrete
characters in phylogenetic trees.  <EM>Evolution</EM> <B>47:</B> 1182-1191.
The program reads MacClade data files, and also text files saved from
MacClade.  The program can be fetched at
its <A HREF= "http://www.zoologi.su.se/research/Lindenfors/CoSta.html">
Web site</A> at <CODE>http://www.zoologi.su.se/research/Lindenfors/CoSta.html</CODE>.
<P>

<A NAME="ACAP">
<HR>
<I>ACAP 2 seems to have been withdrawn by the Ackerley Lab (he is now at
University of California, Berkeley) as its features are available in a more
recent package, <a href="software.etc2.html#Cactus-Pie">Cactus-Pie</a></i>
<P>
David Ackerly</A> (<CODE>dackerly<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>stanford.edu</CODE>)
of the Department of Biological Sciences, Stanford University, Stanford,
California has released <STRONG>ACAP 2</STRONG> (Another Comparative Analysis
Program) to carry out independent contrasts methods for comparative
analysis.  It also also incorporates linear parsimony methods into
the program, in order to calculate consistency indices for continuous
characters.  The program is written in Think Pascal for Macintosh Mac OS
systems, and is available from
<A HREF="http://www.stanford.edu/~dackerly/ACAP.html">its web site</A>
at <CODE>http://www.stanford.edu/~dackerly/ACAP.html</CODE>
as a Macintosh executable which will run on 68k Macintosh or PowerMacintosh
Mac OS computers. 
<P>

<HR>
<A NAME="CLINCH"><P>
<I>The Digital Taxonomy listing has disappeared, and with it the last place
that CLINCH could be downloaded.  Kent Fiala works for SAS Institute and
can be reached by email at
<tt>fiala<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ipass.net</tt></i>
<P>
Kent Fiala</A>
<TT>(fiala<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>ipass.net)</TT> (most recently of SAS Institute) produced <STRONG>CLINCH</STRONG>
(CLadistic INference by Compatibility of Hypothesized characters) version 6.2.
It is a general-purpose compatibility program capable of handling multiple
unordered states.  It is available as a DOS executable, including FORTRAN
source code, from the
<A HREF="http://www.geocities.com/RainForest/Vines/8695/software.html#Cladistics">
Digital Taxonomy web page</A> at
<CODE>http://www.geocities.com/RainForest/Vines/8695/software.html#Cladistics</CODE>.
<P>

<HR>
<A NAME="galaxie">
<i>Nilsson is in Gothenburg and Ursing is now working for an investment
company.   galaxie  does not seem to be in distribution.</i>
<P>
Henrik Nilsson and Bj&oslash;rn Ursing</A>
of the Botanical Institute
at the G&oslash;teborg University and the Center for Genomics and
Bioinformatics at the Karolinska Institute, Stockholm, Sweden
(<tt>henrik.nilsson<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;botany.gu.se</tt>
and <tt>bjorn.ursing<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;cgb.ki.se</tt>)
have produced <strong>galaxie</strong>, a a package of CGI scripts for sequence
identification through automated phylogenetic analysis.  galaxie is <a
href="software.serv.html#galaxie server">a server</a>,
but also makes its scripts available for download.  It is intended for
identification of fungal EST sequences.  It uses
BLAST, <a href="software.etc1.html#Clustal">ClustalW</a> and
<a href="software.pars.html">PHYLIP</a> to find a set of best matches to the
EST sequence, then make a phylogeny of these matches and the original sequence
to help identify the sequence.  The CGI scripts require that the user who
installs them be familiar with such scripts and have a web server, a  and also
have BLAST, ClustalW and PHYLIP installed on their computer.
It is described in the paper:
Nilsson, R. H., K.-B. Larsson, B. M. Ursing. 2004. galaxie - a CGI script
package for sequence identification through automated phylogenetic analysis.
<em>Bioinformatics</em> <b>20:</b> 1447-1452.
It is available as a package of Perl scripts.  It can be downloaded from
<a href="http://galaxie.cgb.ki.se/">its web site</a> at
<code>http://galaxie.cgb.ki.se/</code>
<P>

<A NAME="Bi-De">
<HR>
<P>
<i>Rambaut is now at the Institute of Evolutionary Biology at the
University of Edinburgh.  Bi-De seems not to be still in distribution, probably
because he considers his more recent program
<a href="software.etc2.html#Phyl-O-Gen">Phyl-O-Gen</a> to supersede it.</i>
<P>
<IMG SRC="icons/Bi-De.gif" ALT="[Bi-De icon here]">&nbsp;
Andrew Rambaut</A> of the Department of Zoology,
University of Oxford, (<CODE>andrew.rambaut<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zoo.ox.ac.uk</CODE>)
has written <STRONG>Bi-De</STRONG> version 0.1,
to simulate the evolution of trees using various models of lineage
birth and death, and sampling lineages from among those extant.
It can simulate branching with or without regulation of the number of
lineages.  It also allows the user to specify the relationship between
the number of lineages and the birth rate of lineages.
The program is available free for MacOS system 7.0 or later,
from
<A HREF= "http://evolve.zoo.ox.ac.uk/software.html?name=Bi-De">its web page</A>
at
the University of Oxford Zoology software site at
<CODE>http://evolve.zoo.ox.ac.uk/software.html?name=Bi-De</CODE>.  Its
manual can also be viewed on-line at that site.  Rambaut says that Bi-De
is considered obsolete software, having been almost completely superseded
by their later program Phyl-O-Gen.
<P>

<A NAME="Hetero">
<HR>
<i>Jermiin is at this place.  The web page for Hetero does exist in this
location but the download links are not functional.</i>
<P>
Lars Jermiin</A> of
CSIRO Entomology, Black Mountain Laboratories, Canberra, Australia
(<TT>lars.jermiin<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>csiro.au</TT>)
wrote (when he was at the University of Sydney) <strong>Hetero</strong>, 
version 1.0, a program to
simulate evolution of DNA sequences on four-species trees.
The program allows many different kinds of heterogeneity of processes and
rates, including different models of change on different branches.  It gives
a number of different kinds of summaries of the properties of the resulting 
sequences, as well as writing them to files for use by other programs.
It is described in a paper: Jermiin, L. S., S. Y. W. Ho, F. Ababneh, J.
Robinson, and A. W. D. Larkum. 2003. Hetero: a program to simulate the
evolution of DNA
on a four-taxon tree. <em>Applied Bioinformatics</em> <b>2:</b> 159-163.
It is distributed as executables for Sun Solaris, for Windows, and for
Mac OS X from <a href="http://www.bio.usyd.edu.au/about_us/honorary_staff/jermiin_lars/hetero.shtml">its web
site</a>
at <code>http://www.bio.usyd.edu.au/about_us/honorary_staff/jermiin_lars/hetero.shtml</code>.  Source code
is also offered to those users who obtain a license for use of source
code from the book <em>Numerical Recipes</em>.
<P>

<A NAME="Diversi">
<HR>
<P>
<i>... and indeed, Diversi </i>has <i>been superseded by a comparable
function within <a href="software.etc2.html#APE">APE</a>, as Paradis
promised.</i>
<P>
Emmanuel Paradis</A> (<tt>paradis<! death to spam>&nbsp;(at)<! death to
spam>&nbsp;ird.fr</tt>), of the
Institut de recherche pour le d&eacute;veloppement, Marseille, France
has released <STRONG>Diversi</STRONG> version 0.20.0, a program for
the analysis of diversification using phylogenetic data.  It uses
several methods to estimate and test for variations in
diversification rates using phylogenetic data, including
tests for temporal or among-clade variations in diversification rates
using a maximum likelihood method.  The program takes divergence times as
its input.  It can also simulate the branching of trees. 
he tests are described in a paper:
Paradis, E. 1997. Assessing temporal variations in diversification rates from
phylogenies: estimation and hypothesis testing. <EM>Proceedings of the Royal
Society of London B</EM> <B>264:</B> 1141-1147.
It is available as FORTRAN source code and also as a Windows executable, from
<A
HREF="http://www.isem.univ-montp2.fr/PPP/PPPphylogenie/ParadisHome.php#softwares">Paradis's
software web page</A>
at
<code>http://www.isem.univ-montp2.fr/PPP/PPPphylogenie/ParadisHome.php#softwares</code>
or
<A HREF="ftp://evol.isem.univ-montp2.fr/pub/pc/Log-manu">by ftp</A> from
<CODE>evol.isem.univ-montp2.fr</CODE> in
directory <CODE>/pub/pc/Log-manu</CODE>.  Paradis describes DIVERSI as
no longer developed or maintained, and soon to be superseded by his later
program <A HREF="software.etc2.html#APE">APE</A>.
<P>

<A NAME="MULTIDIVTIME_HELPER">
<HR>
<P>
<i>Opazo is at the Instituto Ecologia y
Evoluci&oacute;n
of the Universidad Austral de Chile. His email address is
<tt>jopazo<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;gmail.com</tt>.
Biokubuntu seems to have disappeared, and with it the software distribution.</i>

<P>
Juan Opazo</A>
of the School of Biological Sciences
of the University of Nebraska, Lincoln, Nebraska
<tt>(jopazo<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;biokubuntu.org)</tt>
has written <strong>MULTIDIVTIME HELPER</strong>, a Perl script to create a 
tree file for constraints for use by Multidivtime.  It is designed to 
facilitate the estimation of divergence times using
<a href="software.etc2.html#Multidivtime">Multidivtime</a>. This script will 
create a directory and three subdirectories where the results of each program 
(baseml, estbranches and multidivtime) will be saved, and it will also create 
a tree file with node labels to assign constraints, this file has been 
designed to be opened in <a href="software.etc2.html#TreeView">TreeView</a>.
The script needs <a href="software.etc1.html#PAML">baseml</A>, paml2modelinf,
estbNew, and <a href="software.etc2#Multidivtime">Multidivtime</a> to run.
It is available as Perl script.  It can be downloaded from
<a href="www.biokubuntu.com/enlaces.html">its web site</a>
at <code>www.biokubuntu.com/enlaces.html</code>
<P>

<HR>
<A NAME="Rhino">
<P>
<I>Andrew Rambaut is now at the University of Edinburgh.  He has ceased
distribution of Rhino.</I>
<P></A>
<IMG SRC="icons/rhino.jpg" ALT="[Rhino icon]">&nbsp; Andrew Rambaut, of the
Department of Zoology, University of Oxford (<tt>andrew.rambaut<!deathtospam>
&nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>zoo.ox.ac.uk</tt>)
has released <strong>Rhino</strong> version 1.2.  The program uses nucleotide
sequences or
protein sequences and a given estimate of the tree, and fits a wide variety
of models of sequence evolution.  These include DNA, amino acid, and codon
models, as well as versions of these with nodes of known time, serial
samples, molecular clocks, and relaxed molecular clocks.  It is available
as a Mac OS X or Mac OS 9 executable, and as source code for Unix (which
can also be compiled to work on Windows or Mac OS X).  It is available at
<A HREF="http://evolve.zoo.ox.ac.uk/software.html?id=rhino">its web page</A>
at <CODE>http://evolve.zoo.ox.ac.uk/software.html?id=rhino</CODE>
<P>

<A NAME="TreeIllustrator">
<HR>
<P>
<i>(Frederik Decoutterre may still be with Genohm but there is no sign
there that this is one of their products, and the TreeIllustrator web
site is gone, replaced by a spam fake web site).</i>
<P>
<IMG SRC="icons/treeillustrator.gif" ALT="[TreeIllustrator icon]">&nbsp;&nbsp;
Frederik Decouttere</A>
of Genohm, a
spin-off of the University of Ghent in Belgium (Department of Molecular
Biology)
<tt>(info<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;genohm.com)</tt>
has released <strong>TreeIllustrator</strong>
version 1.6, a program for displaying and manipulating phylogenetic trees.
It allows you to customise your phylogenetic trees and compare them with the
current classification of organisms.  It can import NEXUS and Newick tree
files, including multiple trees.  Nodes can be dragged with the mouse,
names changed,
graphical properties of the image can be changed, the root can be changed,
and subtrees extracted.  There is a browser which accesses the Tree Of Life
and can point out inconsistencies between your tree and that one.
It is described in the paper: Trooskens, G., D. De Beule, F. Decouttere, and
W. Van Criekinge   2005.  Phylogenetic trees: visualizing, customizing and
detecting incongruence. <em>Bioinformatics</em> <b>21(19):</b> 3801-3802.
It is available as Java executables.  There is a free Standard Edition
which can handle up to 100 tips,
a more capable commercial Enterprise Edition, which
can handle very large trees, and
is available at a price of 240 Euros from Genohm.  It is described at
<a href="http://www.genohm.com">the Genohm web site </a>
at <code>http://www.genohm.com</code>, and the
Standard Edition can be downloaded from
<a href="http://www.treeillustrator.com">its web site</a>
at <code>http://www.treeillustrator.com</code>
<P>

<A NAME="RPT">
<HR>
<P><i>This package seems to no longer be downloadable.  All that is available at
the CEH GeneSwytch site is the web server version.</i>
<P>
Paul Swift and Dawn Field</A> of the Centre for Ecology and Hydrology,
Oxford, U.K.
<tt>(pswi<!deathtospam>&nbsp;(at)<!deathtospam>&nbsp;ceh.ac.uk)</tt>
have released <strong>RPT</strong> (Rosette Plotter Tool),
version 2.01, that permits visual comparison of multiple continuous
characteristics.  The Rosette Plotter Tool reads the tree and
produces circular plots ("rosettes") 
which combine relational data with multiple descriptive datasets for ready
comparison.  The data can be taxonomic, phylogenetic or categorical
(eg. habitat type).  Descriptive data can be discrete or continuous, and are
displayed as relative frequencies.
The program accepts NEXUS, PHYLIP and CLUSTAL trees. It
accepts datasets describing over 2000 items with up to 20 measured characters.
It is available as a Perl script.  It can be downloaded from
<a href="http://www.genomics.ceh.ac.uk/GeneSwytch/gnuplot/">its web site</a>
at <code>http://www.genomics.ceh.ac.uk/GeneSwytch/gnuplot/</code>.  It is
also available as a <a href="software.serv.html#RPT_server">web server</a>.
<P>

<P>
<hr>
<p>
<CENTER>
<TABLE WIDTH=60%>
<TR ALIGN=CENTER>
<TD BGCOLOR="navy"><STRONG><FONT SIZE=+3 COLOR="cyan">D</FONT><FONT SIZE=+2 COLOR="cyan">efunct web servers</FONT></STRONG></TD>
</TR>
</TABLE>
</CENTER>
<p>
The following servers have disappeared:
<UL>
<LI><i>This server no longer responds and that unit has no pointers to
these programs on its web pages.  Zharkikh is currently with Myriad
Genetics in Utah.  I do not know of any place distributing the programs.
</i><BR>
<A NAME="Zharkikh boot">Andrey Zharkikh</A>
<CODE>zharkikh<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>hgc6.sph.uth.tmc.edu</CODE>)  of the Genetics Centers at  the  University  of  Texas
Health  Sciences Center in Houston has <STRONG>programs for double-bootstrapping</STRONG> of nucleotide
sequences, using his innovative complete-and-partial bootstrap method for getting  less
biased  P values.  They are available free by
<A HREF="http://hgc6.sph.uth.tmc.edu:8080/CP-bootstrap.dir">World Wide Web</A>
at
<CODE>http://hgc6.sph.uth.tmc.edu:8080/CP-bootstrap.dir</CODE>,
or by anonymous ftp at
<CODE>hgc6.sph.uth.tmc.edu/pub/zharkikh/double-bootstrap</CODE>.
His technique is described in the paper by Zharkikh, A. and W.-H. Li. 1995.
Estimation of confidence in phylogeny: Complete-and-Partial bootstrap technique.
<i> Molecular Phylogenetics and Evolution</i> <B>4:i</B> 44-63 and
briefly reviewed in Li, W.-H. and A. Zharkikh. 1995.  Statistical tests of DNA
phylogenies.  <i>Systematic Biology</i> <B>44:</B> 49-63.
<LI><IMG SRC="icons/hp.gif" ALT="[HyperPars icon]">
<A NAME="HyperPars">Allan Dickermann, then of the University of Arizona, had a
server version of his program <STRONG>HyperPars</STRONG> which analyzed
data with recombining sequences, reconstructing histories that included
recombinations.
<LI><i>The sUPGMA server has been discontinued as the sUPGMA program has
been discontinued (see its listing above).</i><BR>
<A NAME="supgma-serv">Allen Rodrigo</A> of the Computational and Evolutionary
Biology Laboratory, School of Biological Sciences, University of Auckland,
Auckland, New Zealand has
<A HREF="http://www.sbs.auckland.ac.nz/rg/ecol_evol/cebl/supgma_applet.html">
A server</A> which runs <STRONG>sUPGMA</STRONG> (Serial sample UPGMA).  It
reconstructs evolutionary histories/genealogies from
distance matrices under the assumption of
a molecular clock when sequences are obtained serially in time.
It can also carry out UPGMA and WPGMA clustering as well as Neighbor-Joining.
the Java code for <A HREF="software.dist.html#supgma">sUPGMA</A>
is also available.
<LI><i>Six of the servers that carried out PHYLIP analyses have now disappeared:</i>
<UL>
<LI><A NAME="Schering-Plough-PHYLIP">The Schering-Plough Research Institute at
Boston University, Boston,
Massachusetts has <A HREF="http://cyrus.bu.edu/phylip/index.html">a
server</A> available using the Singapore software at
<CODE>cyrus.bu.edu/phylip/index.html</CODE>.
<LI><A NAME="Wistar-PHYLIP">The Wistar Institute</A> at the University of
Pennsylvania, in Philadelphia, has made PHYLIP available
<A HREF="http://sing.wistar.upenn.edu:8888/phylip/index.html">
as a server</A> using the
Singapore software at
<CODE>http://sing.wistar.upenn.edu:8888/phylip/index.html</CODE>.
<LI><A NAME="Singapore-PHYLIP-server">The Internet Bioinformatics Group of the Internet Research
and Development Unit of the National University of Singapore has
<A HREF="http://sdmc.krdl.org.sg:8080/~lxzhang/phylip/">a server</A> at
<CODE>http://sdmc.krdl.org.sg:8080/~lxzhang/phylip/</CODE>. <i>(There is a
different server now in Singapore -- see the list of servers).</i>
<LI><A NAME="CNB-PHYLIP">Joaquin Dopazo at the <A HREF="http://www.cnb.uam.es/cgi-bin/dopazo/phylip/phylipwww">Centro Nacional de Biotecnologia</A>, Madrid, Spain has made a server
available.
<LI> The University of Nebraska at Lincoln made available
the <a href="http://biocore.unl.edu/WEBPHYLIP/">WEBPHYLIP</a> server originally
constructed by Louxin Zhang at the National University of Singapore.
<LI> The Canadian Bioinformatics Resource at the NRC Institute for Marine
Biosciences, Halifax, Nova Scotia made available a
<a href="http://www.cbr.nrc.ca/cgi-bin/WebPhylip/index.html">WEBPHYLIP</a> server as well.
</UL>
<LI> <i>This VOSTORG server no longer seems to exist. See the VOSTORG entry
above for the status of the VOSTORG software package.</i><br>
<A NAME="VOSTORG-serv">The Section of Molecular Evolution</A> of the
Institute of Cytology and Genetics in Novosibirsk, Russia has <A HREF=
"http://molevol.bionet.nsc.ru/www_vs.htm">a server</A>
at <CODE>http://molevol.bionet.nsc.ru/www_vs.htm</CODE>
for the <A HREF="software.etc1.html#VOSTORG">VOSTORG</A> package.  The
server can compute distances and find UPGMA, WPGMA, or Neighbor-Joining
phylogenies.
<LI><i>This server seems to have gone inactive, probably long ago when the
RDP project moved to Michigan State University.</i><br>
<A NAME="Sgtree"> <IMG SRC="icons/sgtree_but.gif" ALIGN=BOTTOM ALT="[Suggest-Tree icon here]"></A>
<A HREF="http://geta.life.uiuc.edu/RDP/commands/sgtree.html">The Suggest Tree function</A> of
The old web page of the
Ribosomal Database Project at the University of Illinois
(the main project has now moved to Michigan State University but so far has
not yet implemented this server on its web page there).  You submit a
large- or small-subunit ribosomal RNA sequence and this server will align it
with their database of sequences, place it in the best position it can on the
tree of those sequences, and return the nearby part of that tree.  The server
returns the results by email.
<LI><i>Robertson is now at University of Manchester (<tt>david.robertson (at)
manchester.ac.uk</tt>) but has not resumed the Frag_Tree/Tree server there.
</i><br>
<A NAME="TREE">David Robertson</A> (<CODE>david<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>igs.cnrs-mrs.fr</CODE>)
of the CNRS in Marseille has implemented <STRONG>TREE</STRONG> and
<STRONG>FRAG_TREE</STRONG>,
Neighbor-Joining servers that take a ClustalW .ALN file of aligned
sequences, use <A HREF="software.etc1.html#ClustalW">ClustalW</A> to align them and my program DRAWTREE to draw
the resulting tree.  TREE does the whole alignment, and FRAG_TREE a
specified region of the alignment.  The
<A HREF="http://193.50.234.246/~beaudoin/anrs/Tree.html">TREE server</A> will be
found at <CODE>http://193.50.234.246/~beaudoin/anrs/Tree.html</CODE> and the
<A HREF="http://193.50.234.246/~beaudoin/anrs/Frag.html">FRAG_TREE server</A> will be
found at <CODE>http://193.50.234.246/~beaudoin/anrs/Frag.html</CODE>.
<LI><i>This URL suddenly went dead though Ree does have pages near there
that describe his software</i><br><a name="Drawtree server"> Rick Ree of the Field Museum of Natural History
in Chicago has <a href="http://www.phylodiversity.net/~rick/drawtree/index.html">a server</a> which will draw Postscript or PDF figures of
trees whose nested-parenthesis reprewsentation you provide.  It uses the
<a href="software.pars.html#PHYLIP">PHYLIP</a> program Drawgram rather then
Drawtree.
<LI> The Cladogramer server at the Bioinformatics Center in Singapore seems
to have disappeared (again).  It inferred trees of haplotypes by parsimony,
using the frequency of haplotypes to root the tree.
<LI><A NAME="SplitsTree-server"><IMG SRC="icons/split.gif" ALIGN=BOTTOM ALT="[SplitsTree icon here]"></A>
<A HREF="http://bibiserv.techfak.uni-bielefeld.de/splits/">A server</A> at
the Forschungsschwerpunkt Mathematisierung at the University of Bielefeld,
Germany, that
runs the <STRONG>SplitsTree 3.2</STRONG> program.  You paste in a lower-triangular
distance matrix, or one in the NEXUS format, and it returns the results as
a file stored on your system.  <A HREF="software.etc1.html#SplitsTree">SplitsTree</A> is also available for download.
<LI> Chai Mee Joon has made available <a href="http://kun.homelinux.com/Pise/5.a/phylip.html">a server</a> for many PHYLIP 3.6b programs.
<LI> [concerning a still-available server at the University of Wageningen for 
the version 3.5c DRAWTREE and DRAWGRAM programs]
an earlier version of this same server from the University of Nijmegen
<A HREF="http://www.bioinformatics.nl/tools/plottree.html">is available here</A>.
<LI><A NAME="RASA server">James Lyons-Weiler</A> of the Department of Pathology
of the University of Pittsburgh Medical Center has made available
<A HREF="http://bioinformatics.upmc.edu/RASA.html">a server</A> of his RASA
program (RASA itself is not in distribution right now, but I suspect this
will change soon). RASA computes a tree-independent measure of phylogenetic
signal, taxon variance analysis to identify problematic sequences, and
optimal outgroup analysis to identify outgroups.
<LI> The Bioinformatics Centre of the National University of Singapore has
<A HREF="http://sf01.bic.nus.edu.sg/EMBOSS/">a web server using the
EMBOSS package</A> which uses PHYLIP version 3.5c.
<LI><A NAME="RDP-similarity">The 
Ribosomal Database Project</A>
at Michigan State University has
<A HREF="http://rdp8.cme.msu.edu/cgis/sim_matrix.cgi?su=SSU">a server</A>
(located at <CODE>http://rdp8.cme.msu.edu/cgis/sim_matrix.cgi?su=SSU</CODE>
that takes uploaded sequences of either large or small subunit ribosomal RNA,
aligns them against its existing database of ribosomal RNA sequences, and can
provide a similarity matrix between them.  This is related to a distance
matrix, but is <EM>not</EM> corrected for superimposed changes.
<LI><A NAME="COMPARE-server">Em&iacute;lia Martins</A> of the
Department of Biology of the University of Indiana, Bloomington, Indiana
has made available <A HREF="http://compare.bio.indiana.edu/v46/compare.html">a server</A> for
her program <A HREF="software.etc2.html#COMPARE">COMPARE</A> using Java. It
carries out comparative methods analysis.
<LI><a name="NEXUS for PAUP server"> Jonathan Jeffery of the Instituut Biologie Leiden at the University of
Leiden, Netherlands has made
available <A HREF="http://www.donnasaxby.com/nexus.htm">a web server</A>
that prepares a NEXUS file to enable <A HREF="software.pars.html#PAUP">PAUP*</A>
to carry out the advanced search methods of Kevin Nixon and of Quicke, Taylor,
and Purvis to find most parsimonious trees.
<LI><A NAME="eBURST"></a><a href="http://linux.mlst.net/burst.htm#"><strong>eBURST</strong> server</a> (Based Upon Related Sequence Types) from the Department of
Infectious Disease Epidemiology, Imperial College London clusters MLST (Multi
Locus Sequence Typing) data from bacteria.
<LI> <A NAME="PhyloBLAST server"></A><A HREF="http://www.pathogenomics.bc.ca/phyloBLAST/">PhyloBLAST</A> server
uses BLAST to obtain pairwise distances or alignments among protein sequences, then infers
the tree using <A HREF="software.pars.html#PHYLIP">PHYLIP</A>.
<LI> <A NAME="galaxie server">galaxie <a href="http://galaxie.cgb.ki.se/">server</a> fetches matches to a fungal EST sequence and makes a tree with it and them.
<LI> <A NAME="IMGT/PhyloGene server">The IMGT/PhyloGene <a href="http://imgt.cines.fr/IMGTPhylogeny/index.pl">server</a> allows you to select immunoglobin
<LI> <A NAME="Selecton server">Selecton</A> <a href="http://selecton.bioinfo.tau.ac.il/">server</a> uses likelihood and Bayesian methods to identify site-specific positive and purifying selection.
<LI><A NAME="CIPRES web server"> The <a href="http://8ball.sdsc.edu:8888/cipres-web/Home.do">CIPRES (Cybernetic Infrastructure
for Phylogenetic RESearch) web server</a> runs
these programs on sequence data:
<UL>
<LI> <a href="software.etc1.html#RAXML">RAxML</a> for likelihood inference,
<LI> <a href="software.etc2.html#GARLI">GARLI</a> for likelihood inference,
<LI> <a href="software.pars.html#PAUP">PAUP*</a> for parsimony.
</UL> 
</UL>
<li> <a name="PhyloBuilder"><a href="http://phylogenomics.berkeley.edu/cgi-bin/phylobuild/input_phylobuild.py">PhyloBuilder</a>
web server searches for database matches to a seed sequence, aligns the
sequences, and builds a neighbor-joining tree and a multiple alignment.
<li><a name="TreeNotator"><A HREF="http://phylogenomics.berkeley.edu/treenotator/">TreeNotator
server</A> finds and adds Genbank and protein IDs to a tree.
<LI><i>The server is no longer available, although the Java programs are still
distributed -- see the listing mentioned below</i><A NAME="Phyltree-server">A <A HREF="http://aleph0.clarku.edu/~djoyce/java/Phyltree/cover.html">
web server</A> for
<a href="software.etc2.html#Phyltree">Phylap</a>
a Java applet that generates a random tree, a distance matrix for it, and
uses clustering methods to estimate and display the resulting tree and the
true one.  From David Joyce of the Computer Science Department at
Clark University, Worcester, Massachusetts.
</ul>
<p>
 
<hr>
<p>
<CENTER>
<TABLE WIDTH=60%>
<TR ALIGN=CENTER>
<TD BGCOLOR="navy"><STRONG><FONT SIZE=+3 COLOR="cyan">
D</FONT><FONT SIZE=+2 COLOR="cyan">efunct ftp archives of programs</FONT></STRONG></TD>
</TR>
</TABLE>
</CENTER>
<P>
 
<HR>
<P>
<i>Dan was afterwards at Bristol-Meyers-Squibb parmaceuticals (<tt>ddavison<!deathtospam> &nbsp;<!deathtospam>(at)&nbsp;<!deathtospam>timelord.wfd.pri.bms.com</tt>)</i>
<P>
The Houston ftp archive maintained by Dan Davison at the University
of Houston (<CODE>ftp.bchs.uh.edu</CODE>)
seems to have become inactive. 
<P>
 
<HR>
<P>
<i>The mirrors of the IUBIO ftp archive formerly kept in these places seem to
have disappeared:</i>
<P>
<DL>
<DT><B>Japan</B></DT>
<DD>
<UL>
<LI>a <A HREF="ftp://ftp.gdbnet.ad.jp/ftpsync/ftp.bio.indiana.edu">
full mirror</A> is maintained at <CODE>ftp.gdbnet.ad.jp</CODE> in directory
<CODE>ftpsync/ftp.bio.indiana.edu</CODE>
<LI><A HREF=ftp://ftp.nig.ac.jp/pub/mirror/IUBIO/molbio">a mirror of the molbio section only</A> is maintained at
<CODE>ftp.nig.ac.jp</CODE> in directory <CODE>pub/mirror/IUBIO/molbio</CODE></UL></DD>
<DT><B>U.K.</B>
<DD>A
<A HREF="http://mic3.hensa.ac.uk/hosts/iubio.bio.indiana.edu/molbio/">
mirror of the molbio section</A> is maintained at
<CODE>http://mic3.hensa.ac.uk/hosts/iubio.bio.indiana.edu/molbio/</CODE>.
</DD>
<DT><B>India</B>
<DD>
<A HREF="ftp://imtech.chd.nic.in/pub/mirror_sites/iubio/">A mirror of the
molbio section</A> is available at <CODE>imtech.chd.nic.in</CODE> in directory
<CODE>/pub/mirror_sites/iubio/</CODE>
</DD>
<DT><B>Spain</B></DT>
<DD><A HREF="ftp://ftp.uam.es/pub/mirror/molbio">A mirror of the <CODE>molbio</CODE> section</A> is maintained at <CODE>ftp.uam.es</CODE>
at <CODE>pub/mirror/molbio</CODE> </DD>
<DT><B>Israel</B></DT>
<DD><A HREF="ftp://bioinformatics.weizmann.ac.il/pub/software">A mirror of the <CODE>search</CODE>, <CODE>unix</CODE>, <CODE>ibmpc</CODE>, <CODE>mac</CODE>, and <CODE>vax</CODE> subdirectories of the <CODE>molbio</CODE> section</A>
is maintained at <CODE>bioinformatics.weizmann.ac.il</CODE> in directory <CODE>pub/software</CODE></DD>
</DL>
<p>
 
<hr>
<p>
<CENTER>
<TABLE WIDTH=60%>
<TR ALIGN=CENTER>
<TD BGCOLOR="navy"><STRONG><FONT SIZE=+3 COLOR="cyan">
D</FONT><FONT SIZE=+2 COLOR="cyan">efunct lists of  phylogeny programs<BR> (ones no longer distributed)</FONT></STRONG></TD>
</TR>
</TABLE>
</CENTER>
<P>
Here are some now-defunct entries from my list of
<a href="software.html#otherlists">other lists of phylogeny programs</a>:
<UL>
<LI>Classification and clustering programs are described in
<A HREF="http://astro.u-strasbg.fr/~fmurtagh/mda-sw/online-sw.html">a useful Web page</A>
maintained by Fionn Murtagh at
<CODE>http://astro.u-strasbg.fr/~fmurtagh/mda-sw/online-sw.html</CODE>.
It is described by Murtagh as a "<i>very</i> old" list which is no longer being
updated.
Note that inferring phylogenies and making clusters are different
tasks; the software described on that list will be of most use to people who
are trying to cluster or classify but not as much for inferring phylogenies.
<i>(Fionn Murtagh is now <a href="http://www.cs.rhul.ac.uk/home/fionn/">at Royal Holloway College</a> in London, but has not made this list available
there.  However he has list to his own multivariate and
clustering programs there).</i>
<LI>The National Biotechnology Information Facility has
<A HREF="http://www.nbif.org/links/7_1_1_12_2.php">
A list of phylogeny programs</A>
(including some population genetics programs as well) at
<TT>http://www.nbif.org/links/7_1_1_12_2.php</TT>.
<LI>The <A HREF="http://phylogeny.arizona.edu/tree/programs/programs.html">
list of phylogeny software</A> compiled by David Maddison and Wayne Maddison as
part of their <A HREF="http://phylogeny.arizona.edu/tree/phylogeny.html">"Tree of Life"</A> project on the World Wide Web.  Its URL is:
<CODE>http://phylogeny.arizona.edu/tree/programs/programs.html</CODE>.
this list has not been updated in a while.
<LI>Dan Jacobson posted
<A HREF="http://www.bis.med.jhmi.edu/Dan/software/biol-links.html">
an extensive list</A> of biological software and database sites at
<CODE>http://www.bis.med.jhmi.edu/Dan/software/biol-links.html</CODE>.
it has not, as far as I know, been updated in a while.
<LI>The European Bioinformatics Institute (EBI) has created the
<A HREF="http://search.ebi.ac.uk:8888/compass">BioWurld</A> index of resources,
a semi-automated index
of bioinformatics and molecular biology resources.
It is located at <CODE>http://search.ebi.ac.uk:8888/compass</CODE>.
The resources include
Academic Institutes, Departments, and Projects,   Journals and Newsletters,
Commercial Sites and Services ,   Miscellaneous,  Conferences and Courses,
Online Tools and Services,  Databases,   Research, Educational Resources,
Scientists Homepages, Employment, and Software.  Users are invited to
submit URLs to the index (they will decide which to include).  As yet there
are only a few software entries, and growth of this index has been very slow.
<LI>The <A HREF="http://www.phylo.org/links.html">Community Software</A> list
at the CIPRes (CyberInfrastructure for Phylogenetic Research) project lists
phylogeny programs produced by its members with a one-line description for each.
<LI> The <A HREF="http://corba.ebi.ac.uk/Biocatalog/Phylogeny.html">BioCatalog phylogeny
page</A>
at the European Bioinformatics Institute, located at
<CODE>http://corba.ebi.ac.uk/Biocatalog/Phylogeny.html</CODE>.  Last updated
in 2000. <i> This has disappeared from the web site at EBI (www.ebi.ac.uk),
but a new site called www.biocatalog.com contains a banner promising that
it is being constructed.  Is it the same "BioCatalog" or is it something
unrelated?</i>
<LI> <A HREF="http://www.cladistics.org/education.html">A brief list of
programs</A> at the Willi Hennig Society's home pages.  It reflects a rather
different worldview, centered on the parsimony method. <i>The Hennig
Society site has been reorganized and this listing has been replaced by
information about a single program, <a href="software.pars.html#TNT">TNT</a>.</i>
<LI>Stephen A. Smith, of the Department of Ecology and Evolutionary Biology
at Yale University has created the <A HREF="http://www.yphy.org/phycom/">PhyCom</A> "phylogenetic community" web site which lists news about programs, and has
discussions on issues concerned with phylogenies.  A login is required but
this is free. <i>(This site no longer exists).</i>
</i>
<LI>
The Institut Pasteur in Paris has the
<A HREF="http://www.pasteur.fr/recherche/BNB/bnb-en.html">Bio NetBook</A>,
a search facility for biocomputing web pages, which include web pages of
other lists, web pages of programs or servers, and some pages that simply no
longer exist.  It is located at
<CODE>http://www.pasteur.fr/recherche/BNB/bnb-en.html</CODE>.  Programs for
phylogenies can be found by, for example, selecting <TT>software</TT> from the
Resource Type list and <TT>evolution</TT> from the Biological Domain list
without selecting any Organism. Other categories worth looking at include
<tt>Online Analysis Tools</tt>, <tt>Sequence Analysis</tt> and
<tt>Bio-computing</tt>.
<i>The page at this address says that the Bio NetBook has been
discontinued.</i>
<LI> The <a href="http://digitaltaxonomy.infobio.net">Digital Taxonomy web site</a>, maintained by Mauro Jos&eacute; Cavalcanti
in Rio de Janeiro, Brazil, lists phylogeny programs (and programs in other
areas of systematics and morphometrics), with attention restricted to those
that are free and whose source code is publicly available.  An older version
of the same list (but no longer updated) is available
<a
href="http://www.geocities.com/rainforest/vines/8695/software.html">here</a>.
<i>The <tt>digitaltaxonomy.infobio.net</tt>
site exists but contains only a notice that it us "undergoing major
renovations".</i>
<LI><A HREF="http://www.mybio.net/biowiki/Main_Page">Biodirectory.com</A>
maintains a BioWiki site, with content edited by users.  It requires a free
login and is supported by advertising.  It has lists of phylogeny and alignment
programs.<i>This site has disappeared.  It should not be confused with the
completely different BioWiki site of Ian Holmes's lab at University of California, Berkeley.</i>
<LI> Georg Fuellen at the University of Bielefeld, Germany, has a
very good page on
<A HREF="http://www.techfak.uni-bielefeld.de/bcd/Curric/MulAli/welcome.html">
Multiple Alignment Resources</A> at
<CODE>http://www.techfak.uni-bielefeld.de/bcd/Curric/MulAli/welcome.html</CODE>.
It is described as "only updated occasionally"; the most recent links are
about 2006. <i>(F&uuml;llen is now <a href="http://stubber.math-inf.uni-greifswald.de/bioinformatik/">at the University of Rostock</a> but does not seem to have made this list
available there).</i>
</UL>
<P>
 
<div> 
<P>
<font color="#666666">Notices added in compliance with University of Washington
requirements for web sites hosted at the University: &nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.washington.edu/online/privacy" target="_blank"><font
color="#888888">Privacy</font></a>&nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.washington.edu/online/terms"
target="_blank"><font color="#888888">Terms</font></a></font>
</div>
</BODY>
 </HTML>
